# HHS Public Access 

Author manuscript
Cell. Author manuscript; available in PMC 2023 July 21.
Published in final edited form as:
Cell. 2022 July 21; 185(15): 2806-2827. doi:10.1016/j.cell.2022.03.045.

## Therapeutic in vivo delivery of gene editing agents

Aditya Raguram ${ }^{1,2,3, \dagger}$, Samagya Banskota ${ }^{1,2,3, \dagger}$, David R. Liu ${ }^{1,2,3, *}$<br>${ }^{1}$ Merkin Institute of Transformative Technologies in Healthcare, Broad Institute of MIT and Harvard, Cambridge, Massachusetts, USA<br>${ }^{2}$ Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts, USA<br>${ }^{3}$ Howard Hughes Medical Institute, Harvard University, Cambridge, Massachusetts, USA

## Summary

In vivo gene editing therapies offer the potential to treat the root causes of many genetic diseases. Realizing the promise of therapeutic in vivo gene editing requires the ability to safely and efficiently deliver gene editing agents to relevant organs and tissues in vivo. Here, we review current delivery technologies that have been used to enable therapeutic in vivo gene editing, including viral vectors, lipid nanoparticles, and virus-like particles. Since no single delivery modality is likely to be appropriate for every possible application, we compare the benefits and drawbacks of each method and highlight opportunities for future improvements.

## eToC blurb - Liu

Therapeutic gene editing has made exciting progress in the past few years, in part through key advances in in vivo delivery technologies. David Liu and colleagues review the essential characteristics required for efficient in vivo gene editing delivery vehicles, discussing the pros and cons of viral vectors, lipid nanoparticles, and virus-like particles.

## Introduction

The ability to precisely manipulate and edit the sequence of DNA in human cells could enable powerful new classes of genomic medicines. Millions of people worldwide suffer from heritable genetic disorders (Korf et al., 2019), the root causes of which could in principle be corrected by therapeutic DNA editing agents. While traditional gene augmentation therapies can treat some autosomal recessive or haploinsufficiency disorders

[^0]
[^0]:    ${ }_{1}$ Correspondence to: drliu@fas.harvard.edu.
    ${ }^{\dagger}$ These authors contributed equally. Author order was decided by a coin flip.
    Publisher's Disclaimer: This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.
    Declaration of Interests
    The authors have filed patent applications on gene editing technologies and delivery technologies through the Broad Institute of MIT and Harvard. D.R.L. is a consultant and cofounder of Beam Therapeutics, Prime Medicine, Pairwise Plants, Editas Medicine, Chroma Medicine, and Envelop Therapeutics, companies that use and/or deliver genome editing or genome engineering agents.

by providing a functional copy of a gene, gene editing therapies can directly correct pathogenic mutations in genomic DNA,. As such, gene editing in principle could treat a much wider range of genetic diseases, including autosomal dominant disorders, conditions that arise from too little or too much of a gene product, or other conditions for which simple overexpression of a gene cannot optimally rescue the disease. Even for conditions that could be addressed with existing gene augmentation or gene silencing strategies, gene editing therapies that install mutations to increase or decrease the expression of a target gene could achieve the same effect with a one-time treatment, offering the possibility of a permanent cure. More broadly, the risk of suffering from certain major diseases such as coronary heart disease, even in individuals without pathogenic mutations, can be modulated by precise modification of target genes, raising the possibility that gene editing (if shown to be sufficiently safe and efficacious) may one day be used to decrease disease risk in the general population.

The promise of therapeutic gene editing has motivated intense efforts to bring gene editing therapies to the clinic. Recent advances include the development of robust tools for gene editing in mammalian cells, including programmable nucleases, base editors, and prime editors (Anzalone et al. 2020; Doudna, 2020; Newby and Liu, 2021). These gene editing agents have been widely applied to treat numerous disorders with a genetic component across a variety of animal models (Newby and Liu, 2021; Rees et al. 2021). Some of these therapeutic gene editing strategies have already entered clinical trials, with promising early results (Gillmore et al. 2021), and many additional clinical and pre-clinical gene editing programs are underway.

Most current gene editing clinical trials involve ex vivo editing (Wang et al. 2020) in which cells are removed from a patient's body, edited while outside the body, and reintroduced into the patient. This approach is feasible for some important cell types, including hematopoietic stem cells (HSCs) (Ferrari et al. 2021), but most cell types are not amenable to ex vivo manipulation and transplantation into patients. In vivo gene editing, where cells are edited directly within the body, offers the greatest promise for treating genetic disorders. However, in vivo gene editing requires the ability to efficiently and safely deliver editing agents to a large enough fraction of relevant cells directly within the body, which can present a major challenge.

Therapeutic methods for delivering gene editing agents in vivo must efficiently target desired cells and deliver sufficient quantities of editing agents into those cells. To achieve this goal, numerous delivery technologies have been developed and tested in mice, nonhuman primates (NHPs), and other animals (Newby and Liu, 2021; Taha et al. 2022; van Haasteren et al. 2020; Wang et al. 2020; Wei et al. 2020a; Yin et al. 2017a). Two promising delivery strategies—adeno-associated virus (AAV) delivery and lipid nanoparticle (LNP) delivery—have shown initial successes in recent in vivo gene editing clinical trials (Gillmore et al. 2021). These developments suggest that current state-of-the-art delivery methods have the potential to enable powerful new in vivo gene editing therapies in the coming years.

In this review, we summarize three types of gene editing agents that have been used for therapeutic in vivo gene editing and overview essential characteristics of efficient in vivo delivery vehicles. We then describe methods that are commonly used to deliver therapeutic gene editing agents in vivo, with a focus on viral and non-viral delivery methods currently used in ongoing clinical trials, including AAV and LNP delivery. Finally, we discuss virus-like particle (VLP) delivery, a promising emerging method that combines key benefits of both viral and non-viral delivery. As no single delivery approach is likely to be ideal for all possible applications, we note advantages and disadvantages of each strategy and highlight opportunities for further development.

## Therapeutic gene editing strategies

Several types of gene editing agents have been used for in vivo gene editing. Within space limitations, we provide below a brief overview of these gene editing agents (Figure 1) and then discuss approaches for delivering such agents in vivo. Modern methods for therapeutic gene editing have been reviewed extensively elsewhere (Anzalone et al. 2020; Rees and Liu, 2018), and we direct the reader to these reviews for additional details.

### Nucleases

Until recently, all robust strategies for performing gene editing in mammalian cells involved using a nuclease to generate a double-strand break (DSB) at a specific location in genomic DNA. Meganucleases, zinc finger nucleases (ZFNs), and transcription activator-like effector nucleases (TALENs) were the first enzymes used for gene editing in mammalian cells (Urnov, 2018). However, because these editing agents rely on complex protein:DNA interactions to bind specific DNA sequences prior to cutting, a new editing protein must be designed and constructed for each new genomic target site of interest. In practice, this design and/or construction process can be time- and resource-intensive. CRISPR-Cas nucleases revolutionized gene editing because these enzymes can recognize their targets by simple RNA:DNA base-pairing interactions between the target DNA and a single-guide RNA (sgRNA) molecule loaded inside the Cas protein (Doudna, 2020; Jinek et al. 2012). This remarkable feature allows researchers to program CRISPR-Cas nucleases to target and cut different genomic loci simply by changing the sequence of a ~20 base-pair portion of the sgRNA, without needing to design a new Cas protein. Currently, CRISPR-Cas nucleases are by far the most widely used enzymes for generating targeted DSBs in mammalian cells.

In mammalian cells, DSBs are most frequently repaired by non-homologous end joining (NHEJ) (Lieber, 2010) or microhomology-mediated end joining (MMEJ) (Seol et al. 2018), both of which result in an uncontrollable distribution of small insertions and deletions (indels) at the target site. In some situations, DSBs can be repaired via homology-directed repair (HDR) (San Filippo et al. 2008). This process can be templated by an exogenous DNA donor that can contain any arbitrary sequence flanked by regions of homology to the target site and can, in principle, be used to install any desired sequence in genomic DNA. However, HDR is inefficient in most cell types, including non-dividing cells, and successful HDR editing outcomes are generally accompanied by a substantial amount of indels (Chapman et al. 2012; Cox et al. 2015). For these reasons, nuclease-mediated

gene editing is a poor general strategy for precisely correcting a mutated gene back to the wild-type sequence with minimal undesired editing byproducts. Instead, nuclease-generated DSBs are most useful for disrupting coding sequences of genes by introducing frameshift mutations, or for disrupting regulatory motifs in non-coding sequences to modulate gene expression. Indeed, the vast majority of in vivo gene editing with nucleases involves the disruption of targeted genomic loci to induce therapeutic effects (Cox et al. 2015; Li et al. 2020a). Importantly, DSB-mediated indel outcomes are not controllable and typically consist of mixtures of numerous different detectable sequence outcomes (Nambiar et al. 2022), each of which can have a different impact on disease biology.

While programmable nucleases have been applied therapeutically, several undesired consequences of nuclease-mediated gene editing have been reported. In addition to small indels at the target site, generating DSBs in genomic DNA can lead to large deletions (Kosicki et al. 2018; Song et al. 2020), chromosomal translocations (Giannoukos et al. 2018; Stadtmauer et al. 2020; Turchiano et al. 2021; Webber et al. 2019), chromothripsis (Leibowitz et al. 2021), or other chromosomal abnormalities (Alanis-Lobato et al. 2021). Indeed, nuclease-edited chimeric antigen receptor (CAR)-T cells contained chromosomal translocations that persisted in vivo (Stadtmauer et al. 2020). These editing outcomes, while rare, pose safety risks that could negatively impact certain clinical uses of nucleases. These drawbacks of nuclease editing, combined with the fact that nuclease-initiated HDR is inefficient in most therapeutically relevant cell types, have motivated the development of alternative strategies for more precise gene editing.

## Base editors

Base editors (BEs) overcome many of the limitations of nucleases by enabling precise gene correction through single-nucleotide conversions in genomic DNA without requiring DSBs (Rees and Liu, 2018). BEs are composed of DNA-modifying enzymes fused to programmable DNA-binding domains, and many BEs of different types have been reported to date. The first reported BEs were cytosine base editors (CBEs), which perform targeted C•G-to-T•A conversions and consist of cytidine deaminases fused to catalytically impaired Cas enzymes and uracil glycosylase inhibitors (UGIs) (Komor et al. 2016; Nishida et al. 2016). In canonical CBEs, the catalytically impaired Cas enzyme first binds to a specific genomic locus without generating a DSB. Base pairing between the guide RNA and the target DNA strand exposes a single-stranded DNA bubble that is accessible to deamination by the fused cytidine deaminase domain. Because the fused cytidine deaminase is specific for single-stranded DNA substrates, deamination is restricted to a small window within the exposed DNA strand. Deamination of cytosine generates uracil, which is partially protected from base excision by the fused UGIs, resulting in a U•G mismatch at the target DNA locus. The catalytically impaired Cas enzyme selectively nicks the unedited G-containing strand only—without creating a DSB—which biases cellular mismatch repair to replace the unedited strand by using the edited strand as a template. The resulting U•A base pair is eventually converted into a T•A base pair by cellular repair machinery. If the uracil-containing intermediate is not protected and instead the excision of uracil is promoted, the result is often a C•G-to-G•C conversion rather than a C•G-to-T•A conversion (Komor et

al., 2017). This phenomenon was exploited to engineer C•G-to-G•C base editors (CGBEs) (Chen et al., 2021a; Koblan et al., 2021a; Kurt et al., 2021).

Adenine base editors (ABEs) perform A•T-to-G•C conversions via an analogous mechanism (Gaudelli et al., 2017). Since no known natural enzyme catalyzes the deamination of deoxyadenosine that is needed to base edit adenine in DNA, all ABEs described to date are derived from laboratory-evolved deoxyadenosine deaminases fused to catalytically impaired Cas enzymes (Gaudelli et al., 2017; Gaudelli et al., 2020; Richter et al., 2020). ABEs are an especially useful class of base editor because they reverse the most common type of pathogenic point mutation (C•G to T•A), which accounts for approximately half of known pathogenic single-nucleotide polymorphisms (SNPs) (Rees and Liu, 2018).

Mitochondria have their own genomes, and mutations in mitochondrial DNA cause many genetic diseases (Gorman et al., 2016). Editing of mitochondrial DNA is challenged by the lack of efficient methods to deliver guide RNAs into mitochondria, precluding the efficient use of CRISPR-Cas-based gene editing systems. As a result, until recently, only targeted gene destruction, rather than precise gene editing, was possible in mitochondria. The development of DdCBEs, a special class of CBEs that use a unique cytidine deaminase domain that operates on double-stranded DNA substrates, enabled the first precise editing of mitochondrial DNA in living systems (Mok et al., 2020). DdCBEs use TALE proteins instead of Cas domains to direct deamination to a particular DNA locus, which enables CRISPR-free base editing of mitochondrial DNA, in addition to nuclear DNA (Guo et al., 2022; Lee et al., 2021; Lim et al., 2022; Mok et al., 2020; Silva-Pinheiro et al., 2022).

Since the development of the original CBE and ABE, hundreds of base editors with different properties have been reported by many laboratories (Anzalone et al., 2020; Rees and Liu, 2018). BEs have been applied therapeutically for various ex vivo and in vivo gene editing applications to correct disease-causing point mutations or to install single-nucleotide variants that prevent or rescue disease phenotypes (Newby and Liu, 2021).

### Prime editors

While BEs can, in principle, correct the majority of pathogenic SNPs (Rees and Liu, 2018), they cannot perform all possible single-nucleotide conversions and also cannot mediate targeted insertions or deletions. To address these limitations, we developed prime editors (PEs), which enable the programmable installation of any single-nucleotide conversion, small insertion, small deletion, or a combination thereof, without generating DSBs (Anzalone et al., 2019). PEs consist of a reverse transcriptase fused to a Cas9 nickase domain and use an engineered prime editing guide RNA (pegRNA) to both direct the Cas9 nickase to a specific target locus and encode the particular edit of interest. PEs first nick the non-target DNA strand and use the resulting free 3' end to prime reverse transcription using the pegRNA extension as a template. After the desired edit is incorporated into the newly synthesized strand, an additional nick can be used to bias cellular DNA repair to replace the unedited strand by using the edited strand as a template.

Several examples of in vivo gene editing using PEs have been reported (Newby and Liu, 2021). Recent improvements to both the PE protein and pegRNA have enabled highly

efficient prime editing in various cell types and should be useful for future in vivo prime editing efforts (Chen et al. 2021b; Nelson et al. 2021).

## Essential characteristics of efficient in vivo delivery vehicles

Gene editing agents can be delivered into cells either as DNA or mRNA encoding their expression or directly as proteins or (if a guide RNA is required) ribonucleoproteins (RNPs). In all cases, successful in vivo delivery of gene editing agents requires overcoming several biological and molecular barriers to the intracellular delivery of macromolecules. Specifically, an efficient in vivo delivery vehicle must (1) package and protect its cargo from sequestration or destruction before it enters cells, (2) bind desired cells, (3) traverse the target cell membrane to access the cellular interior, and (4) release its cargo into the appropriate intracellular compartment (Figure 2).

Most robust in vivo delivery vehicles encapsulate their cargos in protein or lipid shells to protect them from sequestration or degradation prior to cell entry (Mitchell et al. 2021). This protection enables the cargo to survive in circulation or at the site of administration until the vehicle encounters the target cell types. Additionally, delivery vehicles must avoid recognition by the immune system, as immune activation can cause them to be targeted for degradation. Some vehicle compositions are prone to activating the complement system, which can lead to vehicle clearance by phagocytic immune cells, and antibody-mediated recognition of delivery vehicles can also lead to undesirable phagocytic clearance (Hoshyar et al. 2016; Mitchell et al. 2021).

Before delivering their cargos into cells, in vivo delivery vehicles must first be able to bind those cells within the body. This ability to access target cells is highly dependent on the route of administration of the delivery vehicle. Many intravenously injected vehicles can efficiently access some tissues, such as the liver, but cannot efficiently access others, such as the central nervous system (CNS), due to intrinsic biological barriers (e.g. the blood-brain barrier) (Daneman and Prat, 2015). Locally injecting vehicles into the CNS (e.g. via intrathecal injection) or eye (e.g. via subretinal injection) can circumvent biological barriers and enable access to certain important cell populations (Bottros and Christo, 2014; Peng et al. 2017). However, the ability to physically access a particular cell type does not guarantee efficacious delivery to that cell type. Delivery vehicles must be able to target desired cells, and typically use parts of their surfaces or specific targeting moieties to engage receptors on the surfaces of target cells and promote subsequent cell entry (Mitchell et al. 2021; Paunovska et al. 2022).

Following successful engagement of target cells, delivery vehicles must enter those cells by crossing the cell membrane. In many cases, binding of delivery vehicles to cell surface receptors promotes endocytosis of those delivery vehicles into endosomes (Bareford and Swaan, 2007; Kazmierczak et al. 2020). Vehicles and cargos that remain sequestered in endosomes will eventually be degraded (Smith et al. 2019; Varkouhi et al. 2011). Therefore, successful vehicles must escape endosomes to release their cargos outside of endosomes and into the cell cytosol. Many successful delivery vehicles exploit the acidic environment of endosomes to trigger changes in the vehicle's structure that promote

endosomal escape and cargo release (Mitchell et al., 2021; Staring et al., 2018). Importantly, efficient delivery vehicles must be stable enough to protect their cargo while outside of cells, but must be able to disassemble and release their cargo after entering cells and escaping endosomes.

Over multiple decades, researchers have identified and engineered several classes of delivery vehicles that can overcome these complex molecular obstacles to intracellular delivery. Current state-of-the-art delivery systems, including viral vectors, LNPs, and VLPs, can satisfy these key criteria for efficient in vivo delivery vehicles and are therefore well suited for the in vivo delivery of gene editing agents.

## Viral delivery

Viruses naturally evolved to overcome barriers to in vivo delivery and can natively deliver nucleic acid cargos to many cell types. Because of these favorable characteristics, viruses are promising vehicles for delivering gene editing agents. Many viral vectors have been developed for in vivo gene therapy applications and used to deliver therapeutic genes in over 1,000 clinical trials (Ginn et al. 2018). Most in vivo gene editing applications have utilized adeno-associated viruses (AAVs), and a few pre-clinical studies have used lentiviruses or adenoviruses. Notably, an ongoing clinical trial uses AAVs to deliver gene editing agents into the eye to treat a form of genetic blindness (Sheridan, 2018). Below we provide insights into recent examples of in vivo gene editing using viral vectors and highlight opportunities for future advances.

## Adeno-associated virus (AAV) delivery

The adeno-associated virus (AAV) is a ~25-nm non-enveloped virus composed of 60 copies of viral proteins VP1, VP2, and VP3 assembled into an icosahedral capsid (Drouin and Agbandje-McKenna, 2013). AAVs package a single-stranded DNA genome of ~5 kb (Naso et al. 2017; Wu et al. 2010). Because AAVs have been used to deliver in vivo gene therapies in animal models of human disease (Deverman et al. 2018; Wang et al. 2019), in clinical trials (Mendell et al. 2021), and in FDA-approved therapies (Mendell et al. 2017; Russell et al. 2017), they are currently the most popular viral vectors for delivering macromolecular therapeutics encoded as DNA.

AAV delivery offers many advantages. AAVs have a well-understood and favorable safety profile, are highly biocompatible, and can carry payloads efficiently to a variety of clinically relevant tissues, including the eye (Maguire et al. 2008), liver, brain (Wang et al. 2019), cardiac muscle, and skeletal muscle (Wang et al. 2005). Furthermore, different naturally occurring AAV capsid serotypes can be used to direct AAVs to transduce various tissues in vivo (Wu et al. 2006). Laboratory evolution and rational engineering of AAV capsids have further expanded the available tissue-targeting specificities of AAVs (Asokan et al. 2010; Byrne et al. 2020; Dalkara et al. 2013; Deverman et al. 2016; Goertsen et al. 2022; Li et al. 2008b; Maheshri et al. 2006; Shen et al. 2013; Tabebordbar et al. 2021; Zinn et al. 2015), although few engineered or laboratory-evolved AAVs have entered the clinic as of this writing. The availability of numerous AAV serotypes allows researchers to choose an

appropriate serotype for different applications that require targeting distinct cell populations in vivo.

The size of the nucleic acid cargo is an important consideration when using AAV, as it has a packaging capacity of only ~5 kb of DNA (Dong et al. 1996; Wu et al. 2010). The AAV vector genome must be flanked by two inverted terminal repeats (ITRs) that are required for packaging the vector genome during AAV production, which leaves ~4.7 kb for a transgene cassette. This packaging capacity limits the potential scope of AAVs as delivery vehicles for gene editing agents, as most BEs and PEs that use a canonical S. pyogenes Cas9 (SpCas9) DNA-targeting domain are too large to fit into a single AAV. In addition to packaging DNA encoding the editing agent and, if needed, guide RNA(s), AAVs must also encode promoters driving editor and guide RNA expression, and cis-regulatory elements for efficient activity in vivo. These additional components further increase the required transgene size and limit the effective packaging capacity of a single AAV. To overcome these size limitations, researchers have developed several approaches that enable gene editing agents to be packaged into AAV vectors.

### Development of dual-AAV strategies that effectively reconstitute full-length proteins

To address the packaging limitations of AAV, multiple groups (Chemello et al. 2021; Chen et al. 2020; Chew et al. 2016; Fine et al. 2015; Levy et al. 2020; Li et al. 2008a; Lim et al. 2020; Ryu et al. 2018; Truong et al. 2015; Villiger et al. 2018; Xu et al. 2021) developed strategies for splitting gene editing agents into two halves, such that each half can be packaged separately into individual AAV vectors. These two AAVs are then administered simultaneously, and in cells that are co-transduced by both AAVs, reconstitution of the full-length gene editing agent is achieved via molecular mechanisms acting at either the DNA, pre-mRNA, or protein levels (Tornabene and Trapani, 2020).

Both mRNA and protein trans-splicing strategies have been used to reconstitute full-length gene editing agents that are split into two AAVs. Kim and colleagues used an mRNA trans-splicing strategy to deliver ABEs into mice (Ryu et al. 2018); intramuscular injection of these AAVs into a mouse model of Duchenne muscular dystrophy (DMD) yielded 3.3% base editing. Multiple laboratories (Chemello et al. 2021; Chen et al. 2020; Chew et al. 2016; Fine et al. 2015; Levy et al. 2020; Li et al. 2008a; Lim et al. 2020; Truong et al. 2015; Villiger et al. 2018; Xu et al. 2021) developed split-intein systems in which gene editing agents are reconstituted via protein trans-splicing. In these systems, gene editing agents are split into two halves, each fused to a split intein, and then packaged into two separate AAV capsids. In co-transduced cells, both halves of the editor protein are expressed, and dimerization of the split inteins promotes a partial or complete trans-protein splicing reaction that reconstitutes the full-length editor protein (Aranko et al. 2014).

When we compared mRNA trans-splicing and protein trans-splicing methods, the split intein-mediated protein reconstitution strategy provided, on average, 4.5-fold higher base editing efficiency across multiple tissues in mice (Levy et al. 2020). This efficiency difference likely arises because of the two-step process required for successful trans-mRNA splicing that involves AAV genome concatemerization (Duan et al. 2001) followed by transcription and splicing of the ITR sequences, which have been reported to destabilize

pre-mRNA (Xu et al. 2004). Therefore, the split intein-mediated protein reconstitution strategy is potentially a simpler and more robust strategy for splitting gene editing agents for dual-AAV delivery.

Several studies successfully used the split-intein dual-AAV strategy and achieved editing efficiencies ranging from 9--60% across various therapeutic organs, including the liver, eye, CNS, cardiac muscle, and skeletal muscle (Koblan et al. 2021b; Lau and Suh, 2017; Levy et al. 2020; Rothgangl et al. 2021; Villiger et al. 2018; Yeh et al. 2020). We applied the dual-AAV base editing strategy in a mouse model of Hutchinson-Gilford progeria syndrome (HGPS) and corrected the C•G-to-T•A mutation in the LMNA gene responsible for HGPS (Koblan et al. 2021b). We achieved up to 30% correction of the gene in heart tissue and observed a large reduction in the amount of progerin protein in most tissues examined. Recently, Schwank and coworkers used a dual-AAV9 strategy to deliver an ABE targeting Pcsk9 into mice and achieved 60% base editing in the bulk liver, with a 6.8-fold reduction of serum Pcsk9 protein and a 3.3-fold reduction in serum cholesterol (Rothgangl et al. 2021). Split-intein dual-AAVs have also been used in the CNS to knock out mutant Huntington (HTT) gene (Yang et al. 2017), to correct the disease-causing mutation in a mouse model of Niemann-Pick disease (Levy et al. 2020), and to introduce strop codons in SOD1 to slow disease progression in a mouse model of amyotrophic lateral sclerosis (ALS) (Gaj et al. 2017). Dual-AAVs have also been used to achieve therapeutic levels of gene editing in the skeletal muscle, eye, and ear (Chemello et al. 2021; Jo et al. 2021; Ryu et al. 2018; Yeh et al. 2020).

PEs, which are ~1 kb larger than corresponding BEs, also need to be split into multiple AAVs for successful delivery, and some early reports of in vivo prime editing have used split-intein AAV vectors (Böck et al. 2022; Liu et al. 2021a; Zheng et al. 2022). Schwank and coworkers reported 14% prime editing at the Dnmt1 test site in the mouse liver with dual-AAV8 vectors (Böck et al. 2022), and Xue and coworkers reported 6% prime editing in the mouse liver (Zheng et al. 2022). Dual-AAV delivery of PEs currently yields lower editing efficiency compared to dual-AAV delivery of Cas9 nuclease or BEs, but recent improvements to both the PE protein and pegRNA will likely be useful for improving in vivo prime editing efficiencies (Chen et al. 2021b; Nelson et al. 2021).

### Development of single-AAV vectors enabled by smaller Cas orthologs

While dual-AAV approaches described above have mediated therapeutic editing in mouse models of human disease, single-AAV delivery would offer critical advantages for research and clinical use by simplifying manufacturing and characterization. A single-AAV delivery strategy can also reduce the total dose of AAV required to achieve a desired level of gene editing. Moreover, single-AAV approaches might enable increased editing efficiencies in tissues that are currently difficult to transduce by obviating the need for simultaneous transduction of multiple AAVs.

Efforts to identify smaller orthologs of Cas9 or to generate small engineered Cas9 variants have enabled single-AAV delivery of CRISPR gene editing agents (Kim et al. 2017; Shams et al. 2021; Wang et al. 2020). The Cas9 nuclease from Staphylococcus aureus (SaCas9) is commonly used in single-AAV approaches as it has a gene size of 3.2 kb, which can be

packaged in a single AAV along with one or two sgRNA expression cassettes. Zhang and coworkers harnessed an SaCas9 nuclease-encoding single-AAV vector to knock out Pcsk9 and reduce serum cholesterol in mice (Ran et al. 2015). In addition, an ongoing clinical trial uses a subretinally administered single AAV to deliver SaCas9 nuclease and two sgRNAs to delete a disease-causing mutation in the CEP290 gene in patients suffering from Leber's congenital amaurosis 10 (LCA10) (Maeder et al. 2019; Sheridan, 2018).

More recently, the discovery of other compact Cas9 variants such as Nme2Cas9 (3.24kb, PAM=N_{4}CC) (Edraki et al. 2019; Liu et al. 2021b), CjCas9 (2.95kb, PAM=N_{4}RYAC) (Kim et al. 2017; Li et al. 2020b), and SauriCas9 (3.18kb, PAM=N_{2}GG) (Hu et al. 2020) has increased the number of Cas9 enzymes that can in principle be packaged into single-AAV vectors. These compact Cas9 variants have also broadened the targeting scope of single-AAV gene editing agents beyond that of SaCas9 (3.16kb, PAM=NNGRRT) or engineered variants such as SaKKH (3.16kb, PAM=NNNRRT) (Kleinstiver et al. 2015). In one example, Kim and coworkers developed a single-AAV system using CjCas9 nuclease and administered it subretinally into a mouse model of age-related macular degeneration to knock out VEGF-A (Kim et al. 2017). They observed 20% indels in the retina and retinal pigment epithelium (RPE) cells, which enabled therapeutic rescue with reduced neovascularization. Recently, multiple groups (Davis et al. 2022; Zhang et al. 2022) have used these smaller Cas9 variants to develop single-AAV approaches for packaging BEs that enable higher base editing efficiencies with lower total AAV doses compared to dual-AAV systems. As AAVs can cause dose-limiting toxicity in patients (Kuzmin et al. 2021), reducing total AAV dosing can increase therapeutic potential. Single-AAV approaches thus may offer favorable safety profiles compared to dual-AAV systems.

### Minimizing long-term expression of gene editing agents following AAV

### delivery

One of the outstanding limitations of AAV delivery is that it results in persistent cargo expression in transduced cells. Because AAV genomes are maintained episomally in the nucleus, expression can persist for years (Chu et al. 2003; Vassalli et al. 2003). While this prolonged cargo expression is desirable for gene augmentation therapy applications, it is undesirable for gene editing applications as persistent expression of gene editing agents increases the risks of various types of off-target editing (Anzalone et al. 2020; Doman et al. 2018). Moreover, prolonged expression of Cas9, a non-human protein, can trigger an immune response in which edited cells that express Cas9 are targeted for destruction by the immune system (Charlesworth et al. 2019; Chew et al. 2016; Crudele and Chamberlain, 2018; Wagner et al. 2019; Wagner et al. 2021). To address this issue, various strategies for transiently expressing AAV-delivered gene editing agents have been developed.

Multiple research groups have developed self-inactivating CRISPR/Cas9 AAV systems that use a guide RNA targeting the Cas9 enzyme. This strategy introduces indels into the AAV genome as long as the editor is expressed, which inactivates editor expression over time (Ibraheim et al. 2021; Li et al. 2019a; Li et al. 2019b). Although this approach functioned in mice without significantly reducing on-target editing, editor expression was not completely diminished. Additionally, cleaved AAV products were found to integrate at the on-target genomic locus, raising safety concerns. In another approach, Chen and coworkers developed a unique CBE variant that only becomes active at the on-target site

by proteolysis of a fused deaminase inhibitory domain, which limits the presence of active CBE in cells (Wang et al. 2021). This strategy enabled efficient on-target base editing in the mouse liver with no observable off-target DNA or RNA base editing above background levels.

A particularly noteworthy strategy for temporally regulating the expression of AAV-delivered gene editing agents was reported recently by Davidson and coworkers (Monteys et al. 2021). They developed a universal switch called X^{on} that exploits small molecule-controlled alternative RNA splicing to precisely control AAV transgene expression. In the presence of the small molecule inducer, drug-modulated splicing results in mRNA that includes the exon containing the start codon and leads to full-length protein expression from the AAV genome. However, in the absence of the small molecule, the exon containing the start codon is excluded from the mRNA, which prevents successful protein expression. In this study, the authors also developed a smaller version of X^{on} that can fit along with SaCas9 into a single AAV and showed successful temporal control of gene editing in vivo (Monteys et al. 2021). Given this promising result and the use of a clinical small molecule to trigger the X^{on} system, this approach, as well as others that temporally restrict AAV expression, could be useful for in vivo therapeutic gene editing.

In addition to strategies for temporally regulating the expression of AAV-delivered gene editing agents, researchers have also developed methods to spatially control AAV cargo expression. By using tissue-specific AAV capsids, promoters, or miRNAs, expression of the gene editor cargo can be limited to a particular tissue, which will minimize the potential for off-target editing in non-target tissues. Although naturally occurring or laboratory-evolved AAV capsids have expanded the tissue-targeting scope of AAVs to many tissues, the ability to target a particular tissue does not necessarily entail specificity for that particular tissue over others (Deverman et al. 2016; Tabebordbar et al. 2021). Using tissue-specific promoters to drive editor expression is an attractive strategy, but the size limitation imposed by AAV packaging with Cas9 limits promoter choices. Another strategy to modulate cargo expression in different tissues is to incorporate binding sites for an endogenous miRNA in the 3'UTR of the Cas9 expression cassette (Xiao et al. 2019). In this approach, Cas9 protein expression can be silenced in tissues that highly express the miRNA, and expression will be limited to tissues that lack miRNA expression.

Sontheimer, Niopek, and their respective coworkers combined the miRNA approach with natural inhibitors of Cas proteins known as anti-CRISPRs (Acrs) to limit Cas9 expression to tissues that express the miRNA (Hoffmann et al. 2019; Lee et al. 2019). In this approach, Acr protein expression is silenced in the presence of a miRNA, which in turn allows expression of Cas9 only in tissues that highly express that miRNA. Overall, spatially and temporally controlling the expression of AAV-delivered gene editing agents offers useful strategies to maximize gene editing specificity and thus may improve the safety of future therapeutic applications.

## Lentiviral delivery

Lentiviruses (LVs) are enveloped viruses derived from HIV-1 that are made replication-incompetent by deletions in the 3' LTR and by splitting the necessary components for virus

production into multiple constructs (Dull et al. 1998; Naldini et al. 1996). LVs deliver RNA cargo that is reverse-transcribed and stably integrated semi-randomly into the genome of transduced cells. Integrase-deficient lentiviral vectors (IDLVs) have also been engineered, in which the integrase domain has been inactivated so that the viral cDNA persists episomally following reverse transcription (Wanisch and Yanez-Munoz, 2009). LVs have been used primarily for ex vivo gene delivery, mostly in HSCs and T cells, and are currently used in two FDA-approved chimeric antigen receptor (CAR)-T therapies (Mullard, 2017).

LVs possess several advantages that make them attractive for genome editing. First, LVs can accommodate up to 10 kb of cargo DNA (Sweeney and Vink, 2021), which is sufficient to package virtually all known gene editing agents into a single vector. The large cargo packaging capacity of LVs also makes them well suited for multiplex genome editing using CRISPR-based agents, which requires the packaging of multiple sgRNA expression cassettes (Kabadi et al. 2014). Second, LVs can efficiently transduce both dividing and non-dividing cells (Kumar et al. 2001). Third, IDLV genomes can also be used as HDR templates (Lombardo et al. 2007). Finally, the tropism of lentiviruses can be readily modulated by changing the envelope glycoprotein used to pseudotype the virions (Cronin et al. 2005; Joglekar and Sandoval, 2017).

There are few examples of using LVs for in vivo gene editing. Palczewski and coworkers administered an ABE- and sgRNA-encoding LV subretinally to correct a premature stop codon in the Rpe65 gene in a mouse model of Leber congenital amaurosis (Suh et al. 2021). A single dose of lentivirus injected into 4-week-old mice resulted in 15% base editing at the target site and restored near-normal levels of visual function. In vivo delivery using LVs to other organs, including the bone marrow, brain, and liver, has also been demonstrated, although these applications are limited to gene augmentation therapy and not gene editing (Dalsgaard et al. 2018; Milone and O'Doherty, 2018; Richter et al. 2017).

A significant disadvantage of using LVs for in vivo delivery applications is the potential for genomic integration, which can lead to detrimental outcomes. Although episomal transgenes from IDLV vectors are designed to be non-integrating, they retain residual genome integration frequencies (Kymalainen et al. 2014; Wang et al. 2010) and also still lead to prolonged expression of the editing agent, which increases risks associated with off-target editing. Notably, the use of LVs in in vivo gene augmentation therapy clinical studies has raised concerns about genotoxicity, immunogenicity, and the high cost of manufacturing (Milone and O'Doherty, 2018), all of which may limit the use of LVs for in vivo gene editing applications.

### Adenoviral delivery

Adenovirus (Ad) is an icosahedral non-enveloped virus, 90--100nm in size, with a large (36 kb) genome (Lee et al. 2017). Adenoviruses deliver DNA cargo that is then episomally maintained in the nucleus of transduced cells (Lee et al. 2017). Ad is the most commonly used viral vector in gene therapy clinical trials worldwide (accounting for >20%), primarily because of its large cargo packaging capacity, well-defined biology, genetic stability, high transduction efficiency, and ability to be produced at high titers on a large scale (Lee et al. 2017). Moreover, there are 57 known serotypes of Ad that infect humans (Lee et al.

2017) and ~100 that infect primates (Nelson and Gersbach, 2016), allowing researchers to modulate Ad tropism by using different capsids.

In 2017, Musunuru and coworkers used an Ad to systemically deliver an early-generation CBE into mice, resulting in 28% base editing of Pcsk9 in the liver and a 28% reduction in cholesterol levels four weeks after injection (Chadwick et al. 2017). In another study, Lieber and coworkers used Ads to deliver ABEs to HSCs in vivo (Li et al. 2021). The ABEs were designed to disrupt repressor binding sites in the fetal hemoglobin promoter, which can upregulate fetal hemoglobin expression as a potential therapeutic strategy to treat sickle cell disease and β-thalassemia (Li et al. 2021). This study was the first to report therapeutic in vivo base editing in hematopoietic stem and progenitor cells (HSPCs). Their approach used two Ads: one Ad delivered the base editor cargo and MGMJ^{P140K}, a selectable marker, flanked by inverted repeats for genomic integration, and the second Ad contained the transposase and recombinase machinery required to integrate the selection marker into the genome of transduced cells. After 16 weeks post-Ad treatment and four selection rounds, the researchers observed 20% editing of the target site in HSPCs, which led to therapeutic levels of fetal hemoglobin expression. Ads have also been used recently for in vivo prime editing applications. Schwank and coworkers used an Ad to deliver PE2 without the RNaseH domain to neonatal or adult mice (Böck et al. 2022). They observed 58% and 36% prime editing in the hepatocytes of neonates and adult mice, respectively.

While using Ads to deliver gene editing agents has yielded efficient in vivo editing, it has also led to the generation of neutralizing antibodies against Cas9 (Wang et al. 2015), potentially due to the immunogenic nature of the vector. Hence, drawbacks of using Ads for in vivo gene editing applications include immunogenicity and its inherently high adjuvant nature that can lead to T cell-mediated cytotoxicity (Geutskens et al. 2000; Raper et al. 2003). Efforts to make the virus “stealth-like” by minimizing the expression of viral antigens can significantly reduce its immunogenicity (Lee et al. 2017). The use of adenoviruses as COVID-19 vaccines has generated new excitement around the technology. However, broader applications of Ad for in vivo gene editing will require further engineering efforts.

### The future of in vivo gene editing using viral vectors

Overall, viral vectors have shown great promise for delivering gene editing agents in vivo across many pre-clinical studies and one ongoing clinical trial. To date, viral vectors offer some of the highest gene editing efficiencies observed across many organs due to their inherent abilities to potently transduce diverse cell types in vivo and deliver their nucleic acids cargos. Future improvements to viral vectors will require careful efforts to overcome the challenges outlined above, including the immunogenicity of the vector, prolonged expression of the gene editing agent, off-target gene editing, potential for genomic integration, manufacturing cost, and dose-limiting toxicity (Figure 3). Vector engineering approaches to improve the potency and tissue specificity could reduce the required dose and reduce the cost of manufacturing of viral delivery platforms. Methods to durably silence cargo expression after on-target editing will also substantially improve the safety profile of viral delivery. As discussed below, hybrid viral and non-viral strategies

could offer the best of both worlds by combining the robust efficiency of viral delivery with the transient nature of non-viral delivery approaches.

## Lipid nanoparticle (LNP) delivery

Lipid nanoparticles (LNPs) have grown increasingly popular as non-viral vehicles for delivering gene editing agents in vivo. For decades, LNPs have been used to deliver nucleic acid cargos, including siRNAs and therapeutic mRNAs (Cullis and Hope, 2017; Paunovska et al. 2022). To deliver their encapsulated payloads into target cells, they first enter cells through endocytosis, escape endosomes by disrupting endosomal membranes after endosome acidification, and subsequently gain access to the target cell cytosol (Gilleron et al. 2013; Wittrup et al. 2015). LNPs are completely synthetic and are typically composed of four components: a cationic or ionizable lipid, a helper lipid, a polyethylene glycol (PEG)-lipid, and cholesterol (Paunovska et al. 2022) (Figure 4). Varying the identities of these components can yield LNPs with different properties, including distinct pharmacokinetic profiles and abilities to target different cell types (Paunovska et al. 2022). Following extensive development and optimization, LNPs have been approved for use in humans by the US FDA, including via intravenous administration to deliver therapeutic siRNAs to hepatocytes (Adams et al. 2018) and via intramuscular administration to deliver mRNA vaccines (Baden et al. 2021; Polack et al. 2020). As discussed below, LNPs are already being used in a clinical trial to deliver Cas9 nuclease mRNA to the liver and are poised to become a delivery vehicle of choice for many clinical in vivo gene editing applications.

## LNPs for liver delivery

Most intravenously administered nanoparticles accumulate in the liver (Paunovska et al. 2022). Specifically, many LNPs become coated with ApoE lipoproteins in the bloodstream, which leads to LNP uptake by hepatocytes mediated by ApoE:LDL receptor interactions (Akinc et al. 2019; Paunovska et al. 2022). For these reasons, LNPs have thus far been most commonly used to deliver therapeutic cargos to the liver.

While LNPs were originally optimized for delivering siRNAs, key advances in LNP formulations enabled efficient encapsulation and delivery of mRNAs instead of siRNAs. Anderson and coworkers optimized a lipid formulation for mRNA delivery (Kauffman et al. 2015) and used this formulation to deliver SpCas9 nuclease mRNA to the mouse liver (Yin et al. 2016). The SpCas9-encoding mRNA was chemically modified to include pseudouridine and 5-methylcytidine, which was important for increasing mRNA stability and reducing cellular innate immune responses to foreign RNAs (Kariko et al. 2008). Initially, only SpCas9 mRNA was delivered within LNPs; an sgRNA expression cassette and HDR donor DNA template were provided on an AAV8 vector that was co-administered along with the LNPs (Yin et al. 2016). This approach led to 24% indels and 0.8% correction of a tyrosinemia-causing mutation in the bulk mouse liver, which was sufficient to cure the disease through the increased fitness of edited hepatocytes and the eventual replacement of non-edited liver cells with edited ones in the treated animals. Subsequently, Anderson and coworkers demonstrated that chemically modifying the sgRNA to include a specific

combination of 2'OMe, 2'F, and phosphorothioate linkages enabled more efficient editing when these sgRNAs were encapsulated in LNPs (Yin et al. 2017b). In mice, an intravenous injection of LNPs that co-encapsulated SpCas9 mRNA and chemically modified sgRNAs targeting Pcsk9 led to 80% editing and a reduction of serum Pcsk9 to undetectable levels (Yin et al. 2017b). These results were some of the first to demonstrate the promise of using LNPs encapsulating Cas9 nuclease mRNA and chemically modified sgRNAs to achieve therapeutic levels of gene editing in mice.

Many other groups have also developed LNP formulations that enable efficient delivery of Cas9 nuclease mRNA and sgRNAs to the mouse liver. Siegwart and coworkers developed zwitterionic amino lipid (ZAL) nanoparticles, which successfully co-delivered Cas9 nuclease mRNA and sgRNA to the liver of nuclease reporter mice (Miller et al. 2017). Dong, Tan, and coworkers developed TT3-based lipid-like nanoparticles (LLNs) and demonstrated that they could achieve 30% indels at Pcsk9 in the liver after intravenously injecting LLN-encapsulated Cas9 nuclease mRNA and sgRNA into mice (Jiang et al. 2017). Xu and coworkers used bioreducible LNPs containing integrated disulfide bonds to encapsulate Cas9 nuclease mRNA and sgRNA, achieving 20% editing of Pcsk9 and 39% editing of Angptl3 in the mouse liver (Liu et al. 2019; Qiu et al. 2021). Collectively, these studies demonstrate that various LNP compositions can support efficient delivery of Cas9 nuclease mRNA and sgRNAs to the liver.

In an ongoing phase 1 clinical trial, Gillmore and coworkers recently demonstrated the efficacy of an mRNA LNP approach for in vivo liver gene editing in six patients with hereditary transthyretin amyloidosis (Gillmore et al. 2021). Knockdown of transthyretin (TTR) protein levels reduces ongoing TTR amyloid formation, which can improve disease outcomes (Adams et al. 2018). Previously, the researchers had reported that LNP-delivered Cas9 nuclease mRNA and a mouse Ttr-targeting sgRNA successfully disrupted Ttr in the liver and led to a substantial and durable reduction in Ttr protein levels (Finn et al. 2018). Additional preclinical studies in cynomolgus monkeys demonstrated 73% TTR disruption in the liver and a corresponding >94% reduction in serum TTR protein that was sustained over a period of 12 months (Gillmore et al. 2021). The clinical data revealed that patients who had received the intravenously administered LNP-based drug at a dose of 0.1 mg/kg or 0.3 mg/kg exhibited reductions in serum TTR levels of 53% or 87%, respectively, with minimal adverse effects reported. These results were the first to establish in vivo Cas9 nuclease gene editing in the liver mediated by RNA-encapsulating LNPs as a therapeutic strategy in humans.

In addition to delivering Cas9 nuclease mRNA, LNPs have also been used to deliver base editor mRNA to the livers of mice and non-human primates. Xue and coworkers observed 12.5% base editing of a tyrosinemia mutation in the mouse liver mediated by LNP delivery of ABE mRNA (Jiang et al. 2020), and Schwank and coworkers observed 10% base editing of a phenylketonuria mutation in the mouse liver mediated by LNP delivery of SaCas9-BE3 mRNA (Villiger et al. 2021). Recently, separate studies led by Kathiresan, Schwank, and their respective coworkers reported highly efficient (>60%) base editing to disrupt a splice site in PCSK9 in the livers of mice and cynomolgus monkeys, which was mediated by LNP delivery of ABE mRNA (Musunuru et al. 2021; Rothgangl et al. 2021). A single LNP

administration in cynomolgus monkeys led to a substantial (90%) and sustained (>8 months) knockdown of serum PCSK9 protein and a 60% reduction in blood cholesterol (Musunuru et al. 2021). These promising preclinical results using ABE mRNA LNPs in NHPs, combined with the promising clinical results using Cas9 nuclease mRNA LNPs in human patients, suggest that LNPs could be used in the future to mediate in vivo liver base editing treatments for indications such as hypercholesterolemia and other genetic liver diseases.

### LNPs for non-liver delivery

Because most intravenously administered LNPs naturally accumulate in the liver, achieving non-liver gene editing mediated by LNPs is challenging (Wei et al. 2020a). One approach for subverting the natural liver-targeting nature of LNPs is to administer them by local injection rather than intravenous injection. Multiple laboratories have previously reported successful nuclease editing and base editing in the mouse inner ear and retina following local administration of lipid-encapsulated RNPs (Gao et al. 2018; Jang et al. 2021; Yeh et al. 2018; Zuris et al. 2015). However, the ability to use systemically administered LNPs to deliver gene editing agents to non-liver tissues would greatly expand the therapeutic applicability of LNP delivery.

Many groups have pursued the development of LNPs that target non-liver tissues. Dahlman and coworkers developed strategies for simultaneously screening hundreds of different LNPs in vivo to identify LNP compositions that enable non-liver delivery (Dahlman et al. 2017; Sago et al. 2018). These strategies mark distinct LNP formulations with unique DNA barcodes, inject pooled barcoded LNP libraries into mice, and sequence the barcodes extracted from a tissue of interest to reveal the identity of the LNP(s) that enabled delivery to that tissue. Using these strategies, Dahlman and coworkers identified LNPs that delivered Cas9 nuclease mRNA and sgRNA in mice to splenic endothelial cells as efficiently as to hepatocytes (Sago et al. 2018). Siegwart and coworkers developed selective organ targeting (SORT) LNPs by adding an additional charged lipid component to modulate the internal charge of the particles without substantially disrupting the standard four-component nature of efficient LNPs (Cheng et al. 2020; Wei et al. 2020a). They found that changing the charge and concentration of this additional component was sufficient to direct LNPs to either the lung or spleen without targeting the liver in mice (Cheng et al. 2020). These SORT LNPs were used to deliver Cas9 mRNA and sgRNA specifically to the lung, achieving 15% editing of bulk lung tissue. Some SORT LNPs can be formulated with permanently cationic lipids and therefore can be assembled in neutral instead of acidic buffers, which enabled packaging of Cas9 RNPs into LNPs for the first time (Wei et al. 2020b). Together, these studies and others have demonstrated that LNP compositions can be altered to modulate tissue-targeting capabilities, although specific rules for retargeting LNPs in this way remain unknown.

Another strategy for directing LNPs to non-liver tissues involves conjugating targeting groups such as antibody fragments to the surface of LNPs (Kedmi et al. 2018; Paunovska et al. 2022; Veiga et al. 2018). A particularly noteworthy example of this strategy was recently reported by Epstein and coworkers, who used intravenously administered anti-CD5 antibody-conjugated LNPs to target T cells and transiently generate chimeric antigen

receptor T cells that could treat cardiac injury in mice (Rurik et al. 2022). While these active targeting approaches have not yet been applied to deliver gene editing agents to non-liver tissues, they offer the potential to enable non-liver in vivo gene editing using systemically administered LNPs in the near future.

### Advantages of LNP delivery and future prospects

LNP delivery offers several advantages over viral delivery, especially when delivering gene editing agents. LNP delivery results in transient expression of gene editing agents, which is known to minimize the potential for off-target editing relative to prolonged expression from episomal or integrated viral genomes (Newby and Liu, 2021). Prolonged expression of gene editing agents could also result in immune recognition of edited cells, which might impact the long-term persistence of edited cells (Wagner et al. 2021). Additionally, since LNPs are synthetic, the immunogenicity of LNPs is much lower than that of viruses and can support repeat dosing in some cases (Kenjo et al. 2021). Currently used LNP components are typically biodegradable and non-toxic in vivo (Maier et al. 2013; Witzigmann et al. 2020). Doses of LNPs that are sufficient to support robust gene editing have not shown significant adverse effects in mice or NHPs and have thus far shown good safety profiles in humans (Gillmore et al. 2021). Importantly, LNP manufacturing for large-scale production has been demonstrated to be feasible (Schoenmaker et al. 2021), opening up avenues for additional clinical programs that use LNP delivery for in vivo gene editing.

The development of LNPs that enable efficient non-liver delivery remains a critical goal for the therapeutic gene editing field. Understanding the mechanisms by which different LNP formulations enable different tissue-targeting properties might enable better methodologies for engineering new LNPs with desired targeting capabilities (Dilliard et al. 2021). Cell types of high interest include hematopoietic stem cells (HSCs), as LNPs capable of delivering gene editing agents to bone marrow HSCs following an intravenous or intraosseous injection could revolutionize the treatment of genetic blood disorders by obviating the need to harvest, edit ex vivo, and transplant patient HSCs. Overall, given their recent successes as delivery vehicles for multiple types of therapeutic RNAs in humans (Adams et al. 2018; Baden et al. 2021; Gillmore et al. 2021; Polack et al. 2020), LNPs are likely to be used extensively for in vivo gene editing in the liver and potentially in other organs.

## Virus-like particle (VLP) delivery

Virus-like particles (VLPs) have emerged as potentially promising vehicles for delivering gene editing agents. VLPs are non-infectious assemblies of viral proteins that package desired cargo mRNAs, proteins, or RNPs in addition to or instead of viral genetic material (Lyu et al. 2020). Because VLPs are derived from existing viral scaffolds, they exploit natural properties of viruses that enable efficient intracellular delivery, including their ability to encapsulate cargos, escape endosomes, and be reprogrammed to target different cell types. However, unlike viruses, VLPs transiently deliver gene editing agents as mRNA or protein instead of as DNA, which substantially reduces the risks of off-target gene editing and viral genome integration (Chandler et al. 2017). For these reasons, VLPs are attractive vehicles

for delivering gene editing agents as they can offer key benefits of both viral and non-viral delivery.

Nearly all reported VLP architectures for delivering mRNA or protein cargos are based on retroviruses, as retroviruses possess several characteristics that are ideal for VLPs. Immature retroviral particles are spherical and typically lack rigid structural symmetry (Zhang et al. 2015), which allows increased flexibility to encapsulate desired cargos compared to most non-enveloped icosahedral viruses. Furthermore, the large particle diameter (100--200 nm) of retroviruses (Zhang et al. 2015) provides more physical space for packaging large cargos such as Cas9. Finally, retroviruses are inherently modular with respect to cell targeting and cargo packaging; cell-type specificity is dictated by the envelope glycoproteins, and cargo packaging is controlled by the capsid proteins (Cronin et al. 2005). This modularity suggests that a VLP capsid architecture that efficiently packages desired cargo could be readily combined with various existing envelope glycoproteins that are currently used to modulate the tropism of retroviruses. While retroviral VLPs have been explored for decades as delivery vehicles for mRNAs and proteins, recent efforts were the first to realize the potential of VLPs to mediate efficient in vivo delivery of gene editing agents.

### mRNA-packaging VLPs

Packaging a desired mRNA cargo within VLPs requires a molecular mechanism by which a specific mRNA can be recognized by viral capsid proteins and subsequently incorporated into virions. Retroviral RNA genomes contain a packaging signal (Ψ) that directs the encapsulation of viral RNA into virions, and thus an mRNA cargo engineered to contain Ψ should similarly be incorporated into virions. Some of the first mRNA-packaging VLPs designed by Baum and coworkers used Ψ to encapsulate Cre-recombinase mRNA into murine leukemia virus (MLV) particles (Galla et al. 2004). Importantly, this Ψ-containing mRNA was additionally modified so that it would not be reverse transcribed by the MLV reverse transcriptase, resulting in transient delivery of mRNA rather than stable integration of viral cDNA into the genomes of transduced cells. However, only two copies of Ψ-containing RNA could be packaged per viral particle, which motivated the development of alternative strategies to package greater amounts of mRNA cargo into VLPs.

To improve the mRNA-packaging potential of VLPs, Pagès and coworkers used the interaction between the MS2 coat protein (MS2cp) and MS2 aptamer (MS2apt) to direct packaging of mRNA cargo into modified HIV-1 particles (Prel et al. 2015). In their designs, they replaced the ZF2 domain of the HIV-1 nucleocapsid with MS2cp and included twelve copies of MS2apt at the 3' end of a luciferase mRNA cargo. This approach enabled the packaging of 5-6 copies of luciferase mRNA per VLP, an improvement over Ψ-mediated RNA packaging. Therefore, the strategy of modifying retroviral capsid proteins to include MS2cp and cargo mRNAs to include MS2apt was adopted as a promising way to generate mRNA-packaging VLPs (Figure 5).

Several groups have since demonstrated the use of MS2cp/MS2apt to package Cas9 nuclease mRNA into VLPs. Galla and coworkers fused two copies of MS2cp to the C-terminus of MLV gag along with two copies of MS2apt within the 3' UTR of SpCas9 mRNA and at the 3' end of the sgRNA (Knopp et al. 2018). This strategy enabled successful delivery

to HEK293T cells, Jurkat cells, and primary human fibroblasts, but insufficient delivery of the sgRNA limited gene editing efficiencies. Lu and coworkers fused two copies of MS2cp directly downstream of the HIV-1 nucleocapsid ZF2 domain along with one copy of MS2apt within the 3' UTR of SaCas9 mRNA (Lu et al. 2019). In this system, the SaCas9 sgRNA was encoded in a separate IDLV that was used to co-transduce target cells along with SaCas9 mRNA-containing VLPs. This SaCas9 VLP plus IDLV system exhibited efficient editing in HEK293T cells, but its efficiency was not evaluated in other cell types or in vivo.

Cai and coworkers developed a similar system (mLPs) for packaging SpCas9 mRNA into HIV-1 VLPs (Ling et al. 2021). In mLPs, one copy of MS2cp is fused to the N-terminus of HIV-1 gag-pol and six copies of MS2apt are added within the 3' UTR of SpCas9 mRNA. Additionally, Cai and coworkers produced all-in-one mLPs that packaged both SpCas9 mRNA as well as an IDLV genome expressing an SpCas9 sgRNA. mLPs displayed efficient editing in HEK293T, NIH3T3, K562, and Jurkat cells. Notably, a single subretinal injection of mLPs into mice mediated 44% knockout of Vegfa in retinal pigment epithelial (RPE) cells, which was sufficient to prevent wet age-related macular degeneration. In a separate study, Cai and coworkers also demonstrated that an intracorneal injection of mLPs loaded with SpCas9 mRNA and two sgRNA expression cassettes cured herpetic stromal keratitis in mice by simultaneously targeting two essential herpesvirus genes (Yin et al. 2021). These results highlight the in vivo therapeutic utility of Cas9 nuclease mRNA-packaging VLPs for the treatment of ocular diseases.

One drawback of using mRNA-packaging VLPs for the delivery of Cas9-based gene editing agents is that there are various challenges associated with sgRNA delivery. Guide RNAs that are not chemically modified are rapidly degraded unless they are protected by complexing with Cas9 protein (Allen et al. 2020). Guide RNAs packaged alongside Cas9 mRNAs in VLPs may therefore be substantially degraded before Cas9 protein is synthesized in the transduced cells. Although sgRNA expression cassettes encoded on IDLVs enable efficient editing by Cas9 mRNA VLPs, such cassettes persist as episomal DNA in transduced cells. As previously noted, while IDLVs exhibit substantially minimized rates of genomic integration compared to integration-competent lentiviral vectors, they still support a detectable frequency of genomic integration (Kymalainen et al. 2014; Wang et al. 2010), which increases the risks of this approach.

### Protein- or RNP-packaging VLPs

Packaging desired protein or RNP cargos within VLPs requires a strategy for localizing target proteins into VLPs as they form. To accomplish this, researchers have fused desired cargo proteins to viral structural proteins, including at various locations within retroviral gag polyproteins; this strategy directs the cargo into virions during the capsid self-assembly process (Kaczmarczyk et al. 2011; Voelkel et al. 2010) (Figure 5). In most cases, the gag and cargo are linked by a short peptide sequence that is cleaved by the co-encapsulated viral protease following virion maturation (after the cargo is successfully packaged), enabling cargo release into the transduced cells (Kaczmarczyk et al. 2011; Voelkel et al. 2010). This approach has been used to package and deliver various protein cargos within VLPs (Cai et al. 2014).

Similar strategies were used to package Cas9 nuclease protein into VLPs. Manjunath and coworkers reported VLPs that contained SpCas9 fused to the N-terminus of HIV-1 gag-pol via a HIV-1 protease-cleavable linker and expressed sgRNAs from a cassette encoded on a co-packaged lentiviral genome (Choi et al. 2016). This VLP construct achieved 14--28% editing in Jurkat cells. Doudna and coworkers reported related VLPs that contained SpCas9 fused to the C-terminus of HIV-1 gag via a HIV-1 protease-cleavable linker (Hamilton et al. 2021). In this Cas9-VLP construct, the sgRNA was either encoded on a co-packaged lentiviral genome or expressed from a non-lentiviral plasmid during VLP production. The latter approach for sgRNA packaging relies on the high affinity between Cas9 and its sgRNA, which enables gag--Cas9 fusions to be loaded with sgRNAs prior to packaging within VLPs. These Cas9-VLPs enabled up to 90% editing in Jurkat cells and up to 60% editing in primary human T cells. These efficiencies represented substantial improvements over previous results, likely because of the improved Cas9 fusion orientation at the C-terminus rather than the N-terminus of HIV-1 gag, as well as the ability to package Cas9/sgRNA RNPs rather than Cas9 protein alone. Additionally, Doudna and coworkers demonstrated that Cas9-VLPs could be targeted to specific T cell subpopulations by pseudotyping the particles with different envelope glycoproteins (Hamilton et al. 2021).

Ricci and coworkers also leveraged gag--Cas9 fusions to generate RNP-packaging VLPs (Mangeot et al. 2019). These VLPs, also termed “nanoblades”, contained SpCas9 fused to the C-terminus of MLV gag via a MLV protease-cleavable linker and expressed sgRNAs from a non-viral plasmid during VLP production to enable direct packaging of Cas9 RNPs. Nanoblades displayed efficient editing in vitro in HEK293T cells (80-90%), primary human T cells (30%), primary human HSPCs (40%), and other cell types. Notably, a single intravenous injection of nanoblades into mice achieved up to 10% editing in the liver, representing the first demonstration of the in vivo efficacy of Cas9 nuclease RNP-packaging VLPs.

Other groups have developed strategies for packaging Cas9 RNPs into VLPs that do not involve gag--Cas9 fusions. Indikova and Indik fused Cas9 to the C-terminus of HIV-1 VPR, an accessory protein that is packaged into HIV-1 particles via interactions with the p6 domain of HIV-1 gag (Indikova and Indik, 2020). This VLP construct achieved >90% editing in HEK293T cells but lower efficiency in primary human T cells (15%) compared to the Doudna group's HIV-1 gag-fused Cas9-VLPs. Lu and coworkers utilized aptamer and aptamer binding protein interactions, which were previously used to package Cas9 mRNA within VLPs, to instead package Cas9 RNPs (Lu et al. 2021; Lyu et al. 2019). They replaced the tetraloop of the sgRNA with a com aptamer, fused a Com-binding protein directly downstream of the HIV-1 nucleocapsid ZF2 domain, and expressed these constructs along with free Cas9 protein in VLP producer cells. In this approach, RNP packaging is driven by the sgRNA:VLP capsid interaction and requires Cas9 protein to complex with aptamer-containing sgRNA prior to RNP loading into particles. Lu and coworkers also showed that this strategy could be used to package adenine base editor RNPs in addition to Cas9 nuclease RNPs (Lyu et al. 2021). These VLP constructs achieved 70-80% editing in HEK293T cells. Hotta and coworkers employed a distinct strategy for RNP packaging that used the small molecule AP21967 (a rapamycin analog) to dimerize FRB--Cas9 fusions with FKBP12--HIV-1gag fusions during particle formation in producer cells (Gee et al. 2020).

This strategy, also termed “NanoMEDIC”, mediated 40% deletion of dystrophin exon 45 in DMD patient-derived iPSCs and 6% deletion of exon 45 in gastrocnemius muscle tissue following intramuscular injection into mice.

While various Cas9 RNP-packaging VLPs exhibited promising efficiencies in vitro, all of the systems described above were either not tested in vivo or exhibited low in vivo efficacy (<10% editing). We recently developed engineered VLPs (eVLPs) based on Moloney MLV (MMLV) that package Cas9 nuclease or base editor RNPs and mediate potent, therapeutic levels of gene editing across multiple organs in mice (Banskota et al. 2022). We identified key bottlenecks that limit VLP potency in vivo and engineered solutions in eVLPs to overcome these bottlenecks. First, we engineered the protease-cleavable linker sequence between the MMLV gag and protein cargo to improve cargo release after eVLP maturation while minimizing premature cleavage prior to particle formation. Next, we added nuclear export sequences to modulate the localization of the MMLV gag--cargo fusion selectively in producer cells and substantially improve cargo loading into eVLPs. Finally, we engineered an optimal stoichiometry of viral structural components (MMLV gag-pro-pol) and cargo to maximize eVLP efficiencies. eVLPs mediated efficient gene editing in vitro in HEK293T cells (>95% editing), 3T3 fibroblasts (>80% editing), Neuro-2a cells (>90% editing), primary human and mouse fibroblasts (>90% editing), and primary human T cells (50--60% editing). eVLPs successfully delivered base editor RNPs in vivo following a local injection into the mouse brain, resulting in 5% bulk editing and 60% editing in cells enriched for VLP transduction. Additionally, a single subretinal injection of eVLPs into a mouse model of genetic blindness efficiently corrected the disease-causing point mutation (20--30% editing in mouse RPE cells) and improved visual function. Finally, a single intravenous injection of eVLPs into mice achieved 63% editing of Pcsk9 in the liver and 78% knockdown of serum Pcsk9 levels, which are comparable to Pcsk9 editing and knockdown efficiencies observed with AAV and LNP mRNA delivery. These results demonstrate that rational engineering of VLP architectures to improve potency was required to enable efficient in vivo editing by RNP-packaging VLPs and establish eVLPs as the most potent RNP-packaging VLPs reported to date.

### Advantages of VLP delivery and future prospects

A major advantage of using VLPs to deliver gene editing agents is that VLP delivery results in minimal off-target editing. We and others have demonstrated that VLPs offer substantially minimized off-target editing relative to plasmid and viral delivery in vitro (Banskota et al. 2022; Mangeot et al. 2019). Additionally, we recently demonstrated that RNP-packaging eVLPs offer minimal off-target editing in vivo, including minimized off-target DNA base editing in the mouse liver relative to AAV delivery and minimized off-target RNA base editing in the mouse retina relative to lentiviral delivery (Banskota et al. 2022). Given that the latest generation of RNP-packaging VLPs exhibit on-target editing efficiencies comparable to those achieved by mRNA-packaging VLPs or LNPs, we anticipate that RNP-packaging VLPs will be the preferred delivery vehicles for many applications due to the fact that they offer the shortest exposure to gene editing agents and therefore the lowest potential for off-target editing.

We and others have demonstrated that the cell-type specificity of VLPs in vitro can be altered by using different envelope glycoproteins (Banskota et al. 2022; Hamilton et al. 2021; Mangeot et al. 2019). To further improve the broad therapeutic applicability of VLP delivery, it will be important to demonstrate delivery to additional organs, which could be achieved in part by using different envelope glycoproteins or other targeting moieties. The ability to do so would realize the full potential of VLPs as a delivery modality that combines the programmable tropism of viruses with the transient delivery of mRNAs and RNPs.

While we showed that systemic administration of eVLPs was non-toxic in mice, future studies should further characterize the safety profile of VLPs in vivo. Because all of the VLPs reviewed above are derived from viral scaffolds, the immunogenicity of VLPs should also be evaluated. Recently, Zhang and coworkers reported that the mammalian retrovirus-like protein PEG10 can be programmed to package desired mRNA cargos, including Cas9 nuclease (Segel et al. 2021). With in vivo validation and further development to improve delivery efficiency, the PEG10-based “SEND” platform could potentially offer minimized immunogenicity relative to retroviral VLPs, as it uses an endogenous mammalian protein scaffold.

Finally, it will be critical to establish the feasibility of scaling up VLP production to quantities required for pre-clinical studies in large animal models and beyond. If successful, such studies could pave the way for the use of VLPs in the clinic as delivery vehicles for gene editing agents that offer several of the most important features of both viral and non-viral delivery technologies.

## Future prospects and conclusions

As shown by the examples summarized above, the era of therapeutic in vivo gene editing in humans has already arrived. Extensive development and optimization of CRISPR-Cas technologies have yielded robust tools for gene editing, including programmable nucleases, base editors, and prime editors. Pairing these gene editing tools with efficient in vivo delivery methods, including viral vectors, LNPs, and VLPs, has led to numerous demonstrations of in vivo gene editing, from proof-of-concept applications in animal models to therapeutic outcomes in humans.

With current in vivo delivery modalities, gene editing agents can be readily delivered to cells in the liver via intravenous injection and to cells in the eye via intraocular injection. For this reason, in vivo gene editing therapies in the near future will likely treat diseases that can be addressed through editing the liver or eye. Efficient delivery to non-liver tissues following intravenous administration remains a major challenge for most delivery vehicles. The use of naturally occurring and newly engineered AAV capsids is a promising strategy for targeting non-liver tissues, including the CNS (Goertsen et al. 2022), skeletal muscle (Tabebordbar et al. 2021), and heart (Koblan et al. 2021b). Analogous strategies could prove useful for retargeting VLPs to target new cell types by using different envelope glycoproteins. While systematic rules for reformulating LNPs to target different cell populations are not well understood, emerging methods for conjugating targeting moieties to the surface of LNPs could prove especially useful (Paunovska et al. 2022). For every distinct therapeutic

application of in vivo gene editing, it will also be important to understand whether tissuespecific targeting and editing is required, or if targeting and editing a desired tissue in addition to the liver is acceptable. Cell type-specific delivery within a particular tissue could offer advantages for certain therapeutic applications (Kwon et al. 2020; Nance et al. 2019; Tabebordbar et al. 2016). Importantly, in vivo gene editing strategies should only target somatic cell populations and must always avoid collateral germline editing, as clinical human germline editing raises serious ethical concerns (Lander et al. 2019; Saha et al. 2021).

Immunogenicity concerns associated with the in vivo delivery of gene editing agents are complex and remain to be characterized comprehensively. Preexisting immunity to delivery vehicles could interfere with in vivo gene editing therapies, as preexisting antibodies could directly neutralize viral vectors (Verdera et al. 2020; Weber, 2021). Preexisting cellular immunity to Cas9 or other components of gene editing agents could lead to immune-mediated clearance of transduced and edited cells (Crudele and Chamberlain, 2018; Wagner et al. 2021). Over time, prolonged expression of gene editing agents in edited cells could provoke adaptive immune responses (Wagner et al. 2021), which could also lead to clearance of transduced and edited cells. While a single transient administration of a gene editing agent in vivo in the absence of any preexisting immunity has been shown to be effective (Finn et al. 2018; Gillmore et al. 2021; Musunuru et al. 2021), such an administration could in some cases trigger an adaptive immune response that would limit the efficacy of repeat dosing or the use of related editing agents in the future (Rothgangl et al. 2021). These concerns highlight the advantages of delivery methods that can support a one-time, transient, and potent delivery of a therapeutic gene editing agent in vivo, such as LNPs and VLPs.

An additional advantage of transiently delivering gene editing agents is that transient delivery leads to reduced off-target editing compared to prolonged delivery (Anzalone et al. 2020; Banskota et al. 2022; Doman et al. 2020; Newby and Liu, 2021). Minimizing off-target gene editing in vivo is especially important, as even highly rare off-target editing events could install cancer-causing mutations. While viral delivery generally leads to prolonged expression, methods to turn off the expression of the gene editing agent after on-target editing is complete could be useful for reducing off-target editing. mRNA delivery (e.g. by LNPs) is transient and offers favorable on-target vs. off-target editing profiles, but RNP delivery minimizes the potential for off-target editing as it offers the shortest exposure to gene editing agents (Banskota et al. 2022; Newby et al. 2021). However, current methods for efficiently delivering RNPs to multiple organs in vivo are limited to eVLPs. As RNP delivery vehicles are the most attractive from a safety perspective, the development of improved RNP delivery vehicles for therapeutic in vivo gene editing will likely be highly impactful.

As additional in vivo gene editing therapies move rapidly toward the clinic, the availability of robust in vivo delivery methods will be critical. Future advances in delivery technologies will help to enable a wide range of in vivo gene editing therapies and potentially other macromolecular therapeutic approaches.

This work was supported by NIH UG3AI150551, U01AI142756, R35GM118062, RM1HG009490, HHMI, and the Bill and Melinda Gates Foundation. A.R. gratefully acknowledges support from an NSF graduate research fellowship. We thank Dr. Anahita Vieira for assistance editing the manuscript. Some images in Figures 2, 3, and 4 were created with BioRender.com.

## References

Adams D, Gonzalez-Duarte A, O'Riordan WD, Yang CC, Ueda M, Kristen AV, Tournev I, Schmidt HH, Coelho T, Berk JL, et al. (2018). Patisiran, an RNAi Therapeutic, for Hereditary Transthyretin Amyloidosis. N Engl J Med 379, 11-21. [PubMed: 29972753]
Akinc A, Maier MA, Manoharan M, Fitzgerald K, Jayaraman M, Barros S, Ansell S, Du X, Hope MJ, Madden TD, et al. (2019). The Onpattro story and the clinical translation of nanomedicines containing nucleic acid-based drugs. Nat Nanotechnol 14, 1084-1087. [PubMed: 31802031]
Alanis-Lobato G, Zohren J, McCarthy A, Fogarty NME, Kubikova N, Hardman E, Greco M, Wells D, Turner JMA, and Niakan KK (2021). Frequent loss of heterozygosity in CRISPR-Cas9-edited early human embryos. Proc Natl Acad Sci U S A 118.
Allen D, Rosenberg M, and Hendel A (2020). Using Synthetically Engineered Guide RNAs to Enhance CRISPR Genome Editing Systems in Mammalian Cells. Front Genome Ed 2, 617910. [PubMed: 34713240]
Anzalone AV, Koblan LW, and Liu DR (2020). Genome editing with CRISPR-Cas nucleases, base editors, transposases and prime editors. Nat Biotechnol 38, 824-844. [PubMed: 32572269]
Anzalone AV, Randolph PB, Davis JR, Sousa AA, Koblan LW, Levy JM, Chen PJ, Wilson C, Newby GA, Raguram A, et al. (2019). Search-and-replace genome editing without double-strand breaks or donor DNA. Nature 576, 149-157. [PubMed: 31634902]
Aranko AS, Wlodawer A, and Iwai H (2014). Nature's recipe for splitting inteins. Protein Eng Des Sel 27, 263-271. [PubMed: 25096198]
Asokan A, Conway JC, Phillips JL, Li C, Hegge J, Sinnott R, Yadav S, DiPrimio N, Nam HJ, Agbandje-McKenna M, et al. (2010). Reengineering a receptor footprint of adeno-associated virus enables selective and systemic gene transfer to muscle. Nat Biotechnol 28, 79-82. [PubMed: 20037580]
Baden LR, El Sahly HM, Essink B, Kotloff K, Frey S, Novak R, Diemert D, Spector SA, Rouphael N, Creech CB, et al. (2021). Efficacy and Safety of the mRNA-1273 SARS-CoV-2 Vaccine. N Engl J Med 384, 403-416. [PubMed: 33378609]
Banskota S, Raguram A, Suh S, Du SW, Davis JR, Choi EH, Wang X, Nielsen SC, Newby GA, Randolph PB, et al. (2022). Engineered virus-like particles for efficient in vivo delivery of therapeutic proteins. Cell 185, 250-265 e216. [PubMed: 35021064]
Bareford LM, and Swaan PW (2007). Endocytic mechanisms for targeted drug delivery. Adv Drug Deliv Rev 59, 748-758. [PubMed: 17659804]
Böck D, Rothgangl T, Villiger L, Schmidheini L, Matsushita M, Mathis N, Ioannidi E, Rimann N, Grisch-Chan HM, Kreutzer S, et al. (2022). In vivo prime editing of a metabolic liver disease in mice. Sci Transl Med 14, eabl9238. [PubMed: 35294257]
Bottros MM, and Christo PJ (2014). Current perspectives on intrathecal drug delivery. J Pain Res 7, 615-626. [PubMed: 25395870]
Byrne LC, Day TP, Visel M, Strazzeri JA, Fortuny C, Dalkara D, Merigan WH, Schaffer DV, and Flannery JG (2020). In vivo-directed evolution of adeno-associated virus in the primate retina. JCI Insight 5.
Cai Y, Bak RO, and Mikkelsen JG (2014). Targeted genome editing by lentiviral protein transduction of zinc-finger and TAL-effector nucleases. Elife 3, e01911. [PubMed: 24843011]
Chadwick AC, Wang X, and Musunuru K (2017). In Vivo Base Editing of PCSK9 (Proprotein Convertase Subtilisin/Kexin Type 9) as a Therapeutic Alternative to Genome Editing. Arterioscler Thromb Vasc Biol 37, 1741-1747. [PubMed: 28751571]

Chandler RJ, Sands MS, and Venditti CP (2017). Recombinant Adeno-Associated Viral Integration and Genotoxicity: Insights from Animal Models. Hum Gene Ther 28, 314-322. [PubMed: 28293963]

Chapman JR, Taylor MR, and Boulton SJ (2012). Playing the end game: DNA double-strand break repair pathway choice. Mol Cell 47, 497-510. [PubMed: 22920291]

Charlesworth CT, Deshpande PS, Dever DP, Camarena J, Lemgart VT, Cromer MK, Vakulskas CA, Collingwood MA, Zhang L, Bode NM, et al. (2019). Identification of preexisting adaptive immunity to Cas9 proteins in humans. Nat Med 25, 249-254. [PubMed: 30692695]

Chemello F, Chai AC, Li H, Rodriguez-Caycedo C, Sanchez-Ortiz E, Atmanli A, Mireault AA, Liu N, Bassel-Duby R, and Olson EN (2021). Precise correction of Duchenne muscular dystrophy exon deletion mutations by base and prime editing. Sci Adv 7.

Chen L, Park JE, Paa P, Rajakumar PD, Prekop HT, Chew YT, Manivannan SN, and Chew WL (2021a). Programmable C:G to G:C genome editing with CRISPR-Cas9-directed base excision repair proteins. Nat Commun 12, 1384. [PubMed: 33654077]

Chen PJ, Hussmann JA, Yan J, Knipping F, Ravisankar P, Chen PF, Chen C, Nelson JW, Newby GA, Sahin M, et al. (2021b). Enhanced prime editing systems by manipulating cellular determinants of editing outcomes. Cell 184, 5635-5652 e5629. [PubMed: 34653350]

Chen Y, Zhi S, Liu W, Wen J, Hu S, Cao T, Sun H, Li Y, Huang L, Liu Y, et al. (2020). Development of Highly Efficient Dual-AAV Split Adenosine Base Editor for In Vivo Gene Therapy. Small Methods 4.

Cheng Q, Wei T, Farbiak L, Johnson LT, Dilliard SA, and Siegwart DJ (2020). Selective organ targeting (SORT) nanoparticles for tissue-specific mRNA delivery and CRISPR-Cas gene editing. Nat Nanotechnol 15, 313-320. [PubMed: 32251383]

Chew WL, Tabebordbar M, Cheng JK, Mali P, Wu EY, Ng AH, Zhu K, Wagers AJ, and Church GM (2016). A multifunctional AAV-CRISPR-Cas9 and its host response. Nat Methods 13, 868-874. [PubMed: 27595405]

Choi JG, Dang Y, Abraham S, Ma H, Zhang J, Guo H, Cai Y, Mikkelsen JG, Wu H, Shankar P, et al. (2016). Lentivirus pre-packed with Cas9 protein for safer gene editing. Gene Ther 23, 627-633. [PubMed: 27052803]

Chu D, Sullivan CC, Weitzman MD, Du L, Wolf PL, Jamieson SW, and Thistlethwaite PA (2003). Direct comparison of efficiency and stability of gene transfer into the mammalian heart using adeno-associated virus versus adenovirus vectors. J Thorac Cardiovasc Surg 126, 671-679. [PubMed: 14502138]

Cox DB, Platt RJ, and Zhang F (2015). Therapeutic genome editing: prospects and challenges. Nat Med 21, 121-131. [PubMed: 25654603]

Cronin J, Zhang XY, and Reiser J (2005). Altering the tropism of lentiviral vectors through pseudotyping. Curr Gene Ther 5, 387-398. [PubMed: 16101513]

Crudele JM, and Chamberlain JS (2018). Cas9 immunity creates challenges for CRISPR gene editing therapies. Nat Commun 9, 3497. [PubMed: 30158648]

Cullis PR, and Hope MJ (2017). Lipid Nanoparticle Systems for Enabling Gene Therapies. Mol Ther 25, 1467-1475. [PubMed: 28412170]

Dahlman JE, Kauffman KJ, Xing Y, Shaw TE, Mir FF, Dlott CC, Langer R, Anderson DG, and Wang ET (2017). Barcoded nanoparticles for high throughput in vivo discovery of targeted therapeutics. Proc Natl Acad Sci U S A 114, 2060-2065. [PubMed: 28167778]

Dalkara D, Byrne LC, Klimczak RR, Visel M, Yin L, Merigan WH, Flannery JG, and Schaffer DV (2013). In vivo-directed evolution of a new adeno-associated virus for therapeutic outer retinal gene delivery from the vitreous. Sci Transl Med 5, 189ra176.

Dalsgaard T, Cecchi CR, Askou AL, Bak RO, Andersen PO, Hougaard D, Jensen TG, DagnaesHansen F, Mikkelsen JG, Corydon TJ, et al. (2018). Improved Lentiviral Gene Delivery to Mouse Liver by Hydrodynamic Vector Injection through Tail Vein. Mol Ther Nucleic Acids 12, 672-683. [PubMed: 30092403]

Daneman R, and Prat A (2015). The blood-brain barrier. Cold Spring Harb Perspect Biol 7, a020412. [PubMed: 25561720]

Davis JR, Wang X, Witte I, Huang TP, Levy JM, Raguram A, Banskota S, Seidah N, Musunuru K, and Liu DR (2022). Efficient in vivo delivery of adenine base editors in a single adeno-associated virus. in revision.
Deverman BE, Pravdo PL, Simpson BP, Kumar SR, Chan KY, Banerjee A, Wu WL, Yang B, Huber N, Pasca SP, et al. (2016). Cre-dependent selection yields AAV variants for widespread gene transfer to the adult brain. Nat Biotechnol 34, 204-209. [PubMed: 26829320]
Deverman BE, Ravina BM, Bankiewicz KS, Paul SM, and Sah DWY (2018). Gene therapy for neurological disorders: progress and prospects. Nat Rev Drug Discov 17, 641-659. [PubMed: 30093643]
Dilliard SA, Cheng Q, and Siegwart DJ (2021). On the mechanism of tissue-specific mRNA delivery by selective organ targeting nanoparticles. Proc Natl Acad Sci U S A 118.
Doman JL, Raguram A, Newby GA, and Liu DR (2020). Evaluation and minimization of Cas9independent off-target DNA editing by cytosine base editors. Nat Biotechnol 38, 620-628. [PubMed: 32042165]
Dong JY, Fan PD, and Frizzell RA (1996). Quantitative analysis of the packaging capacity of recombinant adeno-associated virus. Hum Gene Ther 7, 2101-2112. [PubMed: 8934224]
Doudna JA (2020). The promise and challenge of therapeutic genome editing. Nature 578, 229-236. [PubMed: 32051598]
Drouin LM, and Agbandje-McKenna M (2013). Adeno-associated virus structural biology as a tool in vector development. Future Virol 8, 1183-1199. [PubMed: 24533032]
Duan D, Yue Y, and Engelhardt JF (2001). Expanding AAV packaging capacity with trans-splicing or overlapping vectors: a quantitative comparison. Mol Ther 4, 383-391. [PubMed: 11592843]
Dull T, Zufferey R, Kelly M, Mandel RJ, Nguyen M, Trono D, and Naldini L (1998). A thirdgeneration lentivirus vector with a conditional packaging system. J Virol 72, 8463-8471. [PubMed: 9765382]
Edraki A, Mir A, Ibraheim R, Gainetdinov I, Yoon Y, Song CQ, Cao Y, Gallant J, Xue W, Rivera-Perez JA, et al. (2019). A Compact, High-Accuracy Cas9 with a Dinucleotide PAM for In Vivo Genome Editing. Mol Cell 73, 714-726 e714. [PubMed: 30581144]
Ferrari S, Vavassori V, Canarutto D, Jacob A, Castiello MC, Javed AO, and Genovese P (2021). Gene Editing of Hematopoietic Stem Cells: Hopes and Hurdles Toward Clinical Translation. Front Genome Ed 3, 618378. [PubMed: 34713250]
Fine EJ, Appleton CM, White DE, Brown MT, Deshmukh H, Kemp ML, and Bao G (2015). Transspliced Cas9 allows cleavage of HBB and CCR5 genes in human cells using compact expression cassettes. Sci Rep 5, 10777. [PubMed: 26126518]
Finn JD, Smith AR, Patel MC, Shaw L, Youniss MR, van Heteren J, Dirstine T, Ciullo C, Lescarbeau R, Seitzer J, et al. (2018). A Single Administration of CRISPR/Cas9 Lipid Nanoparticles Achieves Robust and Persistent In Vivo Genome Editing. Cell Rep 22, 2227-2235. [PubMed: 29490262]
Gaj T, Ojala DS, Ekman FK, Byrne LC, Limsirichai P, and Schaffer DV (2017). In vivo genome editing improves motor function and extends survival in a mouse model of ALS. Sci Adv 3, eaar3952. [PubMed: 29279867]
Galla M, Will E, Kraunus J, Chen L, and Baum C (2004). Retroviral pseudotransduction for targeted cell manipulation. Mol Cell 16, 309-315. [PubMed: 15494317]
Gao X, Tao Y, Lamas V, Huang M, Yeh WH, Pan B, Hu YJ, Hu JH, Thompson DB, Shu Y, et al. (2018). Treatment of autosomal dominant hearing loss by in vivo delivery of genome editing agents. Nature 553, 217-221. [PubMed: 29258297]
Gaudelli NM, Komor AC, Rees HA, Packer MS, Badran AH, Bryson DI, and Liu DR (2017). Programmable base editing of $\mathrm{A}^{*} \mathrm{~T}$ to $\mathrm{G}^{*} \mathrm{C}$ in genomic DNA without DNA cleavage. Nature 551, 464-471. [PubMed: 29160308]
Gaudelli NM, Lam DK, Rees HA, Sola-Esteves NM, Barrera LA, Born DA, Edwards A, Gehrke JM, Lee SJ, Liquori AJ, et al. (2020). Directed evolution of adenine base editors with increased activity and therapeutic application. Nat Biotechnol 38, 892-900. [PubMed: 32284586]
Gee P, Lung MSY, Okuzaki Y, Sasakawa N, Iguchi T, Makita Y, Hozumi H, Miura Y, Yang LF, Iwasaki M, et al. (2020). Extracellular nanovesicles for packaging of CRISPR-Cas9 protein and sgRNA to induce therapeutic exon skipping. Nat Commun 11, 1334. [PubMed: 32170079]

Geutskens SB, van der Eb MM, Plomp AC, Jonges LE, Cramer SJ, Ensink NG, Kuppen PJ, and Hoeben RC (2000). Recombinant adenoviral vectors have adjuvant activity and stimulate T cell responses against tumor cells. Gene Ther 7, 1410-1416. [PubMed: 10981668]
Giannoukos G, Ciulla DM, Marco E, Abdulkerim HS, Barrera LA, Bothmer A, Dhanapal V, Gloskowski SW, Jayaram H, Maeder ML, et al. (2018). UDiTaS, a genome editing detection method for indels and genome rearrangements. BMC Genomics 19, 212. [PubMed: 29562890]
Gilleron J, Querbes W, Zeigerer A, Borodovsky A, Marsico G, Schubert U, Manygoats K, Seifert S, Andree C, Stoter M, et al. (2013). Image-based analysis of lipid nanoparticle-mediated siRNA delivery, intracellular trafficking and endosomal escape. Nat Biotechnol 31, 638-646. [PubMed: 23792630]
Gillmore JD, Gane E, Taubel J, Kao J, Fontana M, Maitland ML, Seitzer J, O'Connell D, Walsh KR, Wood K, et al. (2021). CRISPR-Cas9 In Vivo Gene Editing for Transthyretin Amyloidosis. N Engl J Med 385, 493-502. [PubMed: 34215024]
Ginn SL, Amaya AK, Alexander IE, Edelstein M, and Abedi MR (2018). Gene therapy clinical trials worldwide to 2017: An update. J Gene Med 20, e3015. [PubMed: 29575374]
Goertsen D, Flytzanis NC, Goeden N, Chuapoco MR, Cummins A, Chen Y, Fan Y, Zhang Q, Sharma J, Duan Y, et al. (2022). AAV capsid variants with brain-wide transgene expression and decreased liver targeting after intravenous delivery in mouse and marmoset. Nat Neurosci 25, 106-115. [PubMed: 34887588]
Gorman GS, Chinnery PF, DiMauro S, Hirano M, Koga Y, McFarland R, Suomalainen A, Thorburn DR, Zeviani M, and Turnbull DM (2016). Mitochondrial diseases. Nat Rev Dis Primers 2, 16080. [PubMed: 27775730]
Guo J, Chen X, Liu Z, Sun H, Zhou Y, Dai Y, Ma Y, He L, Qian X, Wang J, et al. (2022). DdCBE mediates efficient and inheritable modifications in mouse mitochondrial genome. Mol Ther Nucleic Acids 27, 73-80. [PubMed: 34938607]
Hamilton JR, Tsuchida CA, Nguyen DN, Shy BR, McGarrigle ER, Sandoval Espinoza CR, Carr D, Blaeschke F, Marson A, and Doudna JA (2021). Targeted delivery of CRISPR-Cas9 and transgenes enables complex immune cell engineering. Cell Rep 35, 109207. [PubMed: 34077734]
Hoffmann MD, Aschenbrenner S, Grosse S, Rapti K, Domenger C, Fakhiri J, Mastel M, Borner K, Eils R, Grimm D, et al. (2019). Cell-specific CRISPR-Cas9 activation by microRNA-dependent expression of anti-CRISPR proteins. Nucleic Acids Res 47, e75. [PubMed: 30982889]
Hoshyar N, Gray S, Han H, and Bao G (2016). The effect of nanoparticle size on in vivo pharmacokinetics and cellular interaction. Nanomedicine (Lond) 11, 673-692. [PubMed: 27003448]
Hu Z, Wang S, Zhang C, Gao N, Li M, Wang D, Wang D, Liu D, Liu H, Ong SG, et al. (2020). A compact Cas9 ortholog from Staphylococcus Auricularis (SauriCas9) expands the dNa targeting scope. PLoS Biol 18, e3000686. [PubMed: 32226015]
Ibraheim R, Tai PWL, Mir A, Javeed N, Wang J, Rodriguez TC, Namkung S, Nelson S, Khokhar ES, Mintzer E, et al. (2021). Self-inactivating, all-in-one AAV vectors for precision Cas9 genome editing via homology-directed repair in vivo. Nat Commun 12, 6267. [PubMed: 34725353]
Indikova I, and Indik S (2020). Highly efficient 'hit-and-run' genome editing with unconcentrated lentivectors carrying Vpr.Prot.Cas9 protein produced from RRE-containing transcripts. Nucleic Acids Res 48, 8178-8187. [PubMed: 32619241]
Jang HK, Jo DH, Lee SN, Cho CS, Jeong YK, Jung Y, Yu J, Kim JH, Woo JS, and Bae S (2021). High-purity production and precise editing of DNA base editing ribonucleoproteins. Sci Adv 7.
Jiang C, Mei M, Li B, Zhu X, Zu W, Tian Y, Wang Q, Guo Y, Dong Y, and Tan X (2017). A non-viral CRISPR/Cas9 delivery system for therapeutically targeting HBV DNA and pcsk9 in vivo. Cell Res 27, 440-443. [PubMed: 28117345]
Jiang T, Henderson JM, Coote K, Cheng Y, Valley HC, Zhang XO, Wang Q, Rhym LH, Cao Y, Newby GA, et al. (2020). Chemical modifications of adenine base editor mRNA and guide RNA expand its application scope. Nat Commun 11, 1979. [PubMed: 32332735]
Jinek M, Chylinski K, Fonfara I, Hauer M, Doudna JA, and Charpentier E (2012). A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity. Science 337, 816-821. [PubMed: 22745249]

Jo DH, Jang H-K, Cho CS, Han JH, Ryu G, Jung Y, Bae S, and Kim JH (2021). Therapeutic adenine base editing corrects nonsense mutation and improves visual function in a mouse model of Leber congenital amaurosis. bioRxiv.
Joglekar AV, and Sandoval S (2017). Pseudotyped Lentiviral Vectors: One Vector, Many Guises. Hum Gene Ther Methods 28, 291-301. [PubMed: 28870117]
Kabadi AM, Ousterout DG, Hilton IB, and Gersbach CA (2014). Multiplex CRISPR/Cas9-based genome engineering from a single lentiviral vector. Nucleic Acids Res 42, e147. [PubMed: 25122746]
Kaczmarczyk SJ, Sitaraman K, Young HA, Hughes SH, and Chatterjee DK (2011). Protein delivery using engineered virus-like particles. Proc Natl Acad Sci U S A 108, 16998-17003. [PubMed: 21949376]
Kariko K, Muramatsu H, Welsh FA, Ludwig J, Kato H, Akira S, and Weissman D (2008). Incorporation of pseudouridine into mRNA yields superior nonimmunogenic vector with increased translational capacity and biological stability. Mol Ther 16, 1833-1840. [PubMed: 18797453]
Kauffman KJ, Dorkin JR, Yang JH, Heartlein MW, DeRosa F, Mir FF, Fenton OS, and Anderson DG (2015). Optimization of Lipid Nanoparticle Formulations for mRNA Delivery in Vivo with Fractional Factorial and Definitive Screening Designs. Nano Lett 15, 7300-7306. [PubMed: 26469188]
Kazmierczak Z, Szostak-Paluch K, Przybylo M, Langner M, Witkiewicz W, Jedruchniewicz N, and Dabrowska K (2020). Endocytosis in cellular uptake of drug delivery vectors: Molecular aspects in drug development. Bioorg Med Chem 28, 115556. [PubMed: 32828419]
Kedmi R, Veiga N, Ramishetti S, Goldsmith M, Rosenblum D, Dammes N, Hazan-Halevy I, Nahary L, Leviatan-Ben-Arye S, Harlev M, et al. (2018). A modular platform for targeted RNAi therapeutics. Nat Nanotechnol 13, 214-219. [PubMed: 29379205]
Kenjo E, Hozumi H, Makita Y, Iwabuchi KA, Fujimoto N, Matsumoto S, Kimura M, Amano Y, Ifuku M, Naoe Y, et al. (2021). Low immunogenicity of LNP allows repeated administrations of CRISPR-Cas9 mRNA into skeletal muscle in mice. Nat Commun 12, 7101. [PubMed: 34880218]
Kim E, Koo T, Park SW, Kim D, Kim K, Cho HY, Song DW, Lee KJ, Jung MH, Kim S, et al. (2017). In vivo genome editing with a small Cas9 orthologue derived from Campylobacter jejuni. Nat Commun 8, 14500. [PubMed: 28220790]
Kleinstiver BP, Prew MS, Tsai SQ, Nguyen NT, Topkar VV, Zheng Z, and Joung JK (2015). Broadening the targeting range of Staphylococcus aureus CRISPR-Cas9 by modifying PAM recognition. Nat Biotechnol 33, 1293-1298. [PubMed: 26524662]
Knopp Y, Geis FK, Heckl D, Horn S, Neumann T, Kuehle J, Meyer J, Fehse B, Baum C, Morgan M, et al. (2018). Transient Retrovirus-Based CRISPR/Cas9 All-in-One Particles for Efficient, Targeted Gene Knockout. Mol Ther Nucleic Acids 13, 256-274. [PubMed: 30317165]
Koblan LW, Arbab M, Shen MW, Hussmann JA, Anzalone AV, Doman JL, Newby GA, Yang D, Mok B, Replogle JM, et al. (2021a). Efficient C*G-to-G*C base editors developed using CRISPRi screens, target-library analysis, and machine learning. Nat Biotechnol 39, 1414-1425. [PubMed: 34183861]
Koblan LW, Erdos MR, Wilson C, Cabral WA, Levy JM, Xiong ZM, Tavarez UL, Davison LM, Gete YG, Mao X, et al. (2021b). In vivo base editing rescues Hutchinson-Gilford progeria syndrome in mice. Nature 589, 608-614. [PubMed: 33408413]
Komor AC, Kim YB, Packer MS, Zuris JA, and Liu DR (2016). Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature 533, 420-424. [PubMed: 27096365]
Komor AC, Zhao KT, Packer MS, Gaudelli NM, Waterbury AL, Koblan LW, Kim YB, Badran AH, and Liu DR (2017). Improved base excision repair inhibition and bacteriophage Mu Gam protein yields C:G-to-T:A base editors with higher efficiency and product purity. Sci Adv 3, eaao4774. [PubMed: 28875174]
Korf BR, Pyeritz RE, and Grody WW (2019). Nature and Frequency of Genetic Disease. 47-51.
Kosicki M, Tomberg K, and Bradley A (2018). Repair of double-strand breaks induced by CRISPRCas9 leads to large deletions and complex rearrangements. Nat Biotechnol 36, 765-771. [PubMed: 30010673]

Kumar M, Keller B, Makalou N, and Sutton RE (2001). Systematic determination of the packaging limit of lentiviral vectors. Hum Gene Ther 12, 1893-1905. [PubMed: 11589831]
Kurt IC, Zhou R, Iyer S, Garcia SP, Miller BR, Langner LM, Grunewald J, and Joung JK (2021). CRISPR C-to-G base editors for inducing targeted DNA transversions in human cells. Nat Biotechnol 39, 41-46. [PubMed: 32690971]
Kuzmin DA, Shutova MV, Johnston NR, Smith OP, Fedorin VV, Kukushkin YS, van der Loo JCM, and Johnstone EC (2021). The clinical landscape for AAV gene therapies. Nat Rev Drug Discov 20, 173-174. [PubMed: 33495615]
Kwon JB, Ettyreddy AR, Vankara A, Bohning JD, Devlin G, Hauschka SD, Asokan A, and Gersbach CA (2020). In Vivo Gene Editing of Muscle Stem Cells with Adeno-Associated Viral Vectors in a Mouse Model of Duchenne Muscular Dystrophy. Mol Ther Methods Clin Dev 19, 320-329. [PubMed: 33145368]
Kymalainen H, Appelt JU, Giordano FA, Davies AF, Ogilvie CM, Ahmed SG, Laufs S, Schmidt M, Bode J, Yanez-Munoz RJ, et al. (2014). Long-term episomal transgene expression from mitotically stable integration-deficient lentiviral vectors. Hum Gene Ther 25, 428-442. [PubMed: 24483952]
Lander ES, Baylis F, Zhang F, Charpentier E, Berg P, Bourgain C, Friedrich B, Joung JK, Li J, Liu D, et al. (2019). Adopt a moratorium on heritable genome editing. Nature 567, 165-168. [PubMed: 30867611]
Lau CH , and Suh Y (2017). In vivo genome editing in animals using AAV-CRISPR system: applications to translational research of human disease. F1000Res 6, 2153. [PubMed: 29333255]
Lee CS, Bishop ES, Zhang R, Yu X, Farina EM, Yan S, Zhao C, Zheng Z, Shu Y, Wu X, et al. (2017). Adenovirus-Mediated Gene Delivery: Potential Applications for Gene and Cell-Based Therapies in the New Era of Personalized Medicine. Genes Dis 4, 43-63. [PubMed: 28944281]
Lee H, Lee S, Baek G, Kim A, Kang BC, Seo H, and Kim JS (2021). Mitochondrial DNA editing in mice with DddA-TALE fusion deaminases. Nat Commun 12, 1190. [PubMed: 33608520]
Lee J, Mou H, Ibraheim R, Liang SQ, Liu P, Xue W, and Sontheimer EJ (2019). Tissue-restricted genome editing in vivo specified by microRNA-repressible anti-CRISPR proteins. RNA 25, 1421-1431. [PubMed: 31439808]
Leibowitz ML, Papathanasiou S, Doerfler PA, Blaine LJ, Sun L, Yao Y, Zhang CZ, Weiss MJ, and Pellman D (2021). Chromothripsis as an on-target consequence of CRISPR-Cas9 genome editing. Nat Genet 53, 895-905. [PubMed: 33846636]
Levy JM, Yeh WH, Pendse N, Davis JR, Hennessey E, Butcher R, Koblan LW, Comander J, Liu Q, and Liu DR (2020). Cytosine and adenine base editing of the brain, liver, retina, heart and skeletal muscle of mice via adeno-associated viruses. Nat Biomed Eng 4, 97-110. [PubMed: 31937940]
Li A, Lee CM, Hurley AE, Jarrett KE, De Giorgi M, Lu W, Balderrama KS, Doerfler AM, Deshmukh H, Ray A, et al. (2019a). A Self-Deleting AAV-CRISPR System for In Vivo Genome Editing. Mol Ther Methods Clin Dev 12, 111-122. [PubMed: 30619914]
Li C, Georgakopoulou A, Mishra A, Gil S, Hawkins RD, Yannaki E, and Lieber A (2021). In vivo HSPC gene therapy with base editors allows for efficient reactivation of fetal gamma-globin in beta-YAC mice. Blood Adv 5, 1122-1135. [PubMed: 33620431]
Li F, Hung SSC, Mohd Khalid MKN, Wang JH, Chrysostomou V, Wong VHY, Singh V, Wing K, Tu L, Bender JA, et al. (2019b). Utility of Self-Destructing CRISPR/Cas Constructs for Targeted Gene Editing in the Retina. Hum Gene Ther 30, 1349-1360. [PubMed: 31373227]
Li H, Yang Y, Hong W, Huang M, Wu M, and Zhao X (2020a). Applications of genome editing technology in the targeted therapy of human diseases: mechanisms, advances and prospects. Signal Transduct Target Ther 5, 1. [PubMed: 32296011]
Li J, Sun W, Wang B, Xiao X, and Liu XQ (2008a). Protein trans-splicing as a means for viral vector-mediated in vivo gene therapy. Hum Gene Ther 19, 958-964. [PubMed: 18788906]
Li W, Asokan A, Wu Z, Van Dyke T, DiPrimio N, Johnson JS, Govindaswamy L, Agbandje-McKenna M, Leichtle S, Redmond DE Jr., et al. (2008b). Engineering and selection of shuffled AAV genomes: a new strategy for producing targeted biological nanoparticles. Mol Ther 16, 12521260 .

Li X, Qian X, Wang B, Xia Y, Zheng Y, Du L, Xu D, Xing D, DePinho RA, and Lu Z (2020b). Programmable base editing of mutated TERT promoter inhibits brain tumour growth. Nat Cell Biol 22, 282-288. [PubMed: 32066906]
Lieber MR (2010). The Mechanism of Double-Strand DNA Break Repair by the Nonhomologous DNA End-Joining Pathway. Annual Review of Biochemistry 79, 181-211.
Lim CKW, Gapinske M, Brooks AK, Woods WS, Powell JE, Zeballos CM, Winter J, Perez-Pinera P, and Gaj T (2020). Treatment of a Mouse Model of ALS by In Vivo Base Editing. Mol Ther 28, 1177-1189. [PubMed: 31991108]
Lim K, Cho SI, and Kim JS (2022). Nuclear and mitochondrial DNA editing in human cells with zinc finger deaminases. Nat Commun 13, 366. [PubMed: 35042880]
Ling S, Yang S, Hu X, Yin D, Dai Y, Qian X, Wang D, Pan X, Hong J, Sun X, et al. (2021). Lentiviral delivery of co-packaged Cas9 mRNA and a Vegfa-targeting guide RNA prevents wet age-related macular degeneration in mice. Nat Biomed Eng 5, 144-156. [PubMed: 33398131]
Liu J, Chang J, Jiang Y, Meng X, Sun T, Mao L, Xu Q, and Wang M (2019). Fast and Efficient CRISPR/Cas9 Genome Editing In Vivo Enabled by Bioreducible Lipid and Messenger RNA Nanoparticles. Adv Mater 31, e1902575. [PubMed: 31215123]
Liu P, Liang SQ, Zheng C, Mintzer E, Zhao YG, Ponnienselvan K, Mir A, Sontheimer EJ, Gao G, Flotte TR, et al. (2021a). Improved prime editors enable pathogenic allele correction and cancer modelling in adult mice. Nat Commun 12, 2121. [PubMed: 33837189]
Liu Z, Chen S, Jia Y, Shan H, Chen M, Song Y, Lai L, and Li Z (2021b). Efficient and high-fidelity base editor with expanded PAM compatibility for cytidine dinucleotide. Sci China Life Sci 64, 1355-1367. [PubMed: 33420918]
Lombardo A, Genovese P, Beausejour CM, Colleoni S, Lee YL, Kim KA, Ando D, Urnov FD, Galli C, Gregory PD, et al. (2007). Gene editing in human stem cells using zinc finger nucleases and integrase-defective lentiviral vector delivery. Nat Biotechnol 25, 1298-1306. [PubMed: 17965707]
Lu B, Javidi-Parsijani P, Makani V, Mehraein-Ghomi F, Sarhan WM, Sun D, Yoo KW, Atala ZP, Lyu P, and Atala A (2019). Delivering SaCas9 mRNA by lentivirus-like bionanoparticles for transient expression and efficient genome editing. Nucleic Acids Res 47, e44. [PubMed: 30759231]
Lu Z, Yao X, Lyu P, Yadav M, Yoo K, Atala A, and Lu B (2021). Lentiviral Capsid-Mediated Streptococcus pyogenes Cas9 Ribonucleoprotein Delivery for Efficient and Safe Multiplex Genome Editing. CRISPR J 4, 914-928. [PubMed: 33733873]
Lyu P, Javidi-Parsijani P, Atala A, and Lu B (2019). Delivering Cas9/sgRNA ribonucleoprotein (RNP) by lentiviral capsid-based bionanoparticles for efficient 'hit-and-run' genome editing. Nucleic Acids Res 47, e99. [PubMed: 31299082]
Lyu P, Lu Z, Cho SI, Yadav M, Yoo KW, Atala A, Kim JS, and Lu B (2021). Adenine Base Editor Ribonucleoproteins Delivered by Lentivirus-Like Particles Show High On-Target Base Editing and Undetectable RNA Off-Target Activities. CRISPR J 4, 69-81. [PubMed: 33616436]
Lyu P, Wang L, and Lu B (2020). Virus-Like Particle Mediated CRISPR/Cas9 Delivery for Efficient and Safe Genome Editing. Life (Basel) 10.
Maeder ML, Stefanidakis M, Wilson CJ, Baral R, Barrera LA, Bounoutas GS, Bumcrot D, Chao H, Ciulla DM, DaSilva JA, et al. (2019). Development of a gene-editing approach to restore vision loss in Leber congenital amaurosis type 10. Nat Med 25, 229-233. [PubMed: 30664785]
Maguire AM, Simonelli F, Pierce EA, Pugh EN Jr., Mingozzi F, Bennicelli J, Banfi S, Marshall KA, Testa F, Surace EM, et al. (2008). Safety and efficacy of gene transfer for Leber's congenital amaurosis. N Engl J Med 358, 2240-2248. [PubMed: 18441370]
Maheshri N, Koerber JT, Kaspar BK, and Schaffer DV (2006). Directed evolution of adeno-associated virus yields enhanced gene delivery vectors. Nat Biotechnol 24, 198-204. [PubMed: 16429148]
Maier MA, Jayaraman M, Matsuda S, Liu J, Barros S, Querbes W, Tam YK, Ansell SM, Kumar V, Qin J, et al. (2013). Biodegradable lipids enabling rapidly eliminated lipid nanoparticles for systemic delivery of RNAi therapeutics. Mol Ther 21, 1570-1578. [PubMed: 23799535]
Mangeot PE, Risson V, Fusil F, Marnef A, Laurent E, Blin J, Mournetas V, Massourides E, Sohier TJM, Corbin A, et al. (2019). Genome editing in primary cells and in vivo using viral-derived

Nanoblades loaded with Cas9-sgRNA ribonucleoproteins. Nat Commun 10, 45. [PubMed: 30604748]
Mendell JR, Al-Zaidy S, Shell R, Arnold WD, Rodino-Klapac LR, Prior TW, Lowes L, Alfano L, Berry K, Church K, et al. (2017). Single-Dose Gene-Replacement Therapy for Spinal Muscular Atrophy. N Engl J Med 377, 1713-1722. [PubMed: 29091557]
Mendell JR, Al-Zaidy SA, Rodino-Klapac LR, Goodspeed K, Gray SJ, Kay CN, Boye SL, Boye SE, George LA, Salabarria S, et al. (2021). Current Clinical Applications of In Vivo Gene Therapy with AAVs. Mol Ther 29, 464-488. [PubMed: 33309881]
Miller JB, Zhang S, Kos P, Xiong H, Zhou K, Perelman SS, Zhu H, and Siegwart DJ (2017). Non-Viral CRISPR/Cas Gene Editing In Vitro and In Vivo Enabled by Synthetic Nanoparticle Co-Delivery of Cas9 mRNA and sgRNA. Angew Chem Int Ed Engl 56, 1059-1063. [PubMed: 27981708]
Milone MC, and O'Doherty U (2018). Clinical use of lentiviral vectors. Leukemia 32, 1529-1541. [PubMed: 29654266]
Mitchell MJ, Billingsley MM, Haley RM, Wechsler ME, Peppas NA, and Langer R (2021). Engineering precision nanoparticles for drug delivery. Nat Rev Drug Discov 20, 101-124. [PubMed: 33277608]
Mok BY, de Moraes MH, Zeng J, Bosch DE, Kotrys AV, Raguram A, Hsu F, Radey MC, Peterson SB, Mootha VK, et al. (2020). A bacterial cytidine deaminase toxin enables CRISPR-free mitochondrial base editing. Nature 583, 631-637. [PubMed: 32641830]
Monteys AM, Hundley AA, Ranum PT, Tecedor L, Muehlmatt A, Lim E, Lukashev D, Sivasankaran R, and Davidson BL (2021). Regulated control of gene therapies by drug-induced splicing. Nature 596, 291-295. [PubMed: 34321659]
Mullard A (2017). FDA approves first CAR T therapy. Nat Rev Drug Discov 16, 669.
Musunuru K, Chadwick AC, Mizoguchi T, Garcia SP, DeNizio JE, Reiss CW, Wang K, Iyer S, Dutta C, Clendaniel V, et al. (2021). In vivo CRISPR base editing of PCSK9 durably lowers cholesterol in primates. Nature 593, 429-434. [PubMed: 34012082]
Naldini L, Blomer U, Gallay P, Ory D, Mulligan R, Gage FH, Verma IM, and Trono D (1996). In vivo gene delivery and stable transduction of nondividing cells by a lentiviral vector. Science 272, 263-267. [PubMed: 8602510]
Nambiar TS, Baudrier L, Billon P, and Ciccia A (2022). CRISPR-based genome editing through the lens of DNA repair. Mol Cell 82, 348-388. [PubMed: 35063100]
Nance ME, Shi R, Hakim CH, Wasala NB, Yue Y, Pan X, Zhang T, Robinson CA, Duan SX, Yao G, et al. (2019). AAV9 Edits Muscle Stem Cells in Normal and Dystrophic Adult Mice. Mol Ther 27, 1568-1585. [PubMed: 31327755]
Naso MF, Tomkowicz B, Perry WL 3rd, and Strohl WR (2017). Adeno-Associated Virus (AAV) as a Vector for Gene Therapy. BioDrugs 31, 317-334. [PubMed: 28669112]
Nelson CE, and Gersbach CA (2016). Engineering Delivery Vehicles for Genome Editing. Annu Rev Chem Biomol Eng 7, 637-662. [PubMed: 27146557]
Nelson JW, Randolph PB, Shen SP, Everette KA, Chen PJ, Anzalone AV, An M, Newby GA, Chen JC, Hsu A, et al. (2021). Engineered pegRNAs improve prime editing efficiency. Nat Biotechnol.
Newby GA, and Liu DR (2021). In vivo somatic cell base editing and prime editing. Mol Ther 29, 3107-3124. [PubMed: 34509669]
Newby GA, Yen JS, Woodard KJ, Mayuranathan T, Lazzarotto CR, Li Y, Sheppard-Tillman H, Porter SN, Yao Y, Mayberry K, et al. (2021). Base editing of haematopoietic stem cells rescues sickle cell disease in mice. Nature 595, 295-302. [PubMed: 34079130]
Nishida K, Arazoe T, Yachie N, Banno S, Kakimoto M, Tabata M, Mochizuki M, Miyabe A, Araki M, Hara KY, et al. (2016). Targeted nucleotide editing using hybrid prokaryotic and vertebrate adaptive immune systems. Science 353.
Paunovska K, Loughrey D, and Dahlman JE (2022). Drug delivery systems for RNA therapeutics. Nat Rev Genet.
Peng Y, Tang L, and Zhou Y (2017). Subretinal Injection: A Review on the Novel Route of Therapeutic Delivery for Vitreoretinal Diseases. Ophthalmic Res 58, 217-226. [PubMed: 28858866]

Polack FP, Thomas SJ, Kitchin N, Absalon J, Gurtman A, Lockhart S, Perez JL, Perez Marc G, Moreira ED, Zerbini C, et al. (2020). Safety and Efficacy of the BNT162b2 mRNA Covid-19 Vaccine. N Engl J Med 383, 2603-2615. [PubMed: 33301246]
Prel A, Caval V, Gayon R, Ravassard P, Duthoit C, Payen E, Maouche-Chretien L, Creneguy A, Nguyen TH, Martin N, et al. (2015). Highly efficient in vitro and in vivo delivery of functional RNAs using new versatile MS2-chimeric retrovirus-like particles. Mol Ther Methods Clin Dev 2, 15039. [PubMed: 26528487]

Qiu M, Glass Z, Chen J, Haas M, Jin X, Zhao X, Rui X, Ye Z, Li Y, Zhang F, et al. (2021). Lipid nanoparticle-mediated codelivery of Cas9 mRNA and single-guide RNA achieves liver-specific in vivo genome editing of Angptl3. Proc Natl Acad Sci U S A 118.
Ran FA, Cong L, Yan WX, Scott DA, Gootenberg JS, Kriz AJ, Zetsche B, Shalem O, Wu X, Makarova KS, et al. (2015). In vivo genome editing using Staphylococcus aureus Cas9. Nature 520, 186191. [PubMed: 25830891]

Raper SE, Chirmule N, Lee FS, Wivel NA, Bagg A, Gao GP, Wilson JM, and Batshaw ML (2003). Fatal systemic inflammatory response syndrome in a ornithine transcarbamylase deficient patient following adenoviral gene transfer. Mol Genet Metab 80, 148-158. [PubMed: 14567964]
Rees HA, and Liu DR (2018). Base editing: precision chemistry on the genome and transcriptome of living cells. Nat Rev Genet 19, 770-788. [PubMed: 30323312]
Rees HA, Minella AC, Burnett CA, Komor AC, and Gaudelli NM (2021). CRISPR-derived genome editing therapies: Progress from bench to bedside. Mol Ther 29, 3125-3139. [PubMed: 34619370]
Richter M, Stone D, Miao C, Humbert O, Kiem HP, Papayannopoulou T, and Lieber A (2017). In Vivo Hematopoietic Stem Cell Transduction. Hematol Oncol Clin North Am 31, 771-785. [PubMed: 28895846]

Richter MF, Zhao KT, Eton E, Lapinaite A, Newby GA, Thuronyi BW, Wilson C, Koblan LW, Zeng J, Bauer DE, et al. (2020). Phage-assisted evolution of an adenine base editor with improved Cas domain compatibility and activity. Nat Biotechnol 38, 883-891. [PubMed: 32433547]
Rothgangl T, Dennis MK, Lin PJC, Oka R, Witzigmann D, Villiger L, Qi W, Hruzova M, Kissling L, Lenggenhager D, et al. (2021). In vivo adenine base editing of PCSK9 in macaques reduces LDL cholesterol levels. Nat Biotechnol 39, 949-957. [PubMed: 34012094]
Rurik JG, Tombacz I, Yadegari A, Mendez Fernandez PO, Shewale SV, Li L, Kimura T, Soliman OY, Papp TE, Tam YK, et al. (2022). CAR T cells produced in vivo to treat cardiac injury. Science 375, 91-96. [PubMed: 34990237]
Russell S, Bennett J, Wellman JA, Chung DC, Yu ZF, Tillman A, Wittes J, Pappas J, Elci O, McCague S, et al. (2017). Efficacy and safety of voretigene neparvovec (AAV2-hRPE65v2) in patients with RPE65-mediated inherited retinal dystrophy: a randomised, controlled, open-label, phase 3 trial. Lancet 390, 849-860. [PubMed: 28712537]
Ryu SM, Koo T, Kim K, Lim K, Baek G, Kim ST, Kim HS, Kim DE, Lee H, Chung E, et al. (2018). Adenine base editing in mouse embryos and an adult mouse model of Duchenne muscular dystrophy. Nat Biotechnol 36, 536-539. [PubMed: 29702637]
Sago CD, Lokugamage MP, Paunovska K, Vanover DA, Monaco CM, Shah NN, Gamboa Castro M, Anderson SE, Rudoltz TG, Lando GN, et al. (2018). High-throughput in vivo screen of functional mRNA delivery identifies nanoparticles for endothelial cell gene editing. Proc Natl Acad Sci U S A 115, E9944-E9952. [PubMed: 30275336]
Saha K, Sontheimer EJ, Brooks PJ, Dwinell MR, Gersbach CA, Liu DR, Murray SA, Tsai SQ, Wilson RC, Anderson DG, et al. (2021). The NIH Somatic Cell Genome Editing program. Nature 592, 195-204. [PubMed: 33828315]
San Filippo J, Sung P, and Klein H (2008). Mechanism of eukaryotic homologous recombination. Annu Rev Biochem 77, 229-257. [PubMed: 18275380]
Schoenmaker L, Witzigmann D, Kulkarni JA, Verbeke R, Kersten G, Jiskoot W, and Crommelin DJA (2021). mRNA-lipid nanoparticle COVID-19 vaccines: Structure and stability. Int J Pharm 601, 120586. [PubMed: 33839230]

Segel M, Lash B, Song J, Ladha A, Liu CC, Jin X, Mekhedov SL, Macrae RK, Koonin EV, and Zhang F (2021). Mammalian retrovirus-like protein PEG10 packages its own mRNA and can be pseudotyped for mRNA delivery. Science 373, 882-889. [PubMed: 34413232]
Seol JH, Shim EY, and Lee SE (2018). Microhomology-mediated end joining: Good, bad and ugly. Mutat Res 809, 81-87. [PubMed: 28754468]
Shams A, Higgins SA, Fellmann C, Laughlin TG, Oakes BL, Lew R, Kim S, Lukarska M, Arnold M, Staahl BT, et al. (2021). Comprehensive deletion landscape of CRISPR-Cas9 identifies minimal RNA-guided DNA-binding modules. Nat Commun 12, 5664. [PubMed: 34580310]
Shen S, Horowitz ED, Troupes AN, Brown SM, Pulicherla N, Samulski RJ, Agbandje-McKenna M, and Asokan A (2013). Engraftment of a galactose receptor footprint onto adeno-associated viral capsids improves transduction efficiency. J Biol Chem 288, 28814-28823. [PubMed: 23940044]
Sheridan C (2018). Go-ahead for first in-body CRISPR medicine testing. Nature Biotechnology.
Silva-Pinheiro P, Nash PA, Van Haute L, Mutti CD, Turner K, and Minczuk M (2022). In vivo mitochondrial base editing via adeno-associated viral delivery to mouse post-mitotic tissue. Nat Commun 13, 750. [PubMed: 35136065]
Smith SA, Selby LI, Johnston APR, and Such GK (2019). The Endosomal Escape of Nanoparticles: Toward More Efficient Cellular Delivery. Bioconjug Chem 30, 263-272. [PubMed: 30452233]
Song Y, Liu Z, Zhang Y, Chen M, Sui T, Lai L, and Li Z (2020). Large-Fragment Deletions Induced by Cas9 Cleavage while Not in the BEs System. Mol Ther Nucleic Acids 21, 523-526. [PubMed: 32711379]
Stadtmauer EA, Fraietta JA, Davis MM, Cohen AD, Weber KL, Lancaster E, Mangan PA, Kulikovskaya I, Gupta M, Chen F, et al. (2020). CRISPR-engineered T cells in patients with refractory cancer. Science 367.
Staring J, Raaben M, and Brummelkamp TR (2018). Viral escape from endosomes and host detection at a glance. J Cell Sci 131.
Suh S, Choi EH, Leinonen H, Foik AT, Newby GA, Yeh WH, Dong Z, Kiser PD, Lyon DC, Liu DR, et al. (2021). Restoration of visual function in adult mice with an inherited retinal disease via adenine base editing. Nat Biomed Eng 5, 169-178. [PubMed: 33077938]
Sweeney NP, and Vink CA (2021). The impact of lentiviral vector genome size and producer cell genomic to gag-pol mRNA ratios on packaging efficiency and titre. Mol Ther Methods Clin Dev 21, 574-584. [PubMed: 34095341]
Tabebordbar M, Lagerborg KA, Stanton A, King EM, Ye S, Tellez L, Krunnfusz A, Tavakoli S, Widrick JJ, Messemer KA, et al. (2021). Directed evolution of a family of AAV capsid variants enabling potent muscle-directed gene delivery across species. Cell 184, 4919-4938 e4922. [PubMed: 34506722]
Tabebordbar M, Zhu K, Cheng JKW, Chew WL, Widrick JJ, Yan WX, Maesner C, Wu EY, Xiao R, Ran FA, et al. (2016). In vivo gene editing in dystrophic mouse muscle and muscle stem cells. Science 351, 407-411. [PubMed: 26721686]
Taha EA, Lee J, and Hotta A (2022). Delivery of CRISPR-Cas tools for in vivo genome editing therapy: Trends and challenges. J Control Release 342, 345-361. [PubMed: 35026352]
Tornabene P, and Trapani I (2020). Can Adeno-Associated Viral Vectors Deliver Effectively Large Genes? Hum Gene Ther 31, 47-56. [PubMed: 31916856]
Truong DJ, Kuhner K, Kuhn R, Werfel S, Engelhardt S, Wurst W, and Ortiz O (2015). Development of an intein-mediated split-Cas9 system for gene therapy. Nucleic Acids Res 43, 6450-6458. [PubMed: 26082496]
Turchiano G, Andrieux G, Klermund J, Blattner G, Pennucci V, El Gaz M, Monaco G, Poddar S, Mussolino C, Cornu TI, et al. (2021). Quantitative evaluation of chromosomal rearrangements in gene-edited human stem cells by CAST-Seq. Cell Stem Cell 28, 1136-1147 e1135. [PubMed: 33626327]
Urnov FD (2018). Genome Editing B.C. (Before CRISPR): Lasting Lessons from the "Old Testament". CRISPR J 1, 34-46. [PubMed: 31021186]
van Haasteren J, Li J, Scheideler OJ, Murthy N, and Schaffer DV (2020). The delivery challenge: fulfilling the promise of therapeutic genome editing. Nat Biotechnol 38, 845-855. [PubMed: 32601435]

Varkouhi AK, Scholte M, Storm G, and Haisma HJ (2011). Endosomal escape pathways for delivery of biologicals. J Control Release 151, 220-228. [PubMed: 21078351]
Vassalli G, Bueler H, Dudler J, von Segesser LK, and Kappenberger L (2003). Adeno-associated virus (AAV) vectors achieve prolonged transgene expression in mouse myocardium and arteries in vivo: a comparative study with adenovirus vectors. Int J Cardiol 90, 229-238. [PubMed: 12957756]
Veiga N, Goldsmith M, Granot Y, Rosenblum D, Dammes N, Kedmi R, Ramishetti S, and Peer D (2018). Cell specific delivery of modified mRNA expressing therapeutic proteins to leukocytes. Nat Commun 9, 4493. [PubMed: 30374059]
Verdera HC, Kuranda K, and Mingozzi F (2020). AAV Vector Immunogenicity in Humans: A Long Journey to Successful Gene Transfer. Mol Ther 28, 723-746. [PubMed: 31972133]
Villiger L, Grisch-Chan HM, Lindsay H, Ringnalda F, Pogliano CB, Allegri G, Fingerhut R, Haberle J, Matos J, Robinson MD, et al. (2018). Treatment of a metabolic liver disease by in vivo genome base editing in adult mice. Nat Med 24, 1519-1525. [PubMed: 30297904]
Villiger L, Rothgangl T, Witzigmann D, Oka R, Lin PJC, Qi W, Janjuha S, Berk C, Ringnalda F, Beattie MB, et al. (2021). In vivo cytidine base editing of hepatocytes without detectable off-target mutations in RNA and DNA. Nat Biomed Eng 5, 179-189. [PubMed: 33495639]
Voelkel C, Galla M, Maetzig T, Warlich E, Kuehle J, Zychlinski D, Bode J, Cantz T, Schambach A, and Baum C (2010). Protein transduction from retroviral Gag precursors. Proc Natl Acad Sci U S A 107, 7805-7810. [PubMed: 20385817]
Wagner DL, Amini L, Wendering DJ, Burkhardt LM, Akyuz L, Reinke P, Volk HD, and SchmueckHenneresse M (2019). High prevalence of Streptococcus pyogenes Cas9-reactive T cells within the adult human population. Nat Med 25, 242-248. [PubMed: 30374197]
Wagner DL, Peter L, and Schmueck-Henneresse M (2021). Cas9-directed immune tolerance in humans-a model to evaluate regulatory T cells in gene therapy? Gene Ther 28, 549-559. [PubMed: 33574580]
Wang D, Mou H, Li S, Li Y, Hough S, Tran K, Li J, Yin H, Anderson DG, Sontheimer EJ, et al. (2015). Adenovirus-Mediated Somatic Genome Editing of Pten by CRISPR/Cas9 in Mouse Liver in Spite of Cas9-Specific Immune Responses. Hum Gene Ther 26, 432-442. [PubMed: 26086867]
Wang D, Tai PWL, and Gao G (2019). Adeno-associated virus vector as a platform for gene therapy delivery. Nat Rev Drug Discov 18, 358-378. [PubMed: 30710128]
Wang D, Zhang F, and Gao G (2020). CRISPR-Based Therapeutic Genome Editing: Strategies and In Vivo Delivery by aAV Vectors. Cell 181, 136-150. [PubMed: 32243786]
Wang L, Xue W, Zhang H, Gao R, Qiu H, Wei J, Zhou L, Lei YN, Wu X, Li X, et al. (2021). Eliminating base-editor-induced genome-wide and transcriptome-wide off-target mutations. Nat Cell Biol 23, 552-563. [PubMed: 33972728]
Wang W, Bartholomae CC, Arens A, Gabriel R, Glimm H, von Kalle C, and Schmidt M (2010). Genome Wide Characterization of Insertion Profiles of Integration Deficient Lentiviral Vectors Using Two-Sided Non-Restrictive and Standard LAM-PCR. Molecular Therapy 18.
Wang Z, Zhu T, Qiao C, Zhou L, Wang B, Zhang J, Chen C, Li J, and Xiao X (2005). Adenoassociated virus serotype 8 efficiently delivers genes to muscle and heart. Nat Biotechnol 23, 321-328. [PubMed: 15735640]
Wanisch K, and Yanez-Munoz RJ (2009). Integration-deficient lentiviral vectors: a slow coming of age. Mol Ther 17, 1316-1332. [PubMed: 19491821]
Webber BR, Lonetree CL, Kluesner MG, Johnson MJ, Pomeroy EJ, Diers MD, Lahr WS, Draper GM, Slipek NJ, Smeester BA, et al. (2019). Highly efficient multiplex human T cell engineering without double-strand breaks using Cas9 base editors. Nat Commun 10, 5222. [PubMed: 31745080]
Weber T (2021). Anti-AAV Antibodies in AAV Gene Therapy: Current Challenges and Possible Solutions. Front Immunol 12, 658399. [PubMed: 33815421]
Wei T, Cheng Q, Farbiak L, Anderson DG, Langer R, and Siegwart DJ (2020a). Delivery of Tissue-Targeted Scalpels: Opportunities and Challenges for In Vivo CRISPR/Cas-Based Genome Editing. ACS Nano 14, 9243-9262. [PubMed: 32697075]

Wei T, Cheng Q, Min YL, Olson EN, and Siegwart DJ (2020b). Systemic nanoparticle delivery of CRISPR-Cas9 ribonucleoproteins for effective tissue specific genome editing. Nat Commun 11, 3232. [PubMed: 32591530]

Wittrup A, Ai A, Liu X, Hamar P, Trifonova R, Charisse K, Manoharan M, Kirchhausen T, and Lieberman J (2015). Visualizing lipid-formulated siRNA release from endosomes and target gene knockdown. Nat Biotechnol 33, 870-876. [PubMed: 26192320]

Witzigmann D, Kulkarni JA, Leung J, Chen S, Cullis PR, and van der Meel R (2020). Lipid nanoparticle technology for therapeutic gene regulation in the liver. Adv Drug Deliv Rev 159, 344-363. [PubMed: 32622021]

Wu Z, Asokan A, and Samulski RJ (2006). Adeno-associated virus serotypes: vector toolkit for human gene therapy. Mol Ther 14, 316-327. [PubMed: 16824801]

Wu Z, Yang H, and Colosi P (2010). Effect of genome size on AAV vector packaging. Mol Ther 18, 80-86. [PubMed: 19904234]

Xiao Y, Muhuri M, Li S, Qin W, Xu G, Luo L, Li J, Letizia AJ, Wang SK, Chan YK, et al. (2019). Circumventing cellular immunity by miR142-mediated regulation sufficiently supports rAAV-delivered OVA expression without activating humoral immunity. JCI Insight 5.

Xu L, Zhang C, Li H, Wang P, Gao Y, Mokadam NA, Ma J, Arnold WD, and Han R (2021). Efficient precise in vivo base editing in adult dystrophic mice. Nat Commun 12, 3719. [PubMed: 34140489]

Xu Z, Yue Y, Lai Y, Ye C, Qiu J, Pintel DJ, and Duan D (2004). Trans-splicing adeno-associated viral vector-mediated gene therapy is limited by the accumulation of spliced mRNA but not by dual vector coinfection efficiency. Hum Gene Ther 15, 896-905. [PubMed: 15353044]

Yang S, Chang R, Yang H, Zhao T, Hong Y, Kong HE, Sun X, Qin Z, Jin P, Li S, et al. (2017). CRISPR/Cas9-mediated gene editing ameliorates neurotoxicity in mouse model of Huntington's disease. J Clin Invest 127, 2719-2724. [PubMed: 28628038]

Yeh WH, Chiang H, Rees HA, Edge ASB, and Liu DR (2018). In vivo base editing of post-mitotic sensory cells. Nat Commun 9, 2184. [PubMed: 29872041]

Yeh WH, Shubina-Oleinik O, Levy JM, Pan B, Newby GA, Wornow M, Burt R, Chen JC, Holt JR, and Liu DR (2020). In vivo base editing restores sensory transduction and transiently improves auditory function in a mouse model of recessive deafness. Sci Transl Med 12.

Yin D, Ling S, Wang D, Dai Y, Jiang H, Zhou X, Paludan SR, Hong J, and Cai Y (2021). Targeting herpes simplex virus with CRISPR-Cas9 cures herpetic stromal keratitis in mice. Nat Biotechnol 39, 567-577. [PubMed: 33432198]

Yin H, Kauffman KJ, and Anderson DG (2017a). Delivery technologies for genome editing. Nat Rev Drug Discov 16, 387-399. [PubMed: 28337020]

Yin H, Song CQ, Dorkin JR, Zhu LJ, Li Y, Wu Q, Park A, Yang J, Suresh S, Bizhanova A, et al. (2016). Therapeutic genome editing by combined viral and non-viral delivery of CRISPR system components in vivo. Nat Biotechnol 34, 328-333. [PubMed: 26829318]

Yin H, Song CQ, Suresh S, Wu Q, Walsh S, Rhym LH, Mintzer E, Bolukbasi MF, Zhu LJ, Kauffman K, et al. (2017b). Structure-guided chemical modification of guide RNA enables potent non-viral in vivo genome editing. Nat Biotechnol 35, 1179-1187. [PubMed: 29131148]

Zhang H, Bamidele N, Liu P, Ojelabi O, Gao XD, Rodriguez T, Cheng H, Xie J, Gao G, Wolfe SA, et al. (2022). Adenine Base Editing in vivo with a Single Adeno-Associated Virus Vector. bioRxiv.

Zhang W, Cao S, Martin JL, Mueller JD, and Mansky LM (2015). Morphology and ultrastructure of retrovirus particles. AIMS Biophys 2, 343-369. [PubMed: 26448965]

Zheng C, Liang SQ, Liu B, Liu P, Kwan SY, Wolfe SA, and Xue W (2022). A flexible split prime editor using truncated reverse transcriptase improves dual-AAV delivery in mouse liver. Mol Ther.

Zinn E, Pacouret S, Khaychuk V, Turunen HT, Carvalho LS, Andres-Mateos E, Shah S, Shelke R, Maurer AC, Plovie E, et al. (2015). In Silico Reconstruction of the Viral Evolutionary Lineage Yields a Potent Gene Therapy Vector. Cell Rep 12, 1056-1068. [PubMed: 26235624]

Zuris JA, Thompson DB, Shu Y, Guilinger JP, Bessen JL, Hu JH, Maeder ML, Joung JK, Chen ZY, and Liu DR (2015). Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo. Nat Biotechnol 33, 73-80. [PubMed: 25357182]

![img-0.jpeg](img-0.jpeg)

Figure 1. Overview of therapeutic gene editing technologies
Nucleases create targeted double-strand DNA breaks (DSBs), which generally lead to uncontrolled mixtures of insertions and deletions (indels) that are useful for gene disruption. In certain types of dividing cells, DSBs in the presence of a DNA donor template can also lead to homology-directed repair (HDR) outcomes that can support gene correction, though indel byproducts typically accompany HDR outcomes. Base editors mediate targeted $\mathrm{C} \bullet \mathrm{G}$-to- $\mathrm{T} \bullet \mathrm{A}, \mathrm{A} \bullet \mathrm{T}$-to- $\mathrm{G} \bullet \mathrm{C}$, or $\mathrm{C} \bullet \mathrm{G}$-to- $\mathrm{G} \bullet \mathrm{C}$ conversions with minimal indel byproducts. Prime editors enable targeted single-nucleotide conversions, insertions, deletions, and combinations thereof with minimal indel byproducts. See also Anzalone et al., 2020 for a more detailed description of gene editing mechanisms.

![img-1.jpeg](img-1.jpeg)

Figure 2. Requirements for efficient in vivo delivery of gene editing agents
(A) An appropriate delivery vehicle (gray circles) for gene editing agents must efficiently encapsulate DNA or mRNA encoding gene editing agents, or gene editing proteins or ribonucleoproteins (RNPs). Delivery vehicles must protect their cargos from sequestration or degradation in vivo prior to encountering target cells.
(B) Delivery vehicles must bind target cells, typically by engaging cell surface receptors with complementary molecules on the surface of the delivery vehicle.
(C) Delivery vehicles must traverse the target cell membrane, typically through receptormediated endocytosis.
(D) Following endocytosis, delivery vehicles must either escape endosomes and release their cargo, or fuse with endosomes to release their cargo into the target cell cytosol. The cargo must then be trafficked to the appropriate cellular compartment (typically the nucleus) for successful gene editing to occur.

|  | Type | Size | Genome | Challenges |
| :--: | :--: | :--: | :--: | :--: |
| A Adeno-associated virus | Non-enveloped | $\sim 25 \mathrm{~nm}$ | $\sim 5 \mathrm{~kb}$, ssDNA | - Limited packaging capacity <br> - Prolonged expression leads to risk of off-target editing <br> - Pre-existing immunity to natural serotypes <br> - Low but nonzero risk of viral DNA integration |
| B Lentivirus | Enveloped | $\sim 90 \mathrm{~nm}$ | $\sim 10 \mathrm{~kb}$, ssRNA | - Genomic integration <br> - Prolonged expression leads to risk of off-target editing <br> - Limited efficiency in vivo |
| C Adenovirus | Non-enveloped | $\sim 100 \mathrm{~nm}$ | $8 \mathrm{~kb}-36 \mathrm{~kb}^{*}$ dsDNA | - Immunogenicity <br> - Prolonged expression leads to risk of off-target editing |

*Cargo capacity of adenovirus can be increased up to 36 kb by removing viral genes required for replication.

Figure 3. Overview and comparison of viral delivery methods
(A) Adeno-associated viruses are single-stranded DNA viruses with cargo capacity of 5 kb .
(B) Lentiviral vectors are enveloped viruses with that package a single-stranded RNA genome of up to 10 kb .
(C) Adenoviral vectors are double-stranded DNA viruses with a packaging capacity of 8 kb that can be expanded to 36 kb in "gutless" vectors devoid of all the viral protein-coding genes.

![img-2.jpeg](img-2.jpeg)

Figure 4. Lipid nanoparticle (LNP) delivery
LNPs consist of four key components and can efficiently encapsulate various RNAs. Encapsulated mRNAs are typically modified by including alternative nucleotides during in vitro transcription, such as N1-methylpseudouridine, to increase cellular stability after delivery. Encapsulated guide RNAs are chemically modified at various positions, including with 2'-O-methylation and phosphorothioate linkages, which enhance the stability of the guide RNA.

![img-3.jpeg](img-3.jpeg)

Figure 5. Virus-like particle (VLP) delivery
General schematic of the most important components of mRNA-packaging VLPs (left) and protein- or RNP-packaging VLPs (right). In both types of VLPs, the retroviral gag and gag-pro-pol polyproteins provide structural stability and the viral protease required for cleaving the polyproteins into distinct subunits during particle maturation. In mRNA-packaging VLPs, fusion of gag with an RNA-binding protein (RBP) enables encapsulation of mRNA cargo containing the RNA aptamer recognized by the RBP. If necessary, a guide RNA is typically encoded on an integration-deficient lentivirus (IDLV) genome. In RNP-packaging VLPs, fusion of gag with protein cargo via a viral protease-cleavable linker directs encapsulation of protein into particles as they form. Cleavage of the linker after particle maturation enables the release of free protein cargo into transduced cells. When packaging RNPs, guide RNAs can be co-packaged into particles due to the intrinsic affinity between the Cas9 protein and its guide RNA. In engineered VLPs (eVLPs), cargo packaging, release, and localization have been optimized through protein engineering (Banskota et al., 2022).