# Review <br> CRISPR-Based Gene Therapies: From Preclinical to Clinical Treatments 

Marine Laurent ${ }^{1, \dagger}$, Marine Geoffroy ${ }^{2, \dagger}$, Giulia Pavani ${ }^{3, *}$ and Simon Guiraud ${ }^{2, * *}$ (D)<br>check for updates

Citation: Laurent, M.; Geoffroy, M.; Pavani, G.; Guiraud, S. CRISPR-Based Gene Therapies: From Preclinical to Clinical Treatments. Cells 2024, 13, 800. https://doi.org/10.3390/cells13100800

Academic Editor: Toshifumi Yokota
Received: 26 March 2024
Revised: 3 May 2024
Accepted: 5 May 2024
Published: 8 May 2024

Copyright: (C) 2024 by the authors. Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (https:// creativecommons.org/licenses/by/ $4.0 /$ ).

1 INTEGRARE, UMR_S951, Genethon, Inserm, Univ Evry, Université Paris-Saclay, 91190 Evry, France
2 SQY Therapeutics, 78180 Montigny-le-Bretonneux, France
3 Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA

* Correspondence: pavanig@chop.edu (G.P.); simon.guiraud@sqy-synthena.com (S.G.); Tel.: +1-2674261222 (G.P.); +44-(0)1865-285880 (S.G.)
+ These authors contributed equally to this work as co-first authors.


#### Abstract

In recent years, clustered regularly interspaced short palindromic repeats (CRISPRs) and CRISPR-associated (Cas) protein have emerged as a revolutionary gene editing tool to treat inherited disorders affecting different organ systems, such as blood and muscles. Both hematological and neuromuscular genetic disorders benefit from genome editing approaches but face different challenges in their clinical translation. The ability of CRISPR/Cas9 technologies to modify hematopoietic stem cells ex vivo has greatly accelerated the development of genetic therapies for blood disorders. In the last decade, many clinical trials were initiated and are now delivering encouraging results. The recent FDA approval of Casgevy, the first CRISPR/Cas9-based drug for severe sickle cell disease and transfusion-dependent $\beta$-thalassemia, represents a significant milestone in the field and highlights the great potential of this technology. Similar preclinical efforts are currently expanding CRISPR therapies to other hematologic disorders such as primary immunodeficiencies. In the neuromuscular field, the versatility of CRISPR/Cas9 has been instrumental for the generation of new cellular and animal models of Duchenne muscular dystrophy (DMD), offering innovative platforms to speed up preclinical development of therapeutic solutions. Several corrective interventions have been proposed to genetically restore dystrophin production using the CRISPR toolbox and have demonstrated promising results in different DMD animal models. Although these advances represent a significant step forward to the clinical translation of CRISPR/Cas9 therapies to DMD, there are still many hurdles to overcome, such as in vivo delivery methods associated with high viral vector doses, together with safety and immunological concerns. Collectively, the results obtained in the hematological and neuromuscular fields emphasize the transformative impact of CRISPR/Cas9 for patients affected by these debilitating conditions. As each field suffers from different and specific challenges, the clinical translation of CRISPR therapies may progress differentially depending on the genetic disorder. Ongoing investigations and clinical trials will address risks and limitations of these therapies, including long-term efficacy, potential genotoxicity, and adverse immune reactions. This review provides insights into the diverse applications of CRISPR-based technologies in both preclinical and clinical settings for monogenic blood disorders and muscular dystrophy and compare advances in both fields while highlighting current trends, difficulties, and challenges to overcome.


Keywords: CRISPR/Cas9; gene editing; blood disorders; neuromuscular disorders; $\beta$-hemoglobinopathies; Duchenne muscular dystrophy

## 1. Introduction

Site-specific endonucleases can modify the genome at a specific DNA locus, representing a promising tool to cure genetic disorders, including blood and muscular diseases [1]. Zinc finger nucleases (ZFNs) and transcription activator-like effector nucleases (TALENs) belong to the first generation of gene editing tools, comprising a customizable DNA-binding

domain fused to a DNA cleavage module [2]. From the 1990s to the early 2010s, many scientists contributed to the identification of a microbial adaptive immune system against phage infection that relied on short repeated sequences (clustered regularly interspaced short palindromic repeats, CRISPR) and an RNA-guided DNA nuclease (CRISPR-associated protein 9, Cas9) [5,6,7] (Figure 1). The development of genome editing methods based on CRISPR/Cas9 technologies dramatically revolutionized the gene therapy field, rapidly expanding its therapeutic applications [8,9,10].
![img-0.jpeg](img-0.jpeg)

Figure 1. Timeline highlighting the important milestones in CRISPR gene editing field (black arrows). Achievements in CRISPR-based blood disease research are specified in blue and advents in CRISPRmediated Duchenne muscular dystrophy research are in green.

The CRISPR/Cas9 system uses a chimeric single-guide RNA (sgRNA) to direct an endonuclease (Cas9) to specific DNA targets, where it generates double-strand breaks (DSBs) [9] (Figure 2A). These DSBs can be repaired by the cell through two competing pathways: the error-prone non-homologous end joining, which generates insertions and deletions at the cutting site, or the more precise homology-directed repair, which requires a DNA template to introduce specific modifications [11]. Pioneering CRISPR/Cas9 approaches have been successfully used in several clinical trials for lung cancer (NCT02793856) [12] and for $\beta$-hemoglobinopathies (NCT03655678, NCT03745287, NCT04443907), resulting in the first FDA-approved CRISPR therapy (www.fda.gov/news-events/press-announcements/ fda-approves-first-gene-therapies-treat-patients-sickle-cell-disease, accessed on 8 December 2023). Despite these successes, it is becoming more evident that Cas9-induced DSBs can lead to genotoxic effects that include large insertions/deletions, chromosomal translocation, and chromothripsis [13,14,15,16], raising long-term safety concerns.

![img-1.jpeg](img-1.jpeg)

**Figure 2.** The expanding CRISPR/Cas9 toolbox. (**A**) CRISPR/Cas9, a gene-editing technology, is composed of two essential components: a guide RNA (sgRNA) to target a desired DNA locus, and the endonuclease Cas9 (CRISPR-associated protein 9). CRISPR/Cas9 leads to a double-stranded DNA break, allowing genomic modifications. (**B**) Base editing: nCas9 are fused to cytidine or adenosine deaminases (cytosine base editor or adenine base editor) to induce substitutions or base edits without inducing double-strand breaks. (**C**) Prime editing: nCas9 can be fused to a reverse transcriptase (RT) and extended prime editing guide RNA (pegRNA) (prime editor) to mediate targeted small insertions, deletions, and base-to-base conversions. Twin prime editing is an innovative iteration of prime editor protein based on two pegRNAs for programmable deletion, substitution, insertion, or inversion of larger DNA sequences. (**D**) dCas9 can be fused to transcriptional activators (CRISPRa) or (**E**) transcriptional repressors (CRISPRi) to precisely modulate gene expression. (**F**) dCas9 can be fused to epigenetic effector domains as histone/DNA modifier to alter the epigenetic state at precise DNA locations. *dCas9—dead Cas9*; *nCas9—nickase Cas9*.

Base editing provides a more precise and efficient method for making single-base changes compared with traditional gene editing approaches [17] (Figure 2B). Potentially, around 60% of pathogenic point mutations could be corrected by base editors (Bes). Bes were developed by fusing a Cas9 nickase (nCas9) to a deaminase [18,19]. Unlike traditional CRISPR/Cas9, Bes do not generate DSBs; instead, the deaminase domain promotes base conversions at specific genomic locations upon sgRNA targeting. Cytosine Bes (CBE) can convert a C:G to a T:A base pair, while adenine Bes (ABEs) can convert A:T to G:C. Even if new Bes with less sequence constraints [20], C-to-G conversion capabilities (glycosylase base editors, GBEs [21,22]), and different editing windows have been developed [23], many pathological mutations cannot be targeted with this technology, including small and large deletions and multi-nucleotide changes. New editing tools have been generated.

ated to overcome these limitations, including prime editing [24], CRISPR/integrases [25], CRISPR/transposases [26], and large serine recombinases [27].

Prime editors (PE) can introduce specific mutations without inducing DSBs and perform insertions up to 44 bp [24] (Figure 2C). This system is composed of a nCas9 fused with a reverse transcriptase, and a prime editing gRNA (peg-RNA) that serves both as a guideRNA and a template to install the desired modification at a specific site. By using a pair of partially complementary pegRNAs, it is possible to increase insert size up to 110 bp or generate large genomic deletions (paired-peg, TWIN-PE) [28], though larger cassettes cannot be delivered using this system alone.

Serine site-specific recombinases like Bxb1 can integrate, invert, and excise large DNA fragments [29] by promoting the recombination of attP and attB sites. This reaction is irreversible and unidirectional, making this system ideal for stably integrating expression cassettes into the genome where one of the sites is present [30,31]. This observation prompted the development of novel editing tools like PASTE (programmable addition via site-specific targeting elements), which consists of a PE fused with Bxb1 [25]. PASTE can insert DNA payloads up to 35 Kb at specific genomic locations and could be beneficial for diseases involving large genes.

Apart from its capability to integrate, modify bases, or delete DNA sequences, the CRISPR/Cas9 system can be leveraged to modify gene expression (Figure 2D-F). Dead Cas9 (dCas9) can bind to specific DNA sequences by sgRNA targeting but has no nuclease activity. dCas9 can be combined with various enzymes like transcriptional repressors (CRISPR interference, CRISPRi) and activators (CRISPR activation, CRISPRa) to modulate gene expression over several orders of magnitude [32]. After binding, CRISPRi creates a steric hindrance that prevents transcription factors (TF) from attaching to the promoter region of the targeted gene, resulting in decreased expression. The system is more effective when transcriptional repressor domains (such as Krüppel-associated box, KRAB [33]) are fused to dCas9. Inversely, the CRISPRa system utilizes transcriptional activator domains (such as VP64, RTA, VP64, HSF1, and p65) to recruit RNA polymerase or TFs to promote and increase the expression of downstream genes [34,35]. Gene transcription can be altered at the epigenetic level by modulating DNA accessibility or chromatin structure at specific sites. Different epigenetic enzymes have been employed in combination with dCas9 to manipulate DNA methylation in gene promoters. For example, DNA methylases (DNMT) or demethylases (TETF1) have been used to modify the methylation status of specific CpG islands, therefore changing gene expression [36]. Chromatin accessibility can be altered by recruiting fusion Cas9 proteins containing histone modifiers such as histone acetylases (P300) that facilitate chromatin opening [37] or histone deacetylases (HDAC) that allow histones to wrap DNA more tightly [38].

In this review, we will summarize main preclinical and clinical applications of CRISPRbased technologies, focusing on monogenic blood disorders and muscular dystrophy, comparing advances in both fields, and highlighting current trends, difficulties, and challenges to overcome.

# 2. CRISPR-Based Gene Editing for Blood Disease 

The only curative treatment for many inherited blood disorders is allogeneic transplantation of healthy hematopoietic stem/progenitor cells (HSPC). This procedure is limited by the availability of compatible donors and is associated with serious complications and life-threatening risks [39,40]. Gene therapy strategies based on autologous transplantation of ex vivo transduced HSPCs have been successfully developed. Despite the clinical benefits of these therapies, retro/lentiviral gene addition can result in subtherapeutic or unregulated transgene expression and carries an intrinsic risk of insertional mutagenesis. Hence, the ability to correct patient HSPCs using genome editing technologies represents a game changer for the field. HSPCs are routinely mobilized and isolated from patients and healthy donors. They can be modified ex vivo with different methodologies and reconstitute the entire blood system when transplanted [41]. These intrinsic properties

of HSPCs, together with the extensive experience acquired through decades of lentiviral gene therapies, have accelerated the development of gene editing approaches for many hematological disorders. In this review, we will focus on two families of inherited blood diseases: $\beta$-hemoglobinopathies and primary immunodeficiencies.

# 2.1. $\beta$-Hemoglobinopathies 

$\beta$-hemoglobinopathies are conditions affecting the $\beta$ chain of hemoglobin, the main oxygen carrier in humans. In adults, the hemoglobin tetramer is composed of four chains-two $\alpha$ and two $\beta$-globin subunits ( $\mathrm{HbA}, \alpha_{2} \beta_{2}$ )—both necessary to transport oxygen from the lungs to the tissues by the red blood cells (RBCs) [42]. $\beta$-hemoglobinopathies are the most frequent monogenic disease, affecting 400,000 births worldwide each year, the main forms being sickle cell disease (SCD) and $\beta$-thalassemia [43].

SCD is caused by an A-to-T mutation in the $\beta$-globin gene $(H B B)$ resulting in the formation of hemoglobin $S\left(\mathrm{HbS}, \beta 6^{\mathrm{Glu} \rightarrow \mathrm{Val}}\right)$ [44]. Under low-oxygen tension, HbS polymerizes, making erythrocytes more rigid and with a characteristic sickle shape. These sickled RBCs trigger hemolytic anemia and vaso-occlusion events, causing ischemic damage to tissues and leading to acute pain crisis and organ failure [45]. In contrast, $\beta$-thalassemia is caused by a wide range of mutations, including large deletions that decrease the amount of functional $\beta$-globin protein produced. The imbalance between $\beta$ and $\alpha$-chains causes precipitation of $\alpha$-globin in RBCs, resulting in hemolysis and impaired erythropoiesis [46].

Despite different pathophysiology, both disorders can be ameliorated by the coinheritance of benign mutations that increase levels of fetal hemoglobin ( $\mathrm{HbF}, \alpha_{2} \gamma_{2}$ ), as $\gamma$-globin can replace $\beta$-globin function and has antisickling properties [43,47]. For these reasons, many gene editing approaches for $\beta$-hemoglobinopathies rely on $\gamma$-globin reactivation. To achieve this goal, nucleases can target transcriptional repressors of $\gamma$-globin genes (HBG1/2) like B-cell lymphoma/leukemia 11A (BCL11a) or inhibitory regions in HBG1/2 promoters. Many clinical trials were started in the past decade leveraging different CRISPR/Cas9-based tools (Table 1). Recently, the FDA approved the first CRISPR/Cas9 based treatment for severe SCD and transfusion dependent $\beta$-thalassemia (TDT)—Exa-cel (Casgevy) commercialized by Vertex Pharmaceuticals and CRISPR Therapeutics. This editing treatment consists of autologous HSPCs electroporated ex vivo with Cas9/gRNA mRNA to disrupt the erythroid-specific enhancer of BCL11a. Specifically, the BCL11A locus harbors three DNase I hypersensitivity sites (DHS), serving as erythroid-specific enhancers. These sites are situated at distances of $+62,+58$, and +55 kb from the transcriptional start site [48]. Among these, the DHS at +58 kb is crucial for BCL11a expression in erythroid cells [49,50]. Two clinical trials were initiated in 2018 for SCD (NCT05329649; NCT03745287; NCT05951205) and TDT (NCT05356195; NCT03655678) (Table 1). Initial findings reported successful engraftment of edited HSPCs ( $80 \%$ BCL11a alleles) one year post-treatment. This resulted in significant production of HbF , with a reduction in blood transfusions and vaso-occlusive events (in SCD patients) [51]. Two analogous strategies devised by Bioray (BRL-101) and Edigene (ET-01) to disrupt the +58 DHS of BCL11a are under clinical evaluation for TDT (BRL-101: NCT04211480; NCT05577312; NCT04205435; ET-01: NCT04390971; NCT05752123; NCT04925206) (Table 1). During the 18-month follow-up for BRL-101 (NCT04211480), two patients exhibited successful engraftment of modified HSPCs, achieving an editing rate of $85 \%$ in the bone marrow. This led to a clinically meaningful rise in HbF levels that translated to transfusion independence [52]. These results were also achieved in ten TDT patients [53]. Preliminary results from ET-01 also showed promising outcomes [54]. ZFNs were also utilized to disrupt the GATA-1 binding site in BCL11a locus in TDT (NCT03432364) (Table 1). Despite an initial success, a decline in $\gamma$-globin synthesis was observed in treated patients two years post-administration, requiring resumption of blood transfusions [55,56].

An alternative approach to reactivate HbF is to disrupt the BCL11a binding site on the $\gamma$-globin locus. In TDT patients, transplantation of HSPCs modified with CRISPR/Cas9 at the $\gamma$-globin promoter (ChiCTR2100052858; ChiCTR2100053406) increased HbF production,

eliminating the necessity for blood transfusions [57,58] (Table 1). Alternatively, Cas12 nuclease has been successfully utilized to disrupt repressors binding sites on $H B G 1 / 2$ promoters [59]. This editing strategy showed high editing efficiency, with a $40 \%$ increase in HbF in vivo, and is currently under clinical investigation for TDT (NCT05444894, NCT06041620) and SCD (NCT04853576) (Table 1). Because of their superior safety profile and multiplexing properties, BEs have emerged as novel tools to reactivate $\gamma$-globin genes. This could be achieved by disrupting repressors binding sites at the $H B G 1 / 2$ promoters or by generating de novo DNA motifs for transcriptional activators. Antoniou et al. used BEs to generate several mutations associated with hereditary persistence of HbF and compared their impact on $\gamma$-globin expression [60]. In this study, the generation of a binding site for KLF1 using ABE showed the greatest HbF reactivation in SCD cells. Beam Therapeutic also used ABE to generate a point mutation in $H B G 1 / 2$ promoters to restore $\gamma$-globin expression. This approach is currently being used in a clinical trial for SCD (NCT05456880) (Table 1).

Table 1. Clinical trials of blood diseases using CRISPR/Cas9 gene editing system.

| Disease | Clinical trial | Therapeutic approach | Editor | Name | Sponsor | Study Phase |
| --- | --- | --- | --- | --- | --- | --- |
| SCD | NCT03653247 | Gene disruption of BCL11A using ZFN | ZFN | BIVV003 | Sangamo Therapeutic (Richmond, CA, United States) | Phase 1/2 |
| | NCT05329649 | Gene disruption of BCL11A using CRISPR-Cas9 | CRISPR-Cas9 | CTX001 | Vertex Pharmaceuticals Inc. (Boston, MA, United States) | Approved |
| | NCT05745287 | | | Exa-cel | | |
| | NCT05951205 | | | | | |
| | NCT04443907 | Gene disruption of BCL11A using CRISPR-Cas9 | CRISPR-Cas9 | OTQ923 | Novartis pharmaceuticals (Basel, Switzerland) | Phase 1 |
| | NCT05456880 | Gene disruption of $\gamma$-globin promoter using adenine base editor | ABE | BEAM-101 | Beam Therapeutics Inc. (Cambridge, MA, United States) | Phase 1/2 |
| | NCT04853576 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas12a | CRISPR-
Cas12a | EDIT-301 | Editas Medicine, Inc. (Cambridge, MA, United States) | Phase 1/2 |
| | NCT04819841 | Gene correction using CRISPR-Cas9 and AAV6 | CRISPR-Cas9 | CEDAR | Graphite Bio (San Francisco, CA, United States) | Phase 1/2 |
| | NCT04774536 | Gene correction using CRISPR-Cas9 and ssODN | CRISPR-Cas9 | CRISPR_SCD001 | Mark Walters, MD | Phase 1/2 |
| $\begin{gathered} \beta- \ \text { thalassemia } \end{gathered}$ | NCT03432364 | Gene disruption of $\gamma$-globin promoter using ZFN | ZFN | ST-400 | Sangamo Therapeutics (Richmond, CA, United States) | Phase 1/2 |
| | NCT05356195 | Gene disruption of BCL11A using CRISPR-Cas9 | CRISPR-Cas9 | CTX001 | Vertex Pharmaceuticals Inc. (Boston, MA, United States) | Approved |
| | NCT03655678 | Gene disruption of BCL11A using CRISPR-Cas9 | CRISPR-Cas9 | BRL-101 | Bioray Laboratories (Boston, MA, United States) | Phase 1 |
| | NCT05577312 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas9 | CRISPR-Cas9 | ET-01 | EdiGene Inc. (Guangzhou, China) | Phase 1 |
| | NCT04390971 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas9 | CRISPR-Cas9 | ET-01 | Guangzhou Reforgene Medicine (Guangzhou, China) | Phase 1 |
| | NCT05752123 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas9 | CRISPR-Cas9 | RM-001 | Editas Medicine, Inc. (Cambridge, MA, United States) | Phase 1/2 |
| | NCT04925206 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas9 | CRISPR-Cas9 | EM-001 | Institute of Hematology \& Blood Diseases Hospital (Tianjin, China) | Interventional |
| | ChiCTR2100052858 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas9 | CRISPR-Cas9 | RM-001 | | Phase 1/2 |
| | NCT052100053406 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas12a | CRISPR-Cas12a | EDIT-301 | Editas Medicine, Inc. (Cambridge, MA, United States) | Phase 1/2 |
| | NCT056041620 | Gene disruption of $\gamma$-globin promoter using CRISPR-Cas12b | CRISPR-
Cas12b | VGB-Ex01 | Institute of Hematology \& Blood Diseases Hospital (Tianjin, China) | Interventional |
| | NCT05442346 | Gene disruption of $\gamma$-globin promoter using glycosylase base editors | Glycosylase base editors | - | Bioray Laboratories (Shanghai, China) | Phase 1/2 |
| | NCT04205435 | Gene correction of CVS-654 mutation using CRISPR-Cas9 | CRISPR-Cas9 | - | Bioray Laboratories (Shanghai, China) | Phase 1/2 Completed |
| | NCT03728322 | Gene correction in induced HSCs using CRISPR-Cas9 | CRISPR-Cas9 | - | Allife Medical Science and Technology Co., Ltd. (Beijing, China) | Early phase 1 |

Epigenetic activation of $\gamma$-globin has been proposed [35], both with zinc finger combined with transcriptional activator VP64 [61,62] and dCas9 fused with acetylase (p300) [37]; however, their $\gamma$-globin reactivation efficiency is lower compared with genome editing approaches.

Direct correction of the SCD mutation in HBB has been attempted, leveraging the homology directed repair pathway. Different corrective templates have been used as DNA donors in combination with CRISPR/Cas9, including adeno associated virus (AAV) [63] or single-stranded oligodeoxynucleotides (ssODN) [64]. This approach was further developed [65] to initiate a clinical trial for SCD (NCT04819841) [66] (Table 1). However, Graphite Bio halted the trial in early 2023 after observing complications in the first patient dosed, including prolonged low blood cell counts requiring transfusion and growth factor support.

Based on similar methods, gene replacement approaches that rely on the integration of the $\beta$-globin gene into the $\alpha$-globin locus have been devised [67,68]. These studies successfully restored HbA expression in thalassemic and SCD erythroblasts. Such approaches might benefit patients with large deletions spanning the $\gamma$ - and $\beta$-globin locus where HbF reactivation is not a viable therapeutic option.

BEs cannot directly correct the A $>$ T mutation present in SCD. However, ABEs have been used to modify the SCD allele to a non-pathogenic $\beta$-globin variant ( $\beta 6^{\text {Glu }- \text { Ala }}, \mathrm{Hb}$ Makassar) [69,70]. An engineered ABE nuclease edited between $45 \%$ to $80 \%$ of the SCD alleles in patient HSPCs, resulting in up to $72 \%$ of Hb -Makassar production in differentiated erythroid cells [70]. The diverse mutational spectrum in $\beta$-thalassemia poses challenges to the development of corrective therapies. Hardouin et al. proposed a base editing approach to treat a common disease-causing point mutation in $H B B$ (IVS1-110 [G>A]) that efficiently rescued $\beta$-globin production [71]. Because of their precise editing activity, PEs can correct the SCD mutation [24]. Everette et al. delivered an optimized ePegRNA and PE3max as mRNA into SCD-HSPCs and achieved up to $41 \%$ of allele correction. Analysis of engrafted cells isolated 4 months after transplantation showed expression of HbA in $42 \%$ of human red cells [72]. Rather than relying on the ex vivo correction of HSPCs, Li et al. developed a novel in vivo modification strategy to correct the underlying SCD mutation. After HSCP mobilization, a helper-dependent adenoviral vector (HDAd) encoding the PE system was administrated intravenously in an SCD mouse model (CD46/Townes). Viral transduction corrected up to $40 \%$ of SCD alleles in vivo and increased HbA synthesis to similar levels [73]. While safety aspects of this delivery method need to be further explored, the direct modification of HSPCs in vivo eliminates the requirement of myeloablation/transplantation, a critical limitation for many gene editing therapies.

Overall, CRISPR/Cas9-derived tools have generated innovative and efficient treatments for $\beta$-hemoglobinopathies. These studies are paving the way for a broader application of this ground-breaking technology to many other genetic disorders.

# 2.2. Primary Immunodeficiencies 

Primary immunodeficiencies are rare diseases comprising more than 130 genetic disorders that affect the function and/or development of the immune system [74]. Some of these diseases, such as adenosine deaminase-severe combined immunodeficiency (ADA-SCID) and X linked-SCID (SCID-X1), have been successfully treated with viral gene therapy [75,76]. However, retroviral and lentiviral vectors carry the intrinsic risk of oncogenic insertional mutagenesis, and therefore alternative strategies are needed. Moreover, many genes involved in primary immunodeficiencies require a thoroughly regulated expression and would benefit from a gene editing approach. Because pathological mutations are rare and scattered along the gene body, most editing strategies for these disorders rely on the targeted integration of a corrective cDNA at the endogenous locus. For many immunodeficiencies, corrected cells have a selective advantage over mutated ones. Even if gene knock-in is not efficient in HSPCs, low correction levels could be sufficient to treat the disorder in patients. In this section, we will describe genome editing approaches developed for a subset of primary immunodeficiencies.

X-linked hyper-IgM syndrome (XHIM) is caused by mutations in the CD40 ligand (CD40L) gene that result in ineffective antibody production, recurrent infections, and autoimmunity in male infants. Ectopic expression of CD40L has been associated with lymphoma in mice [77], therefore lentiviral approaches would not be recommended for this disease. Proof-of-principle studies were performed using Cas9 and other nucleases to integrate CD40L cDNA under the control of its endogenous promoter in primary T cells [78,79] and in healthy HSPCs [80]. Importantly, CD40L activity was restored in gene-edited cells in vitro and in vivo, without signs of aberrant differentiation. Recently, large-scale good manufacturing practice (GMP)-compliant production processes have been developed, bringing this therapy closer to clinical application [81].

RAG1 is another gene involved in primary immunodeficiencies that requires tight regulation. Recombination activating genes (RAGs) are involved in V(D)J recombination and are required for correct lymphocyte development and function. A clinical trial using a lentiviral approach is currently underway (NCT04797260); however, constitutive and/or heterogenous expression of RAG1 raises safety concerns about potential immune dysregulation [82]. Using Cas9 and an AAV6 with a corrective cDNA, Castiello et al. restored RAG1 expression and activity in patient HSPCs, leading to the generation of mature T and B cells in transplanted mice [83].

Chronic granulomatous disease (CGD) is a group of genetic disorders in which neutrophils are unable to kill certain bacteria and fungi. The X-linked form of the disease (X-CGD) is caused by mutations in the CYBB gene. Different editing strategies were proposed to correct X-CGD in HSPCs, using nucleases with AAV6 vectors or ssODN as donor templates [84,85,86]. These studies indicated that preserving the first intron of $C Y B B$ is crucial to achieve a normal level of expression and restore effective ROS production [86]. Unfortunately, gene correction rates need to be extensively optimized to achieve therapeutic effects as corrected X-CGD cells do not have a selective advantage in vivo [87].

Most p47-CGD patients carry a specific deletion ( $\Delta \mathrm{GT}$ ) in exon 2 of the NCF1 gene, making this form of CGD a good candidate for gene editing therapies. Two studies have shown correction of the $\Delta \mathrm{GT}$ mutation using nucleases in combination with a DNA template [88,89], however efficiency must improve to reach therapeutic benefits in preclinical models.

Severe congenital neutropenia (SCN) is a rare inherited disease that impairs neutrophil maturation and is frequently caused by autosomal-dominant mutations in the ELANE gene. Production of an abnormal protein results in neutropenia, despite the presence of one intact ELANE allele. Induction of frameshift alleles with CRISPR/Cas9 can trigger nonsense mediated decay and potentially relieve neutrophil maturation arrest [90,91]. An allele-specific knockout strategy to precisely target the mutated copy of ELANE has been proposed. By targeting heterozygous sites of single-nucleotide polymorphisms (SNPs) associated with ELANE mutations, Sabo et al. were able to rescue neutrophil development and function [92].

CD3 $\delta$-SCID is caused by biallelic mutations in the autosomal CD3D gene that result in a profound deficiency of circulating mature T cells, often leading to infant mortality. CD3D is essential for the productive assembly of TCR complexes, playing an important role in T cell maturation [93]. In a recent study, McAuley et al. delivered ABE to CD3 $\delta$-SCID HSPCs to correct the disease-causing CD3D c.202C>T mutation [94]. High editing efficiency was achieved in these cells, together with functional correction and restoration of T cell maturation. Edited cells persisted 16 weeks in xenografts and displayed normal T cell receptor repertoires, making this approach a promising treatment for CD3 $\delta$-SCID.

Wiskott-Aldrich syndrome (WAS) is an X-linked disorder caused by mutations in the WAS gene that accounts for approximately 3\% of all primary immunodeficiencies [95]. Clinical manifestations depend on the underlying mutation, ranging from mild to severe persistent thrombocytopenia, autoimmunity, increased susceptibility to infections, or congenital neutropenia [96]. Lentiviral gene therapy has been successfully used for WAS [97,98], but bleeding and autoimmune episodes were only partially resolved by the treatment. Several knock-in strategies have been developed using different nucleases and DNA donors in pluripotent stem cells and cell lines [99,100]. Recently, Rai and al. restored WAS function in patient HSPCs by targeting WAS cDNA to its endogenous locus, using Cas9/gRNA and AAV6 [101]. A side-to-side comparison was also performed between genome edited and lentivirus transduced cells. Interestingly, transduced cells showed improved functionality but to a much lesser extent compared with gene edited cells. Limited correction of B-cells, platelets, and myeloid cells was observed in the lentiviral group, possibly due to suboptimal expression of WAS protein. In light of these promising results, preclinical studies are on the way to further assess the safety and efficacy of this therapeutic approach [102].

# 3. CRISPR-Based Gene Editing for Duchenne Muscular Dystrophy 

Similarly to blood diseases, the last decade witnessed the advent of CRISPR-base gene editing in neuromuscular disorders, notably in the Duchenne muscular dystrophy (DMD) field. From the rapid generation of helpful cellular and animal models to pre- and clinical development of innovative therapeutic options, CRISPR gene editing technology has revolutionized DMD research.

DMD is a fatal X-linked neuromuscular disorder characterized by severe and progressive muscle weakness and wasting due to degeneration of skeletal, smooth, and cardiac muscle [103]. Affecting more than 300.000 males worldwide, with a 1:5000 incidence in male newborns [104], DMD is the most prevalent genetic muscular disorder in humans. This orphan disease is caused by a variety of different mutations (mainly deletions ( $\approx 68 \%$ ), point mutations ( $\approx 20 \%$ ), and duplications ( $\approx 11 \%$ )) in the DMD gene (OMIM 300377, Xp21.2-p21.1) [105], known as the largest human gene ( 79 exons over 2.2 Mb of genomic DNA) (Figure 3A). The DMD gene encodes for dystrophin (Uniprot P11532) [106], an essential 427 kDa cytoplasmic protein critical for the maintenance of the biomechanical properties of fiber strength, flexibility, and stability in skeletal muscle [107]. Acting as a molecular shock absorber, the dystrophin protein establishes a mechanical link between the extracellular matrix and the actin cytoskeleton through the dystrophin-associated protein complex (DAPC) and allows myofibers to cope with repeated cycles of muscle contraction and relaxation. The absence of dystrophin destabilizes the DAPC and consequently weakens the link between cytoskeleton and extracellular matrix, triggering muscle cell dysfunction, muscle cell necrosis, and subsequent dystrophic tissue changes resulting in fibro-fatty replacement of skeletal muscle. Affected boys typically develop the first signs of motor dysfunction around the age of 2-3 years with abnormal gait, weakness in proximal muscles and calf muscles, pseudo hypertrophy, and difficulties in running, jumping, and climbing stairs. These symptoms progress relentlessly, loss of ambulation occurs around the age of 10-12 years [108], followed by progressive generalized muscle paralysis including respiratory insufficiency and cardiomyopathy [109]. Most patients decease in young adulthood, with a median survival of 28.1 years ( $95 \%$ CI 25.1, 30.3) [110], as a result of cardiac and respiratory failure. Despite exhaustive clinical attention for respiratory support, management of cardiac complications, and corticosteroid treatment, there is currently no cure for this devastating disease. The urgency to seek a cure for DMD has resulted in parallel efforts to develop exon skipping [111,112], termination codon read through [113] and dystrophin gene therapy [114], and non-dystrophin strategies [115]. Among all, anti-sense oligonucleotide (ASOs)-based exon skipping therapies aiming to restore a functional reading frame by skipping one or more exons to produce a functional dystrophin, are one of the most promising therapeutic options for DMD. Several exonskipping drugs were recently approved and, despite their success, new generation of ASOs with improved chemistries, distribution, and efficiency such as the peptide-conjugated phosphorodiamidate morpholino oligomer (P-PMO) [116] and Tricyclo-DNA (tcDNA) [117] are currently required and under investigation in clinical trials for DMD. Acting at the DNA level, CRISPR/Cas9 technology offers novel and promising opportunities in DMD gene therapy.

![img-2.jpeg](img-2.jpeg)

Figure 3. Dystrophin structure and CRISPR/Cas9 strategies for DMD. (A) Structural organization of the full-length wild-type dystrophin composed of 79 exons: the actin-binding N-terminal domain, the hinge domains (H1-H4), a rod domain consisting of 24 spectrin domains (R1-R24), and a cysteine-rich domain (CRD) next to a carboxy-terminal domain (CTD). The out-of-frame DMD $\Delta 52$ mutation (deletion of exon 52) is one of the most common mutations in DMD patients. (B) CRISPR/Cas9 mediated therapeutic strategies for DMD: NHEJ method (exon deletion, exon skipping, exon reframing), HDR method (exon knock-in), base editing, prime editing, and CRISPR activator.

### 3.1. CRISPR/Cas9 and Generation of DMD Study Models

To study the pathophysiology of the disease and to evaluate the efficacy of therapies prior to clinical trials, cellular and animal models are mandatory. Only few immortalized patient-derived muscle cell lines are currently available [118] as DMD muscle biopsies are difficult to obtain. CRISPR/Cas9 technology allowed researchers to directly generate human DMD lines with specific genetic mutations (Table 2). In the past years, several novel immortalized DMD muscle cell lines were generated with the deletion of exons 50 [119], 51 [119], 52 [119,120], and 53 [119]. Recently, Tremblay's lab produced DMD lines with specific point mutations in exons 9, 20, 35, 43, 55, and 61 using prime editing [121]. These cell lines and associated subsequent assays [120,122,123] represent powerful tools to evaluate and optimize therapeutic strategies, screen for effective drugs, and gain insights in the disease pathophysiology.

Similarly, CRISPR/Cas9 offered a simple and efficient method to generate new animal models of DMD (Table 2). The *mdx* mice is the most widely used DMD model and presents a spontaneous point mutation in exon 23 of the *Dmd* gene, resulting in a premature stop codon [124]. However, the *mdx* mouse develops a moderate and nonprogressive myopathy, with a slightly shorter lifespan, and fails to recapitulate many symptoms observed in DMD patients. Several murine models carrying specific exonic deletions were generated using CRISPR/Cas9, such as ΔEx52 [125], ΔEx50 [126], ΔEx45 [125], ΔEx44 [127], ΔEx43 [125], or the large deletion ΔEx8-34 [128]. Murine DMD models suffering from a non-sense mutation in the *Dmd* exon 20 [129] and frameshift mutation [130] were also produced by CRISPR and base editing. Importantly, all these strains phenocopied the *mdx* mice and suffered from the same limitations. Moreover, these DMD models are of limited use to test human sequence-dependent therapies such as exon skipping and gene editing approaches. Consequently, humanized DMD mouse strains (hDMD mouse) such as the hDMDΔ52/*mdx* [131] and the hDMDΔ45/*mdx* [132] were produced. Partially humanized mice were similarly generated, replacing exon 45 and its adjacent introns with the human sequences ((hEx45KI-mdx44) [133]. Although splicing mechanisms might be different, these CRISPR-generated DMD models provide unvaluable in vivo platforms to assess human sequence-dependent exon 44, 45, and 51 skipping drugs and gene editing approaches.

Rat models of DMD recapitulate more faithfully the human DMD pathophysiology, allow behavioral experiments with high statistical power, and may represent a useful alternative to mice. The first Dmd^{*mdx*} rat was generated by TALEN [134] and suffered from a severe muscle and cardiac pathology, reflecting some lesions and functional abnormalities observed in DMD patients. Using a two sgRNA pairs approach, the Relaix's lab successfully generated the R-DMDdel52 rat model, with a premature stop codon in exon 52 of the rat Dmd gene [135]. Nakamura et al. also reported the generation of a Dmd ΔEx3-16 rat using SpCas9 and two sgRNA [136], and a Dmd ΔEx45 rat model was recently produced [137]. In the near future, humanized DMD rat models may certainly serve preclinical drug development.

With an intermediate size and low maintenance costs, rabbit is another interesting animal model of DMD. The first DMD rabbit was produced by co-injecting SpCas9 mRNA and two gRNAs targeting the DMD exon 51 into rabbit zygotes [138]. This DMD ΔEx51 rabbit model suffered from reduced dystrophin expression, histological defects, impaired physical activity, cardiomyopathy, and premature death. At the crossroads between small and large models, this novel rabbit DMD model could be valuable for preclinical studies.

In addition to rodent models, larger animal models of DMD such as pig, dog, and monkey have also been produced using CRISPR systems. Porcine models have a number of practical advantages and are phenotypically and phylogenetically closer to humans. Several porcine models of DMD, such as the DMD ΔEx52 pig [139], the DMD ΔEx27 Chinese Diannan pig [140], or the DMD mEx13 [141], have been described and exhibited a severe muscular dystrophy similar to human DMD but suffered from a faster disease progression. DMD pigs are an attractive in vivo animal model, but their long gestational period, costly maintenance, and reduced survival makes their preclinical use challenging.

The golden retriever muscular dystrophy (GRMD) dog [142] and several other breeds [143,144] suffered from spontaneous DMD genetic defects and closely mimicked the human disease [145]. Oh et al. reported the generation of a DMD dog with a 57 bp deletion in exon 6 by nuclear transfer using CRISPR/Cas9-edited somatic cells [146]. While expensive and limited by the low number of phenotypically variable individuals, DMD dogs represent a useful large animal model for preclinical studies.

Monkeys have mostly been used for safety evaluation and pharmacological studies in the DMD field. Recently, the first DMD rhesus monkeys (Macaca mulatta) model with various InDel mutations in exon 4 and 46 was generated [147]. These animals showed dystrophin reduction and some histological defects. To date, this study is the only published preliminary assessment of a DMD monkey model.

The ideal animal model should display a robust phenotype over generations, have the same genetic defect, and mimic key hallmarks and progression of the human pathology. None of the 60 animal models of DMD previously described fulfilled these requirements. Each animal model presents key advantages and limitations and should be used for specific applications. As an example, murine models are indicated for in vivo drug screening, while dogs and monkeys are appropriate for assessing immune responses and toxicology. CRISPR technologies have facilitated the generation of novel animal models of DMD, speeding up preclinical development of therapeutic solutions and their translation to DMD patients.

Table 2. CRISPR-mediated animal models of DMD.

|  | Model | DMD <br> Muta- <br> tion | Target | Nuclease | Advantages |  | Limitations | Reference |
| :--: | :--: | :--: | :--: | :--: | :--: | :--: | :--: | :--: |
| $\stackrel{\rightharpoonup}{\infty}$ | human immortalized myoblasts cell line | $\Delta \mathrm{Ex} 50$ | 149 and 150 | SpCas9 | - | Low cost | - | [119] |
|  |  | $\Delta \mathrm{Ex} 51$ | 150 and 151 |  | - | Easily maintainable | - | Cell type number limited |
|  |  | $\Delta \mathrm{Ex} 52$ | 151 and 152 |  | - | Wide number of mutations | - | Does not recapitulate the human disease |
|  |  | $\Delta \mathrm{Ex} 53$ | 152 and 153 |  | - | Human genetic context |  |  |
|  | Mouse | Ex20 | Ex20 | BE3 | - | High high-throughput screening | - | [119] |
|  |  | Ex23 | Ex23 | CjCas9 | - | Low cost |  | [129] |
|  |  | $\Delta \mathrm{Ex} 43$ | 142 and 143 |  | - | Easily maintainable |  | [130] |
|  |  | $\Delta \mathrm{Ex} 44$ | 143 and 144 |  | - | Wide number of mutations | - | [125] |
|  |  | $\Delta \mathrm{Ex} 45$ | 144 and 145 | SpCas9 | - | High statistical power | - | [127] |
|  |  | $\Delta \mathrm{Ex} 5$ | 149 and 150 |  | - | Extensive data/littérature | - | [125] |
|  |  | $\Delta \mathrm{Ex} 5$ | 17 and 134 |  | - | Accepted protocols |  | [128] |
|  | Humanized Mouse | $\Delta \mathrm{Ex} 45$ | 144 and 145 |  | - | (TREAT-NMD). |  | [132,133] |
|  |  | $\Delta \mathrm{Ex} 52$ | 151 and 152 |  |  |  |  | [131] |
|  | Rat | $\Delta \mathrm{Ex} 52$ | 151 and 152 |  | - | Eeasily maintainable |  | [135] |
|  |  |  |  |  | - | Severe skeletal/cardiac muscle | - | Higher cost than mice ( $5 \times$ ) |
|  |  | $\Delta \mathrm{Ex} 3-$ | Ex3 and Ex6 | SpCas9 | - | phenotype | - |  |
|  |  | 16 |  |  | - | Allow behavioural experiments |  | [136] |
| Animal | Rabbit | Ex51 | Ex51 | SpCas9 | - | High statistical power | - | [138] |
|  | Rabbit | Ex51 | Ex51 | SpCas9 | - | Severe muscle dystrophy | - | Expensive to maintain |
|  |  |  |  |  | - | Severe cardiomyopathy | - | Lack of accepted protocols |
|  | Pig | Ex13 | Ex13 | $\begin{gathered} \mathrm{BE} 3, \\ \mathrm{hA3A-} \\ \mathrm{BE} 3 \end{gathered}$ | - | Severe muscular dystrophy Anatomy, size and physiology closer to humans | - | Expensive to maintain <br> Long gestational period <br> Accelerated dystrophy compared to human <br> Stress may induced sudden death <br> Lack of accepted protocols |
|  |  | Ex27 | Ex27 |  | - |  | - | [140] |
|  |  | Ex52 | Ex52 |  | - |  | - | [139] |
|  | Dog | Ex6 | Ex6 | SpCas9 | - | Severe muscular dystrophy Severe cardiomyopathy | - | Expensive to maintain |
|  |  |  |  |  | - | Anatomy, size and physiology closer to humans | High variability <br> Low statistical power | [146] |
|  | Monkey | Ex4 or Ex46 | Ex4 or Ex46 | SpCas9 | - | Greatest similarity to humans | - | Expensive to maintain <br> Long gestational period <br> Recent model, lack of accepted protocols | [147] |

# 3.2. CRISPR-Based Therapies: From Preclinical to Clinical Trials for DMD 

The simplicity, versatility, and precision of CRISPR-/Cas9 offer novel and promising therapeutic opportunities for DMD. Acting at the DNA level, CRISPR/Cas9 has the potential to permanently repair the genetic cause of DMD. Here, we report the most advanced CRISPR/Cas9 therapeutic interventions to correct and restore dystrophin expression, with a focus on preclinical in vivo studies (Figure 3B).

### 3.3. NHEJ-Mediated Exon Deletion

More than $60 \%$ of DMD patients suffer from exon deletion and $11 \%$ of all DMD cases are due to exon duplications [104,148]. Depending on the genetic defect, deletion of one or more exons restores the reading frame, resulting in a shorter but functional dystrophin protein. Many in vitro [149,150] and in vivo studies have demonstrated that the CRISPR/Cas9 system and appropriate pairs of sgRNA can be used to induce exon(s)/intron(s) excision and restore the DMD reading frame (Figure 3B). In 2016, three independent studies demonstrated in neonate and adult $m d x$ mice that SpCas9 or SaCas9 and dual sgRNA treatment delivered by rAAV8, 9 could efficiently delete the mutated exon 23, restore dystrophin expression and consequently improve muscle morphology, and enhance skeletal and cardiac muscle function [151,152,153]. The seminal studies provided evidence for permanent and efficient gene correction in dystrophic animals. Importantly, such an approach can be sustained for up to 19 months in murine animals without inducing serious adverse effects [154,155,156]. Correction of other DMD genetic defects, including deletions $\Delta \mathrm{Ex} 45$ [132], $\Delta \mathrm{Ex} 52$ [157], and $\Delta \mathrm{Ex} 52-53$ [158,159] or the duplication of Ex18-30 [160], demonstrated the potential of this CRISPR-based approach to eliminate mutated exon, point mutation, and exonic duplication in different animal models of DMD, notably in large animal such as pigs [161] and dogs [162], paving the way for clinical translation. Deletions and duplications cluster in two main hotspots in the DMD gene, located at exons $45-55$ and 3-9. Following the initial work from Ousterout et al. and the successful transplantation of CRISPR/Cas9-corrected $\Delta 45-55$ patient cells into immunodeficient mice [163], Young et al. generated a large $45-55$ exonic deletion in an hDMD $\Delta \mathrm{Ex} 45 / m d x$ mice model [132]. Treatment resulted in the expression of a truncated $\Delta 45-55$ dystrophin in muscle animal, demonstrating the clinical relance of the CRISPR/Cas9 platform. Similarly, Duchêne et al. demonstrated the restoration of the DMD $\Delta \mathrm{Ex} 47-58$ reading frame and in vivo dystrophin expression in hDMD $\Delta \mathrm{Ex} 52 / m d x$ mice after AAV9 administration delivering SaCas9 and 2 sgRNA targeting the exon 47 and 58 [157]. Whereas such innovative gene editing strategy is applicable to $\sim 60 \%$ of DMD patients, it should be noted that efficiency is correlated with deletion size and that multi-exon deletions generate a shorter and potentially less functional dystrophin.

### 3.4. NHEJ-Mediated Exon Skipping

More than $70 \%$ of the DMD population is amendable to exon-skipping strategies aiming to excise one specific exon to restore a functional open reading frame and produce a truncated but functional dystrophin protein [164]. Exon-skipping approaches are mutationdependent and specific for subsets of the DMD patient population [165]. Disruption of splicing donor or acceptor sites with a single gRNA and CRISPR/Cas9 introduces InDels, abolishes splice element function, and leads to the skip of the targeted exon (Figure 3B). Several in vitro studies in human DMD myoblasts and human iPSCs reported efficient exon 43, 51, and 53 skipping using this approach [166]. In animal models of DMD, Olson's lab induced skipping of exon 51 in neonate $\Delta \mathrm{Ex} 50 / m d x$ mice using a dual AAV9 approach to deliver SpCas9 and three copies of the same gRNA targeting a region adjacent to the exon 51 splice acceptor site [126]. Systemic treatment conducted a widespread dystrophin expression in skeletal, respiratory, and cardiac muscles, resulting in histological correction of the dystrophic defects and significant increased muscle strength. Similar results were obtained in a humanized $\Delta \mathrm{Ex} 50 / m d x$ model [167]. The same group reported similar results in $\Delta \mathrm{Ex} 50-\mathrm{MD}$ dogs, with a restoration of dystrophin levels ranging from 3 to $90 \%$ of normal,

reaching $92 \%$ in the heart [162]. Correction of other exon deletions such as $\Delta \mathrm{Ex} 43, \Delta \mathrm{Ex} 44$, $\Delta \mathrm{Ex} 45$, and $\Delta \mathrm{Ex} 53$ were also studied in animal models [125,127,168,169]. Of interest, these studies demonstrated that the amount of sgRNA delivered to the muscle was critical for efficient editing in vivo [127] and provided evidence for the long-term benefits of single-cut gene editing therapy, even under conditions of stress [168]. Compared with CRISPRmediated exon deletion using two sgRNA, single-cut exon skipping is a safer approach as it reduces potential off-target effects and DNA damage. Importantly, such exon-skipping strategies may also result in exon reframing.

# 3.5. NHEJ-Mediated Exon Reframing 

Similarly to single-cut exon skipping, the exon reframing approach relies on the use of a single gRNA targeting a frameshift mutation and the subsequent generation of small InDels by NEHJ to restore a functional reading frame. In theory, small indels generated upstream of a deleterious mutation have a 1 in 3 probability of restoring the reading frame. If successful, this approach will restore a larger dystrophin protein compared with exon skipping (Figure 3B). Several in vitro studies [163,170] demonstrated the efficacy of this CRISPR strategy. Positive results were observed in vivo in $\Delta \mathrm{Ex} 50 / \mathrm{mdx}$ mice after AAV9 delivery of CRISPR components to restore the open reading frame of exon 51. Muscle function improved toward wildtype levels even with low gene editing efficiency ( $8-12 \%$ ), resulting in $27-45 \%$ of dystrophin expression in skeletal, respiratory, and cardiac muscles [171]. This approach was applied to several $m d x$ mice, including mEx23 [130], $\Delta \mathrm{Ex} 50$ [126], h $\Delta \mathrm{Ex} 43$ [125], h $\Delta \mathrm{Ex} 44$ [127], h $\Delta \mathrm{Ex} 45$ [125], and h $\Delta \mathrm{Ex} 52$ [125] $m d x$ mice. Interestingly, Min et al. compared both exon skipping and reframing strategies for three DMD mutations and concluded that exon reframing was more efficient than exon skipping in restoring dystrophin [125].

### 3.6. HDR-Mediated Gene Correction

Unlike NHEJ-based strategies, HDR-mediated gene editing has the potential to restore the full-length dystrophin and to correct mutations in the N - and C -terminal region of the protein (Figure 3B). The first in vivo proof-of-concept of this approach was provided in 2014 [172] by zygotic injection of SpCas9/sgRNA and an exogenous DNA template in $m d x$ mouse embryos. This seminal work demonstrated that $\sim 17 \%$ of gene editing was sufficient to restore dystrophin expression to wild-type levels. Consequently, several investigations have attempted to correct mutations in exons 23 [173,174], 44 [170], and 53 [158] of the Dmd gene using similar knock-in approaches. In $m d x^{4 c v}$, intramuscular injection of an AAV6 delivering Cas9/sgRNA and a donor template rescued dystrophin production to $1.8-8.4 \%$ of the wild-type level following a $0.18 \%$ gene editing efficiency [158]. In a GRMD dog model, intramuscular injection of Cas9 components and a corrective ssODN restored dystrophin expression to $6-16 \%$ of normal levels [175]. The NHEJ-based knock-in homology-independent targeted integration (HITI) approach has also been leveraged to correct genetic defects [176,177]. By including two Cas9 cleavage sites flanking the donor sequence, HITI allows efficient integration of a DNA template at the targeted genomic site. Pickar-Oliver et al. successfully knocked-in exon 52 and super-exon 52-79 in an hDMD $\Delta 52$ mouse model [131]. Despite the low efficiency, these results demonstrated that HITI can restore the dystrophin protein in vivo. While the accuracy of HRD- and HITI-based strategies is appealing, the major limitation of these approaches is their low efficiency. Additionally, inverted/unwanted integration of the DNA donor may occur, resulting in potential alteration of dystrophin expression. Extensive optimization and safety studies will be required to advance these therapies to the clinic.

### 3.7. Base Editing

Beyond traditional NHEJ- and HDR-based CRISPR/Cas9 systems, generation of catalytically impaired Cas9 (D10A) fused to a base-modifying enzyme introduced the first iteration of a more precise and safer gene editor [23] (Figure 2B). Cytosine base editors (CBEs) [19] and adenine base editors (ABEs) [18] respectively allow C:G to T:A and A:T to

G:C base pair transitions without DSBs, DNA donor template, or HDR, offering a promising editing avenue to precisely correct single point and non-sense mutations (Figures 1 and 2), accounting for $\sim 27 \%$ of the DMD cases [178]. Described as safer than CBEs due to a higher specificity and lower off-target activity [179,180], several ABE systems were successfully used in different mouse models of DMD (Figure 3B). Ryu et al. used trans-splicing to deliver a full ABE with two AAV9s to correct a point mutation generating a premature stop codon in exon 20 of a murine Dmd gene [181]. Eight weeks post infusion, dystrophin expression was detected in $17 \%$ of the skeletal myofibers. Similarly, Hu and colleagues developed a single AAV9-iABE-NGA systemic treatment designed to correct a non-sense mutation in exon 53 of the $m d x^{4 c v}$ mouse [182]. A widespread restoration of dystrophin expression and subsequent histopathological and functional benefits were noted 10 weeks post-injection. Importantly, therapeutic benefits were maintained 10 months after injection, with a near complete rescue of dystrophin in the heart and $15 \%$ rescue in skeletal muscle fibers. By altering splice sites, BEs can promote exon skipping and correct mutations and deletions in the DMD gene. Systemic administration of AAV9s delivering CBE and sgRNA designed to skip exon 4 in $\mathrm{Dmd}^{\mathrm{E} 4}$ mice resulted in efficient dystrophin restoration and rescue of muscular dystrophy phenotype [183]. Similar results were obtained by Chemello and al. following intramuscular injection of dual AAV9s encoding ABE components as a split-intein trans-splicing system promoting exon-50 skipping in $\Delta \mathrm{Ex} 51$ mice [184].

While these preclinical studies demonstrate feasibility and efficacy of BEs for many DMD mutations, delivery remains a major challenge. Due to their large size, BEs require dual-vector delivery and increased viral doses associated with immunological and toxicological concerns. A novel optimized all-in one BE solution able to fit into a single AAV [185] may lead to further the preclinical development of these innovative therapies for DMD.

# 3.8. Prime Editing 

The versatility of prime editing has opened up an attractive horizon for the treatment of various genetic defects such as DMD [186] (Figures 2C and 3B). Olson's lab applied first prime editing to $\Delta \mathrm{Ex} 51$ DMD patient-derived iPSCs to reframe exon 52 and successfully restored dystrophin expression to $24.8-39.7 \%$ of healthy levels [184]. Similarly, Tremblay's lab used PE to induce substitutions, insertions, and deletions in the splice donors for exons 51 and 53 to correct frameshift mutations in the DMD gene carrying exon 52 or exons 45 to 52 deletions [187] and to correct point mutation in the DMD gene [121,188]. Recently, Anzalone et al. demonstrated that twin PE systems can precisely excise exon 51 in the dystrophin gene in vitro [28], paving the way for an innovative gene editing solution for DMD.

While this technology has the potential to correct a large variety of DMD mutations, further in vivo applications are required to progress and translate this approach to patients and, similarly to base editing, size constraints remain a challenge to be solved for in vivo delivery.

### 3.9. CRISPR/Cas9 Activation

In addition to enabling the engineering of eukaryotic genomes, recent alterations to the CRISPR/Cas9 system have provided opportunities for regulating gene expression [189] (Figure 2D). Modulation of disease-modifier genes can provide therapeutic solutions applicable to all DMD patients irrespective of their dystrophin mutation [115,190]. Utrophin (UTRN), an autosomal and functional paralogue of dystrophin expressed at the sarcolemma of regenerated myofibers [191,192], has the potential to compensate for the primary defect in DMD [193]. Seminal preclinical studies demonstrate that expression of UTRN suppresses functional signs of dystrophinopathy in a dose dependent manner [194] without toxicity [195], and strongly support UTRN as a functional surrogate for dystrophin [190,196]. Using CRISPRa (dCas9-VP160) targeted to the UTRN promoter, Wojtal et al. successfully modulated UTRN expression in DMD myoblasts [197]. Liao and colleague demonstrated a three- four-fold in vivo increase in UTRN levels in $m d x$ mice using a different CRISPRa

system [198] (Figure 3B). While these preclinical studies emphasize the potential of CRISPRbased UTRN upregulation for DMD, such approaches require long-term efficacy and toxicology in vivo assessment.

Recently, Cure Rare Disease, Inc. launched the first CRISPR-based clinical trial for DMD (NCT05514249) [199]. The n-of-1 clinical trial involved a single 27-year-old DMD patient suffering from an exon 1 deletion, lacking the dystrophin muscle isoform (Dp427m). Phase 1 aimed to assess the safety and preliminary efficacy of CRD-TMH-001, an AAV9dCas9-VP64 delivered intravenously at a $1 \times 10^{14} \mathrm{vg} / \mathrm{kg}$ dose and designed to upregulate cortical dystrophin through the conserved Dp427p promoter. Six days after receiving the therapy, the patient showed signs of mild cardiac dysfunction and pericardial effusion, leading to cardiac arrest two days later. Post-mortem examination revealed severe acute-respiratory distress syndrome (ARDS) with diffuse alveolar damage. Investigators concluded that the high dose of AAV triggered the adverse event and not the CRISPR therapy itself. Despite remarkable success in gene therapy, lethal immunotoxicity in highdose systemic AAV therapy was previously reported in treated DMD, SMA, and XLMTM patients [200]. This first reported case of severe ARDS linked to an AAV product provided a painful but valuable lesson that will help de-risk the translation of gene therapies for future patients and remind us of the challenges we still need to overcome.

# 4. From the Same Technology to Different Stages: A Comparison of CRISPR-Based Strategies for Blood and Muscle Fields 

While CRISPR/Cas9 strategies for blood disorders rapidly advanced from early in vitro studies to clinical trials, gene editing for neuromuscular diseases has not yet reached similar milestones. Clinical translation of the same technology may progress differently across medical fields because of disease-specific limitations.

While hematopoietic stem cells can be purified, corrected ex vivo, and re-infused into the patient [201], CRISPR/Cas9 components need to be systemically delivered in order to reach the muscles. For the treatment to be effective, AAV vectors carrying the editing machinery must transduce skeletal, respiratory, and cardiac muscle tissues specifically and efficiently. This requires the choice of adequate serotypes and appropriate dosing, balancing efficacy considerations and risks associated with systemic AAV vector delivery [202]. From the administration route to vector manufacturing, neuromuscular diseases currently face a higher level of complexity compared with blood disorders, explaining why CRISPR/Cas9 therapies have not yet reached the same clinical stages. While extensive optimization efforts are currently ongoing to address such challenges [202], it should be noted that DMD will greatly benefit from the clinical success of CRISPR in blood disorders and future long-term follow-up studies.

A major bottleneck to clinical translation for novel DMD therapies is the lack of suitable animal models. The widely used $m d x$ mouse [124] does not recapitulate the human disease and does not offer the appropriate genetic context for sequence-based approaches, such as exon skipping, strop codon read through, and CRISPR/Cas9 strategies. CRISPR has been an incredible tool to produce novel humanized animal models for DMD, speeding up the translation of sequence-based approaches to patients.

## 5. Conclusions: Challenges and Current Trends

In recent years, CRISPR has emerged as a revolutionary technology with the potential to permanently and precisely edit any DNA locus and correct genetic errors. From CRISPRCas to base and prime editors, the CRISPR tool box has generated innovative preclinical models and corrective strategies for many genetic disorders, bringing more breakthrough therapies closer to the patients. The last decade has witnessed the rapid development of this technology for monogenic blood disorders, from the early proof-of principle studies to the first regulatory approval of a CRISPR-based therapy for SCD and TDT. To date, there are currently 94 clinical trials for hereditary and non-hereditary diseases based on CRISPR technologies (https://clinicaltrials.gov). Nevertheless, as illustrated by blood

and neuromuscular disorders, there are still many hurdles to overcome to expand clinical applications of gene editing therapies, including long-term safety and off-target activity and also cost, accessibility, and manufacturing. Some concerns are specific to the targeted organ system.

Despite the success of Casgevy in SCD and TDT and many promising results from undergoing clinical trials, we still do not know the long-term efficacy and safety of these treatments. While effective at reducing vaso-occlusive events and transfusion dependency in SCD, we need long-term follow-up studies to establish whether Casgevy can also decrease disease burden in other organs like heart, lung, and kidney. These studies will also evaluate the persistence of high HbF expression over time, a parameter that is particularly critical for SCD patients, as levels below $20 \%$ are not considered protective. HSPCs constantly replenish all blood cell types through repeated cell divisions and differentiation steps. By editing HSPCs, the DNA modifications will be inherited by all the HSPC-derived blood lineages. DNA damage and DNA DSBs can lead to genotoxic effects [13-16] and loss of self-renewal in HSPCs. Moreover, unintended modifications can occur when similar DNA sequences are targeted, leading to the generation of off-target mutations that can potentially have deleterious consequences or provide selective advantages to specific blood lineages. The progressive transition to DSB-free CRISPR systems such as base and prime editing and the development of high-fidelity Cas enzymes with lower off-target activity [203] will help reduce the mutational risk. So far, no evidence of DSB-associated genotoxicity has been observed in patients receiving Casgevy. It might still be too early to rule out long-term genotoxicity, as two patients who underwent gene therapy for SCD developed blood malignancies 3-5.5 years post-transplant [204]. Clonal expansion has been observed in the patient who participated in the Graphite Bio trial; however, the clinical significance of this event remains unclear. Developing ways to reduce, detect, and predict genotoxicity of CRISPR-based therapies remains essential to safe clinical implementation. Most CRISPR-based therapies for genetic blood disorder require HSPC mobilization and myeloablation, a procedure associated with serious complications. Ex vivo genetic modification and transplantation of HSPCs is costly and can only be performed by specialized treatment centers with highly trained personnel, allowing a small portion of eligible patients to be treated each year. Moreover, since Casgevy could cost as much as USD 2 million per treatment, it would be practically inaccessible to low-income countries where the majority of TDT and SCD patients live [205].

Neuromuscular disorders cannot be corrected ex vivo and therefore rely on systemic viral delivery of the editing machinery, raising immunogenicity concerns both for the high AAV doses required and the long-term expression of CRISPR components [166]. Despite high transduction efficiency, durable therapeutic benefits, and remarkable clinical successes, the AAV platform suffers from several limitations. First, the limited cargo capacity limits the delivery options of CRISPR/Cas9 components, too large to be expressed from a single AAV vector. While dual-AAV systems are often used, all-in-one constructs [171] with truncated Cas9 nickase represent attractive solutions that are currently under investigation in vivo. The high dose required for therapeutic use has been previously associated with immunotoxicity in AAV-treated patients [200], and is a hurdle to manufacturing. Pre-existing immunity to AAVs and CRISPR components might limit access to therapies and will require careful monitoring and intervention pre- and post-infusion [206]. Optimized forms of Cas9 with modified epitopes hold great promise in reducing host immunogenicity [207]. In addition, the development of innovative AAV serotypes offering greater efficiency at lower doses [205] may overcome, in part, these challenges. Non-viral solutions such as gold and lipid nanoparticles that deliver mRNA to HSPC [203] and other tissues in vivo [204] may also play an important role in delivering CRISPR/Cas systems in the future. Alongside all these considerations, the durability of CRISPR/Cas9 treatment is pivotal for DMD and other muscle disorders. The efficient delivery of CRISPR/Cas9 elements to satellite cells may ensure long-lasting therapeutic benefits in growing patients. Whereas preclinical studies showed that systemic AAV CRISPR delivery can target satellite cells and editing persist

up to 18 months, efficiency needs to be improved and further optimization is required to target this cell population.

The last decade witnessed the great potential of CRISPR/Cas9 systems, however extensive preclinical and clinical investigations are still needed to overcome several therapeutic challenges. Lessons learned from these studies and future CRISPR-based research in the hematological and neuromuscular fields will further the development of novel therapies for other devastating genetic diseases.

Author Contributions: Conceptualization, G.P. and S.G.; Methodology, M.L., M.G., G.P. and S.G.; Investigation, M.L., M.G., G.P. and S.G.; Data curation, M.L., M.G., G.P. and S.G; Writing—original draft preparation, G.P. and S.G.; Writing—review and editing, M.L., M.G., G.P. and S.G.; visualization, M.L., M.G., G.P. and S.G.; Supervision, G.P. and S.G.; Project administration, S.G.; Funding acquisition, G.P. and S.G. All authors have read and agreed to the published version of the manuscript.

Funding: M.G. and S.G. are funded by SQY Therapeutics. M.L. is funded by Genethon. G.P. is funded by Children's Hospital of Philadelphia (CHOP) and the CHOP Cell and Gene Therapy Seed Grant.

Conflicts of Interest: Authors Marine Geoffroy and Simon Guiraud were employed by the company SQY Therapeutics. The remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. The SQY Therapeutics had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.

# References 

1. Wang, J.Y.; Doudna, J.A. Crispr technology: A decade of genome editing is only the beginning. Science 2023, 379, eadd8643. [CrossRef]
2. Gaj, T.; Gersbach, C.A.; Barbas, C.F., 3rd. Zfn, talen, and crispr/cas-based methods for genome engineering. Trends Biotechnol. 2013, 31, 397-405. [CrossRef]
3. Ishino, Y.; Shinagawa, H.; Makino, K.; Amemura, M.; Nakata, A. Nucleotide sequence of the iap gene, responsible for alkaline phosphatase isozyme conversion in escherichia coli, and identification of the gene product. J. Bacteriol. 1987, 169, 5429-5433. [CrossRef]
4. Mojica, F.J.; Juez, G.; Rodriguez-Valera, F. Transcription at different salinities of haloferax mediterranei sequences adjacent to partially modified psti sites. Mol. Microbiol. 1993, 9, 613-621. [CrossRef]
5. Bolotin, A.; Quinquis, B.; Sorokin, A.; Ehrlich, S.D. Clustered regularly interspaced short palindrome repeats (crisprs) have spacers of extrachromosomal origin. Microbiology 2005, 151, 2551-2561. [CrossRef]
6. Deltcheva, E.; Chylinski, K.; Sharma, C.M.; Gonzales, K.; Chao, Y.; Pirzada, Z.A.; Eckert, M.R.; Vogel, J.; Charpentier, E. CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III. Nature 2011, 471, 602-607. [CrossRef]
7. Mojica, F.J.; Diez-Villasenor, C.; Garcia-Martinez, J.; Soria, E. Intervening sequences of regularly spaced prokaryotic repeats derive from foreign genetic elements. J. Mol. Evol. 2005, 60, 174-182. [CrossRef]
8. Gasiunas, G.; Barrangou, R.; Horvath, P.; Siksnys, V. Cas9-crrna ribonucleoprotein complex mediates specific DNA cleavage for adaptive immunity in bacteria. Proc. Natl. Acad. Sci. USA 2012, 109, E2579-E2586. [CrossRef]
9. Jinek, M.; Chylinski, K.; Fonfara, I.; Hauer, M.; Doudna, J.A.; Charpentier, E. A programmable dual-rna-guided DNA endonuclease in adaptive bacterial immunity. Science 2012, 337, 816-821. [CrossRef]
10. Cong, L.; Ran, F.A.; Cox, D.; Lin, S.; Barretto, R.; Habib, N.; Hsu, P.D.; Wu, X.; Jiang, W.; Marraffini, L.A.; et al. Multiplex genome engineering using crispr/cas systems. Science 2013, 339, 819-823. [CrossRef]
11. Kanaar, R.; Wyman, C.; Rothstein, R. Quality control of DNA break metabolism: In the 'end', it's a good thing. EMBO J. 2008, 27, 581-588. [CrossRef]
12. Baylis, F.; McLeod, M. First-in-human phase 1 crispr gene editing cancer trials: Are we ready? Curr. Gene Ther. 2017, 17, 309-319. [CrossRef]
13. Brunner, E.; Yagi, R.; Debrunner, M.; Beck-Schneider, D.; Burger, A.; Escher, E.; Mosimann, C.; Hausmann, G.; Basler, K. Crispr-induced double-strand breaks trigger recombination between homologous chromosome arms. Life Sci. Alliance 2019, 2, e201800267. [CrossRef]
14. Cannan, W.J.; Pederson, D.S. Mechanisms and consequences of double-strand DNA break formation in chromatin. J. Cell Physiol. 2016, 231, 3-14. [CrossRef]
15. Schipler, A.; Mladenova, V.; Soni, A.; Nikolov, V.; Saha, J.; Mladenov, E.; Iliakis, G. Chromosome thripsis by DNA double strand break clusters causes enhanced cell lethality, chromosomal translocations and 53bp1-recruitment. Nucleic Acids Res. 2016, 44, 7673-7690. [CrossRef]
16. Leibowitz, M.L.; Papathanasiou, S.; Doerfler, P.A.; Blaine, L.J.; Sun, L.; Yao, Y.; Zhang, C.Z.; Weiss, M.J.; Pellman, D. Chromothripsis as an on-target consequence of crispr-cas9 genome editing. Nat. Genet. 2021, 53, 895-905. [CrossRef]

17. Rees, H.A.; Minella, A.C.; Burnett, C.A.; Komor, A.C.; Gaudelli, N.M. Crispr-derived genome editing therapies: Progress from bench to bedside. Mol. Ther. 2021, 29, 3125-3139. [CrossRef]
18. Gaudelli, N.M.; Komor, A.C.; Rees, H.A.; Packer, M.S.; Badran, A.H.; Bryson, D.I.; Liu, D.R. Programmable base editing of a*t to g*c in genomic DNA without DNA cleavage. Nature 2017, 551, 464-471. [CrossRef]
19. Komor, A.C.; Kim, Y.B.; Packer, M.S.; Zuris, J.A.; Liu, D.R. Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature 2016, 533, 420-424. [CrossRef]
20. Walton, R.T.; Christie, K.A.; Whittaker, M.N.; Kleinstiver, B.P. Unconstrained genome targeting with near-pamless engineered crispr-cas9 variants. Science 2020, 368, 290-296. [CrossRef]
21. Zhao, D.; Li, J.; Li, S.; Xin, X.; Hu, M.; Price, M.A.; Rosser, S.J.; Bi, C.; Zhang, X. Glycosylase base editors enable c-to-a and c-to-g base changes. Nat. Biotechnol. 2021, 39, 35-40. [CrossRef] [PubMed]
22. Kurt, I.C.; Zhou, R.; Iyer, S.; Garcia, S.P.; Miller, B.R.; Langner, L.M.; Grunewald, J.; Joung, J.K. Crispr c-to-g base editors for inducing targeted DNA transversions in human cells. Nat. Biotechnol. 2021, 39, 41-46. [CrossRef] [PubMed]
23. Rees, H.A.; Liu, D.R. Base editing: Precision chemistry on the genome and transcriptome of living cells. Nat. Rev. Genet. 2018, 19, 770-788. [CrossRef] [PubMed]
24. Anzalone, A.V.; Randolph, P.B.; Davis, J.R.; Sousa, A.A.; Koblan, L.W.; Levy, J.M.; Chen, P.J.; Wilson, C.; Newby, G.A.; Raguram, A.; et al. Search-and-replace genome editing without double-strand breaks or donor DNA. Nature 2019, 576, 149-157. [CrossRef] [PubMed]
25. Yarnall, M.T.N.; Ioannidi, E.I.; Schmitt-Ulms, C.; Krajeski, R.N.; Lim, J.; Villiger, L.; Zhou, W.; Jiang, K.; Garushyants, S.K.; Roberts, N.; et al. Drag-and-drop genome insertion of large sequences without double-strand DNA cleavage using crispr-directed integrases. Nat. Biotechnol. 2023, 41, 500-512. [CrossRef]
26. Ma, S.; Wang, X.; Hu, Y.; Lv, J.; Liu, C.; Liao, K.; Guo, X.; Wang, D.; Lin, Y.; Rong, Z. Enhancing site-specific DNA integration by a cas9 nuclease fused with a DNA donor-binding domain. Nucleic Acids Res. 2020, 48, 10590-10601. [CrossRef] [PubMed]
27. Durrant, M.G.; Fanton, A.; Tycko, J.; Hinks, M.; Chandrasekaran, S.S.; Perry, N.T.; Schaepe, J.; Du, P.P.; Lotfy, P.; Bassik, M.C.; et al. Systematic discovery of recombinases for efficient integration of large DNA sequences into the human genome. Nat. Biotechnol. 2023, 41, 488-499. [CrossRef] [PubMed]
28. Anzalone, A.V.; Gao, X.D.; Podracky, C.J.; Nelson, A.T.; Koblan, L.W.; Raguram, A.; Levy, J.M.; Mercer, J.A.M.; Liu, D.R. Programmable deletion, replacement, integration and inversion of large DNA sequences with twin prime editing. Nat. Biotechnol. 2022, 40, 731-740. [CrossRef]
29. Turan, S.; Zehe, C.; Kuehle, J.; Qiao, J.; Bode, J. Recombinase-mediated cassette exchange (rmce)—A rapidly-expanding toolbox for targeted genomic modifications. Gene 2013, 515, 1-27. [CrossRef]
30. Brown, W.R.; Lee, N.C.; Xu, Z.; Smith, M.C. Serine recombinases as tools for genome engineering. Methods 2011, 53, 372-379. [CrossRef]
31. Duportet, X.; Wroblewska, L.; Guye, P.; Li, Y.; Eyquem, J.; Rieders, J.; Rimchala, T.; Batt, G.; Weiss, R. A platform for rapid prototyping of synthetic gene networks in mammalian cells. Nucleic Acids Res. 2014, 42, 13440-13451. [CrossRef]
32. Gilbert, L.A.; Horlbeck, M.A.; Adamson, B.; Villalta, J.E.; Chen, Y.; Whitehead, E.H.; Guimaraes, C.; Panning, B.; Ploegh, H.L.; Bassik, M.C.; et al. Genome-scale crispr-mediated control of gene repression and activation. Cell 2014, 159, 647-661. [CrossRef]
33. Margolin, J.F.; Friedman, J.R.; Meyer, W.K.; Vissing, H.; Thiesen, H.J.; Rauscher, F.J., 3rd. Kruppel-associated boxes are potent transcriptional repression domains. Proc. Natl. Acad. Sci. USA 1994, 91, 4509-4513. [CrossRef] [PubMed]
34. Cai, R.; Lv, R.; Shi, X.; Yang, G.; Jin, J. Crispr/dcas9 tools: Epigenetic mechanism and application in gene transcriptional regulation. Int. J. Mol. Sci. 2023, 24, 14865. [CrossRef] [PubMed]
35. Fontana, L.; Alahouzou, Z.; Miccio, A.; Antoniou, P. Epigenetic regulation of beta-globin genes and the potential to treat hemoglobinopathies through epigenome editing. Genes 2023, 14, 577. [CrossRef] [PubMed]
36. Liu, X.S.; Wu, H.; Ji, X.; Stelzer, Y.; Wu, X.; Czauderna, S.; Shu, J.; Dadon, D.; Young, R.A.; Jaenisch, R. Editing DNA methylation in the mammalian genome. Cell 2016, 167, 233-247.e217. [CrossRef] [PubMed]
37. Hilton, I.B.; D'Ippolito, A.M.; Vockley, C.M.; Thakore, P.I.; Crawford, G.E.; Reddy, T.E.; Gersbach, C.A. Epigenome editing by a crispr-cas9-based acetyltransferase activates genes from promoters and enhancers. Nat. Biotechnol. 2015, 33, 510-517. [CrossRef]
38. Kwon, D.Y.; Zhao, Y.T.; Lamonica, J.M.; Zhou, Z. Locus-specific histone deacetylation using a synthetic crispr-cas9-based hdac. Nat. Commun. 2017, 8, 15315. [CrossRef]
39. Hawley, R.G.; Ramezani, A.; Hawley, T.S. Hematopoietic stem cells. Methods Enzymol. 2006, 419, 149-179.
40. Skulimowska, I.; Sosniak, J.; Gonka, M.; Szade, A.; Jozkowicz, A.; Szade, K. The biology of hematopoietic stem cells and its clinical implications. FEBS J. 2022, 289, 7740-7759. [CrossRef]
41. Naldini, L. Genetic engineering of hematopoiesis: Current stage of clinical translation and future perspectives. EMBO Mol. Med. 2019, 11, e9958. [CrossRef] [PubMed]
42. Ahmed, M.H.; Ghatge, M.S.; Safo, M.K. Hemoglobin: Structure, function and allostery. Subcell. Biochem. 2020, 94, 345-382. [PubMed]
43. Frati, G.; Miccio, A. Genome editing for beta-hemoglobinopathies: Advances and challenges. J. Clin. Med. 2021, 10, 482. [CrossRef] [PubMed]
44. Savitt, T.L.; Goldberg, M.F. Herrick's 1910 case report of sickle cell anemia. The rest of the story. JAMA 1989, 261, 266-271. [CrossRef] [PubMed]

45. Piel, F.B.; Steinberg, M.H.; Rees, D.C. Sickle cell disease. N. Engl. J. Med. 2017, 376, 1561-1573. [CrossRef] [PubMed]
46. Taher, A.T.; Musallam, K.M.; Cappellini, M.D. Beta-thalassemias. N. Engl. J. Med. 2021, 384, 727-743. [CrossRef] [PubMed]
47. Forget, B.G. Molecular basis of hereditary persistence of fetal hemoglobin. Ann. N. Y Acad. Sci. 1998, 850, 38-44. [CrossRef] [PubMed]
48. Bauer, D.E.; Kamran, S.C.; Lessard, S.; Xu, J.; Fujiwara, Y.; Lin, C.; Shao, Z.; Canver, M.C.; Smith, E.C.; Pinello, L.; et al. An erythroid enhancer of bcl11a subject to genetic variation determines fetal hemoglobin level. Science 2013, 342, 253-257. [CrossRef] [PubMed]
49. Canver, M.C.; Smith, E.C.; Sher, F.; Pinello, L.; Sanjana, N.E.; Shalem, O.; Chen, D.D.; Schupp, P.G.; Vinjamur, D.S.; Garcia, S.P.; et al. Bcl11a enhancer dissection by cas9-mediated in situ saturating mutagenesis. Nature 2015, 527, 192-197. [CrossRef]
50. Wu, Y.; Zeng, J.; Roscoe, B.P.; Liu, P.; Yao, Q.; Lazzarotto, C.R.; Clement, K.; Cole, M.A.; Luk, K.; Baricordi, C.; et al. Highly efficient therapeutic gene editing of human hematopoietic stem cells. Nat. Med. 2019, 25, 776-783. [CrossRef]
51. Frangoul, H.; Altshuler, D.; Cappellini, M.D.; Chen, Y.S.; Domm, J.; Eustace, B.K.; Foell, J.; de la Fuente, J.; Grupp, S.; Handgretinger, R.; et al. Crispr-cas9 gene editing for sickle cell disease and beta-thalassemia. N. Engl. J. Med. 2021, 384, 252-260. [CrossRef]
52. Fu, B.; Liao, J.; Chen, S.; Li, W.; Wang, Q.; Hu, J.; Yang, F.; Hsiao, S.; Jiang, Y.; Wang, L.; et al. Crispr-cas9-mediated gene editing of the bcl11a enhancer for pediatric beta(0)/beta(0) transfusion-dependent beta-thalassemia. Nat. Med. 2022, 28, 1573-1580. [CrossRef]
53. Zheng, B.; Liu, R.; Zhang, X.; Fu, B.; Xu, Y.; Shi, J.; Feng, X.; Wang, L.; Wang, C.; Liang, R.; et al. Efficacy and safety of brl-101, crispr-cas9-mediated gene editing of the bcl11a enhancer in transfusion-dependent $\beta$-thalassemia. Blood 2023, 142, 4995. [CrossRef]
54. Shi, J.; Fang, R.; Gao, Z.; Shi, Z.; Kuang, Z.; Zhang, Y.; Zhang, L.; Yang, H.; Zhang, Y.; Zou, D.; et al. Preliminary safety and efficacy results of edi001: An investigator initiated trial on crispr/cas9-modified autologous cd34+ hematopoietic stem and progenitor cells for patients with transfusion dependent $\beta$-thalassemia. Blood 2022, 140, 10652-10653. [CrossRef]
55. Smith, A.R.; Schiller, G.J.; Vercellotti, G.M.; Kwiatkowski, J.L.; Krishnamurti, L.; Esrick, E.B.M.; William, D.A.; Miller, W.P.; Woolfson, A.; Walters, M.C. Preliminary results of a phase $1 / 2$ clinical study of zinc finger nuclease-mediated editing of bcl11a in autologous hematopoietic stem cells for transfusion-dependent beta thalassemia. Blood 2019, 134, 3544. [CrossRef]
56. Walters, M.C.; Smith, A.R.; Schiller, G.J.; Esrick, E.B.; Williams, D.A.; Gogoleva, T.; Rouy, D.; Cockroft, B.M.; Vercellotti, G.M. Updated results of a phase $1 / 2$ clinical study of zinc finger nuclease-mediated editing of bcl11a in autologous hematopoietic stem cells for transfusion-dependent beta thalassemia. Blood 2021, 138, 3974. [CrossRef]
57. Liu, R.; Wang, L.; Xu, H.; Yin, X.; Liang, J.; Xie, W.; Yang, G.; LI, Y.; Zhou, Y.; Shi, L.; et al. Safety and efficacy of rm-001 in patients with transfusion-dependent $\beta$-thalassemia: Early results from the ongoing of autologous hbg $1 / 2$ promoter-modified cd34+ hematopoietic stem and progenitor cells. HemaSphere 2023, 7, e613965e. [CrossRef]
58. Wang, L.; Xu, H.; Liang, J.; Li, Y.; Shi, L.; Xiao, B.; Huang, J.; Xu, X.; Zhang, X. Initial safety and efficacy study of rm-001, autologous hbg $1 / 2$ promoter-modified cd34+ hematopoietic stem and progenitor cells, in transfusion-dependent $\beta$ eta-thalassemia. HemaSphere 2022, 6, 1347-1348. [CrossRef]
59. De Dreuzy, E.; Heath, J.; Zuris, J.A.; Sousa, P.; Viswanathan, R.; Scott, S.; Da Silva, J.; Ta, T.; Capehart, S.; Wang, T.; et al. Edit-301: An experimental autologous cell therapy comprising cas12a-rnp modified mpb-cd34+ cells for the potential treatment of scd. Blood 2019, 134, 4636. [CrossRef]
60. Antoniou, P.; Hardouin, G.; Martinucci, P.; Frati, G.; Felix, T.; Chalumeau, A.; Fontana, L.; Martin, J.; Masson, C.; Brusson, M.; et al. Base-editing-mediated dissection of a gamma-globin cis-regulatory element for the therapeutic reactivation of fetal hemoglobin expression. Nat. Commun. 2022, 13, 6618. [CrossRef] [PubMed]
61. Costa, F.C.; Fedosyuk, H.; Neades, R.; de Los Rios, J.B.; Barbas, C.F., 3rd; Peterson, K.R. Induction of fetal hemoglobin in vivo mediated by a synthetic gamma-globin zinc finger activator. Anemia 2012, 2012, 507894. [CrossRef]
62. Wilber, A.; Tschulena, U.; Hargrove, P.W.; Kim, Y.S.; Persons, D.A.; Barbas, C.F., 3rd; Nienhuis, A.W. A zinc-finger transcriptional activator designed to interact with the gamma-globin gene promoters enhances fetal hemoglobin production in primary human adult erythroblasts. Blood 2010, 115, 3033-3041. [CrossRef] [PubMed]
63. Dever, D.P.; Bak, R.O.; Reinisch, A.; Camarena, J.; Washington, G.; Nicolas, C.E.; Pavel-Dinu, M.; Saxena, N.; Wilkens, A.B.; Mantri, S.; et al. Crispr/cas9 beta-globin gene targeting in human haematopoietic stem cells. Nature 2016, 539, 384-389. [CrossRef] [PubMed]
64. DeWitt, M.A.; Magis, W.; Bray, N.L.; Wang, T.; Berman, J.R.; Urbinati, F.; Heo, S.J.; Mitros, T.; Munoz, D.P.; Boffelli, D.; et al. Selection-free genome editing of the sickle mutation in human adult hematopoietic stem/progenitor cells. Sci. Transl. Med. 2016, 8, 360ra134. [CrossRef] [PubMed]
65. Lattanzi, A.; Camarena, J.; Lahiri, P.; Segal, H.; Srifa, W.; Vakulskas, C.A.; Frock, R.L.; Kenrick, J.; Lee, C.; Talbott, N.; et al. Development of beta-globin gene correction in human hematopoietic stem cells as a potential durable treatment for sickle cell disease. Sci. Transl. Med. 2021, 13, eabf2444. [CrossRef] [PubMed]
66. Kanter, J.; DiPersio, J.F.; Leavey, P.; Shyr, D.C.; Thompson, A.A.; Porteus, M.H.; Intondi, A.; Lahiri, P.; Dever, D.P.; Petrusich, A.; et al. Cedar trial in progress: A first in human, phase $1 / 2$ study of the correction of a single nucleotide mutation in autologous hscs (gph101) to convert hbs to hba for treating severe scd. Blood 2021, 138, 1864. [CrossRef]

67. Cromer, M.K.; Camarena, J.; Martin, R.M.; Lesch, B.J.; Vakulskas, C.A.; Bode, N.M.; Kurgan, G.; Collingwood, M.A.; Rettig, G.R.; Behlke, M.A.; et al. Gene replacement of alpha-globin with beta-globin restores hemoglobin balance in beta-thalassemia-derived hematopoietic stem and progenitor cells. Nat. Med. 2021, 27, 677-687. [CrossRef]
68. Pavani, G.; Fabiano, A.; Laurent, M.; Amor, F.; Cantelli, E.; Chalumeau, A.; Maule, G.; Tachtsidi, A.; Concordet, J.P.; Cereseto, A.; et al. Correction of beta-thalassemia by crispr/cas9 editing of the alpha-globin locus in human hematopoietic stem cells. Blood Adv. 2021, 5, 1137-1153. [CrossRef] [PubMed]
69. Chu, H.; Ortega, M.; Feliciano, P.; Winton, V.; Xu, C.; Haupt, D.; McDonald, T.; Martinez, S.; Liquori, A.; Marshall, J.; et al. Conversion of hbs to hb g-makassar by adenine base editing is compatible with normal hemoglobin function. Blood 2021, 38, 951. [CrossRef]
70. Newby, G.A.; Yen, J.S.; Woodard, K.J.; Mayuranathan, T.; Lazzarotto, C.R.; Li, Y.; Sheppard-Tillman, H.; Porter, S.N.; Yao, Y.; Mayberry, K.; et al. Base editing of haematopoietic stem cells rescues sickle cell disease in mice. Nature 2021, 595, 295-302. [CrossRef]
71. Hardouin, G.; Antoniou, P.; Martinucci, P.; Felix, T.; Manceau, S.; Joseph, L.; Masson, C.; Scaramuzza, S.; Ferrari, G.; Cavazzana, M.; et al. Adenine base editor-mediated correction of the common and severe ivs1-110 (g > a) beta-thalassemia mutation. Blood 2023, 141, 1169-1179. [CrossRef] [PubMed]
72. Everette, K.A.; Newby, G.A.; Levine, R.M.; Mayberry, K.; Jang, Y.; Mayuranathan, T.; Nimmagadda, N.; Dempsey, E.; Li, Y.; Bhoopalan, S.V.; et al. Ex vivo prime editing of patient haematopoietic stem cells rescues sickle-cell disease phenotypes after engraftment in mice. Nat. Biomed. Eng. 2023, 7, 616-628. [CrossRef] [PubMed]
73. Li, C.; Georgakopoulou, A.; Newby, G.A.; Chen, P.J.; Everette, K.A.; Paschoudi, K.; Vlachaki, E.; Gil, S.; Anderson, A.K.; Koob, T.; et al. In vivo hsc prime editing rescues sickle cell disease in a mouse model. Blood 2023, 141, 2085-2099. [CrossRef]
74. Notarangelo, L.D. Primary immunodeficiencies. J. Allergy Clin. Immunol. 2010, 125, S182-S194. [CrossRef] [PubMed]
75. Cicalese, M.P.; Ferrua, F.; Castagnaro, L.; Rolfe, K.; De Boever, E.; Reinhardt, R.R.; Appleby, J.; Roncarolo, M.G.; Aiuti, A. Gene therapy for adenosine deaminase deficiency: A comprehensive evaluation of short- and medium-term safety. Mol. Ther. 2018, 26, 917-931. [CrossRef]
76. Mamcarz, E.; Zhou, S.; Lockey, T.; Abdelsamed, H.; Cross, S.J.; Kang, G.; Ma, Z.; Condori, J.; Dowdy, J.; Triplett, B.; et al. Lentiviral gene therapy combined with low-dose busulfan in infants with scid-x1. N. Engl. J. Med. 2019, 380, 1525-1534. [CrossRef]
77. Sacco, M.G.; Ungari, M.; Cato, E.M.; Villa, A.; Strina, D.; Notarangelo, L.D.; Jonkers, J.; Zecca, L.; Facchetti, F.; Vezzoni, P. Lymphoid abnormalities in cd40 ligand transgenic mice suggest the need for tight regulation in gene therapy approaches to hyper immunoglobulin m (igm) syndrome. Cancer Gene Ther. 2000, 7, 1299-1306. [CrossRef]
78. Hubbard, N.; Hagin, D.; Sommer, K.; Song, Y.; Khan, I.; Clough, C.; Ochs, H.D.; Rawlings, D.J.; Scharenberg, A.M.; Torgerson, T.R. Targeted gene editing restores regulated cd40l function in x-linked hyper-igm syndrome. Blood 2016, 127, 2513-2522. [CrossRef]
79. Vavassori, V.; Mercuri, E.; Marcovecchio, G.E.; Castiello, M.C.; Schiroli, G.; Albano, L.; Margulies, C.; Buquicchio, F.; Fontana, E.; Beretta, S.; et al. Modeling, optimization, and comparable efficacy of $t$ cell and hematopoietic stem cell gene editing for treating hyper-igm syndrome. EMBO Mol. Med. 2021, 13, e13545. [CrossRef]
80. Kuo, C.Y.; Long, J.D.; Campo-Fernandez, B.; de Oliveira, S.; Cooper, A.R.; Romero, Z.; Hoban, M.D.; Joglekar, A.V.; Lill, G.R.; Kaufman, M.L.; et al. Site-specific gene editing of human hematopoietic stem cells for x-linked hyper-igm syndrome. Cell Rep. 2018, 23, 2606-2616. [CrossRef]
81. Asperti, C.; Canarutto, D.; Porcellini, S.; Sanvito, F.; Cecere, F.; Vavassori, V.; Ferrari, S.; Rovelli, E.; Albano, L.; Jacob, A.; et al. Scalable gmp-compliant gene correction of cd4+ t cells with idlv template functionally validated in vitro and in vivo. Mol. Ther. Methods Clin. Dev. 2023, 30, 546-557. [CrossRef]
82. van Til, N.P.; Sarwari, R.; Visser, T.P.; Hauer, J.; Lagresle-Peyrou, C.; van der Velden, G.; Malshetty, V.; Cortes, P.; Jollet, A.; Danos, O.; et al. Recombination-activating gene 1 (rag1)-deficient mice with severe combined immunodeficiency treated with lentiviral gene therapy demonstrate autoimmune omenn-like syndrome. J. Allergy Clin. Immunol. 2014, 133, 1116-1123. [CrossRef] [PubMed]
83. Castiello, M.C.; Brandas, C.; Ferrari, S.; Porcellini, S.; Sacchetti, N.; Canarutto, D.; Draghici, E.; Merelli, I.; Barcella, M.; Pelosi, G.; et al. Exonic knockout and knockin gene editing in hematopoietic stem and progenitor cells rescues rag1 immunodeficiency. Sci. Transl. Med. 2024, 16, eadh8162. [CrossRef] [PubMed]
84. De Ravin, S.S.; Li, L.; Wu, X.; Choi, U.; Allen, C.; Koontz, S.; Lee, J.; Theobald-Whiting, N.; Chu, J.; Garofalo, M.; et al. Crispr-cas9 gene repair of hematopoietic stem cells from patients with x-linked chronic granulomatous disease. Sci. Transl. Med. 2017, 9, eaah3480. [CrossRef] [PubMed]
85. Sweeney, C.L.; Zou, J.; Choi, U.; Merling, R.K.; Liu, A.; Bodansky, A.; Burkett, S.; Kim, J.W.; De Ravin, S.S.; Malech, H.L. Targeted repair of cybb in x-cgd ipscs requires retention of intronic sequences for expression and functional correction. Mol. Ther. 2017, 25, 321-330. [CrossRef]
86. Sweeney, C.L.; Pavel-Dinu, M.; Choi, U.; Brault, J.; Liu, T.; Koontz, S.; Li, L.; Theobald, N.; Lee, J.; Bello, E.A.; et al. Correction of x-cgd patient hspcs by targeted cybb cdna insertion using crispr/cas9 with 53bp1 inhibition for enhanced homology-directed repair. Gene Ther. 2021, 28, 373-390. [CrossRef] [PubMed]
87. De Ravin, S.S.; Brault, J.; Meis, R.J.; Liu, S.; Li, L.; Pavel-Dinu, M.; Lazzarotto, C.R.; Liu, T.; Koontz, S.M.; Choi, U.; et al. Enhanced homology-directed repair for highly efficient gene editing in hematopoietic stem/progenitor cells. Blood 2021, 137, 2598-2608. [CrossRef] [PubMed]

88. Klatt, D.; Cheng, E.; Hoffmann, D.; Santilli, G.; Thrasher, A.J.; Brendel, C.; Schambach, A. Differential transgene silencing of myeloid-specific promoters in the aavs1 safe harbor locus of induced pluripotent stem cell-derived myeloid cells. Hum. Gene Ther. 2020, 31, 199-210. [CrossRef] [PubMed]
89. Merling, R.K.; Kuhns, D.B.; Sweeney, C.L.; Wu, X.; Burkett, S.; Chu, J.; Lee, J.; Koontz, S.; Di Pasquale, G.; Afione, S.A.; et al. Gene-edited pseudogene resurrection corrects p47(phox)-deficient chronic granulomatous disease. Blood Adv. 2017, 1, 270-278. [CrossRef] [PubMed]
90. Nasri, M.; Ritter, M.; Mir, P.; Dannenmann, B.; Aghaallaei, N.; Amend, D.; Makaryan, V.; Xu, Y.; Fletcher, B.; Bernhard, R.; et al. Crispr/cas9-mediated elane knockout enables neutrophilic maturation of primary hematopoietic stem and progenitor cells and induced pluripotent stem cells of severe congenital neutropenia patients. Haematologica 2020, 105, 598-609. [CrossRef]
91. Rao, S.; Yao, Y.; Soares de Brito, J.; Yao, Q.; Shen, A.H.; Watkinson, R.E.; Kennedy, A.L.; Coyne, S.; Ren, C.; Zeng, J.; et al. Dissecting elane neutropenia pathogenicity by human hsc gene editing. Cell Stem Cell 2021, 28, 833-845.e835. [CrossRef]
92. Sabo, P.; Makaryan, V.; Dicken, Y.; Povodovski, L.; Rockah, L.; Bar, T.; Gabay, M.; Elinger, D.; Segal, E.; Haimov, O.; et al. Mutant allele knockout with novel crispr nuclease promotes myelopoiesis in elane neutropenia. Mol. Ther. Methods Clin. Dev. 2022, 26, 119-131. [CrossRef]
93. Dadi, H.K.; Simon, A.J.; Roifman, C.M. Effect of cd3delta deficiency on maturation of alpha/beta and gamma/delta t-cell lineages in severe combined immunodeficiency. N. Engl. J. Med. 2003, 349, 1821-1828. [CrossRef] [PubMed]
94. McAuley, G.E.; Yiu, G.; Chang, P.C.; Newby, G.A.; Campo-Fernandez, B.; Fitz-Gibbon, S.T.; Wu, X.; Kang, S.L.; Garibay, A.; Butler, J.; et al. Human t cell generation is restored in cd3delta severe combined immunodeficiency through adenine base editing. Cell 2023, 186, 1398-1416.e1323. [CrossRef]
95. Thrasher, A.J.; Burns, S.O. Wasp: A key immunological multitasker. Nat. Rev. Immunol. 2010, 10, 182-192. [CrossRef]
96. Bernard, P.D. Single application in ocalex expansion therapy. Inf. Dent. 1971, 53, 2852-2853.
97. Hacein-Bey Abina, S.; Gaspar, H.B.; Blondeau, J.; Caccavelli, L.; Charrier, S.; Buckland, K.; Picard, C.; Six, E.; Himoudi, N.; Gilmour, K.; et al. Outcomes following gene therapy in patients with severe wiskott-aldrich syndrome. JAMA 2015, 313, 1550-1563. [CrossRef]
98. Magnani, A.; Semeraro, M.; Adam, F.; Booth, C.; Dupre, L.; Morris, E.C.; Gabrion, A.; Roudaut, C.; Borgel, D.; Toubert, A.; et al. Long-term safety and efficacy of lentiviral hematopoietic stem/progenitor cell gene therapy for wiskott-aldrich syndrome. Nat. Med. 2022, 28, 71-80. [CrossRef]
99. Gutierrez-Guerrero, A.; Sanchez-Hernandez, S.; Galvani, G.; Pinedo-Gomez, J.; Martin-Guerra, R.; Sanchez-Gilabert, A.; AguilarGonzalez, A.; Cobo, M.; Gregory, P.; Holmes, M.; et al. Comparison of zinc finger nucleases versus crispr-specific nucleases for genome editing of the wiskott-aldrich syndrome locus. Hum. Gene Ther. 2018, 29, 366-380. [CrossRef]
100. Laskowski, T.J.; Van Caeneghem, Y.; Pourebrahim, R.; Ma, C.; Ni, Z.; Garate, Z.; Crane, A.M.; Li, X.S.; Liao, W.; Gonzalez-Garay, M.; et al. Gene correction of ipscs from a wiskott-aldrich syndrome patient normalizes the lymphoid developmental and functional defects. Stem Cell Rep. 2016, 7, 139-148. [CrossRef] [PubMed]
101. Rai, R.; Romito, M.; Rivers, E.; Turchiano, G.; Blattner, G.; Vetharoy, W.; Ladon, D.; Andrieux, G.; Zhang, F.; Zinicola, M.; et al. Targeted gene correction of human hematopoietic stem cells for the treatment of wiskott—Aldrich syndrome. Nat. Commun. 2020, 11, 4034. [CrossRef] [PubMed]
102. Rai, R.; Thrasher, A.J.; Cavazza, A. Gene editing for the treatment of primary immunodeficiency diseases. Hum. Gene Ther. 2021, 32, 43-51. [CrossRef] [PubMed]
103. Guiraud, S.; Aartsma-Rus, A.; Vieira, N.M.; Davies, K.E.; van Ommen, G.J.; Kunkel, L.M. The pathogenesis and therapy of muscular dystrophies. Annu. Rev. Genom. Hum. Genet. 2015, 16, 281-308. [CrossRef] [PubMed]
104. Bladen, C.L.; Salgado, D.; Monges, S.; Foncuberta, M.E.; Kekou, K.; Kosma, K.; Dawkins, H.; Lamont, L.; Roy, A.J.; Chamova, T.; et al. The treat-nmd.dmd global database: Analysis of more than 7,000 duchenne muscular dystrophy mutations. Hum. Mutat. 2015, 36, 395-402. [CrossRef] [PubMed]
105. Monaco, A.P.; Neve, R.L.; Colletti-Feener, C.; Bertelson, C.J.; Kurnit, D.M.; Kunkel, L.M. Isolation of candidate cdnas for portions of the duchenne muscular dystrophy gene. Nature 1986, 323, 646-650. [CrossRef] [PubMed]
106. Hoffman, E.P.; Brown, R.H., Jr.; Kunkel, L.M. Dystrophin: The protein product of the duchenne muscular dystrophy locus. Cell 1987, 51, 919-928. [CrossRef] [PubMed]
107. Blake, D.J.; Weir, A.; Newey, S.E.; Davies, K.E. Function and genetics of dystrophin and dystrophin-related proteins in muscle. Physiol. Rev. 2002, 82, 291-329. [CrossRef] [PubMed]
108. Emery, A.E. Duchenne muscular dystrophy-meryon's disease. Neuromuscul. Disord. 1993, 3, 263-266. [CrossRef] [PubMed]
109. Bach, J.R.; O'Brien, J.; Krotenberg, R.; Alba, A.S. Management of end stage respiratory failure in duchenne muscular dystrophy. Muscle Nerve 1987, 10, 177-182. [CrossRef]
110. Broomfield, J.; Hill, M.; Guglieri, M.; Crowther, M.; Abrams, K. Life expectancy in duchenne muscular dystrophy: Reproduced individual patient data meta-analysis. Neurology 2021, 97, e2304-e2314. [CrossRef]
111. Wilton-Clark, H.; Yokota, T. Recent trends in antisense therapies for duchenne muscular dystrophy. Pharmaceutics 2023, 15, 778. [CrossRef]
112. Niks, E.H.; Aartsma-Rus, A. Exon skipping: A first in class strategy for duchenne muscular dystrophy. Expert. Opin. Biol. Ther. 2017, 17, 225-236. [CrossRef] [PubMed]

113. Bushby, K.; Finkel, R.; Wong, B.; Barohn, R.; Campbell, C.; Comi, G.P.; Connolly, A.M.; Day, J.W.; Flanigan, K.M.; Goemans, N.; et al. Ataluren treatment of patients with nonsense mutation dystrophinopathy. Muscle Nerve 2014, 50, 477-487. [CrossRef]
114. Happi Mbakam, C.; Tremblay, J.P. Gene therapy for duchenne muscular dystrophy: An update on the latest clinical developments. Expert. Rev. Neurother. 2023, 23, 905-920. [CrossRef] [PubMed]
115. Guiraud, S.; Davies, K.E. Pharmacological advances for treatment in duchenne muscular dystrophy. Curr. Opin. Pharmacol. 2017, 34, 36-48. [CrossRef]
116. Betts, C.; Saleh, A.F.; Arzumanov, A.A.; Hammond, S.M.; Godfrey, C.; Coursindel, T.; Gait, M.J.; Wood, M.J. Pip6-pmo, a new generation of peptide-oligonucleotide conjugates with improved cardiac exon skipping activity for dmd treatment. Mol. Ther. Nucleic Acids 2012, 1, e38. [CrossRef]
117. Goyenvalle, A.; Griffith, G.; Babbs, A.; El Andaloussi, S.; Ezzat, K.; Avril, A.; Dugovic, B.; Chaussenot, R.; Ferry, A.; Voit, T.; et al. Functional correction in mouse models of muscular dystrophy using exon-skipping tricyclo-DNA oligomers. Nat. Med. 2015, 21, 270-275. [CrossRef]
118. Mamchaoui, K.; Trollet, C.; Bigot, A.; Negroni, E.; Chaouch, S.; Wolff, A.; Kandalla, P.K.; Marie, S.; Di Santo, J.; St Guily, J.L.; et al. Immortalized pathological human myoblasts: Towards a universal tool for the study of neuromuscular disorders. Skelet. Muscle 2011, 1, 34. [CrossRef] [PubMed]
119. Alizadeh, F.; Abraghan, Y.J.; Farrokhi, S.; Yousefi, Y.; Mirahmadi, Y.; Eslahi, A.; Mojarrad, M. Production of duchenne muscular dystrophy cellular model using crispr-cas9 exon deletion strategy. Mol. Cell Biochem. 2023, 1-14. [CrossRef]
120. Soblechero-Martin, P.; Albiasu-Arteta, E.; Anton-Martinez, A.; de la Puente-Ovejero, L.; Garcia-Jimenez, I.; Gonzalez-Iglesias, G.; Larranaga-Aiestaran, I.; Lopez-Martinez, A.; Poyatos-Garcia, J.; Ruiz-Del-Yerro, E.; et al. Duchenne muscular dystrophy cell culture models created by crispr/cas9 gene editing and their application in drug screening. Sci. Rep. 2021, 11, 18188. [CrossRef]
121. Happi Mbakam, C.; Rousseau, J.; Tremblay, G.; Yameogo, P.; Tremblay, J.P. Prime editing permits the introduction of specific mutations in the gene responsible for duchenne muscular dystrophy. Int. J. Mol. Sci. 2022, 23, 6160. [CrossRef]
122. Ruiz-Del-Yerro, E.; Garcia-Jimenez, I.; Mamchaoui, K.; Arechavala-Gomeza, V. Myoblots: Dystrophin quantification by in-cell western assay for a streamlined development of duchenne muscular dystrophy (dmd) treatments. Neuropathol. Appl. Neurobiol. 2018, 44, 463-473. [CrossRef] [PubMed]
123. Geoffroy, M.; Pili, L.; Buffa, V.; Caroff, M.; Bigot, A.; Gicquel, E.; Rouby, G.; Richard, I.; Fragnoud, R. Crispr-cas9 ko cell line generation and development of a cell-based potency assay for raav-fkrp gene therapy. Cells 2023, 12, 2444. [CrossRef] [PubMed]
124. Bulfield, G.; Siller, W.G.; Wight, P.A.; Moore, K.J. X chromosome-linked muscular dystrophy (mdx) in the mouse. Proc. Natl. Acad. Sci. USA 1984, 81, 1189-1192. [CrossRef] [PubMed]
125. Min, Y.L.; Chemello, F.; Li, H.; Rodriguez-Caycedo, C.; Sanchez-Ortiz, E.; Mireault, A.A.; McAnally, J.R.; Shelton, J.M.; Zhang, Y.; Bassel-Duby, R.; et al. Correction of three prominent mutations in mouse and human models of duchenne muscular dystrophy by single-cut genome editing. Mol. Ther. 2020, 28, 2044-2055. [CrossRef]
126. Amoasii, L.; Long, C.; Li, H.; Mireault, A.A.; Shelton, J.M.; Sanchez-Ortiz, E.; McAnally, J.R.; Bhattacharyya, S.; Schmidt, F.; Grimm, D.; et al. Single-cut genome editing restores dystrophin expression in a new mouse model of muscular dystrophy. Sci. Transl. Med. 2017, 9, eaan8081. [CrossRef]
127. Min, Y.L.; Li, H.; Rodriguez-Caycedo, C.; Mireault, A.A.; Huang, J.; Shelton, J.M.; McAnally, J.R.; Amoasii, L.; Mammen, P.P.A.; Bassel-Duby, R.; et al. Crispr-cas9 corrects duchenne muscular dystrophy exon 44 deletion mutations in mice and human cells. Sci. Adv. 2019, 5, eaav4324. [CrossRef]
128. Egorova, T.V.; Zotova, E.D.; Reshetov, D.A.; Polikarpova, A.V.; Vassilieva, S.G.; Vlodavets, D.V.; Gavrilov, A.A.; Ulianov, S.V.; Buchman, V.L.; Deykin, A.V. Crispr/cas9-generated mouse model of duchenne muscular dystrophy recapitulating a newly identified large 430 kb deletion in the human dmd gene. Dis. Model. Mech. 2019, 12, dmm037655. [CrossRef]
129. Kim, K.; Ryu, S.M.; Kim, S.T.; Baek, G.; Kim, D.; Lim, K.; Chung, E.; Kim, S.; Kim, J.S. Highly efficient rna-guided base editing in mouse embryos. Nat. Biotechnol. 2017, 35, 435-437. [CrossRef]
130. Koo, T.; Lu-Nguyen, N.B.; Malerba, A.; Kim, E.; Kim, D.; Cappellari, O.; Cho, H.Y.; Dickson, G.; Popplewell, L.; Kim, J.S. Functional rescue of dystrophin deficiency in mice caused by frameshift mutations using campylobacter jejuni cas9. Mol. Ther. 2018, 26, 1529-1538. [CrossRef]
131. Pickar-Oliver, A.; Gough, V.; Bohning, J.D.; Liu, S.; Robinson-Hamm, J.N.; Daniels, H.; Majoros, W.H.; Devlin, G.; Asokan, A.; Gersbach, C.A. Full-length dystrophin restoration via targeted exon integration by aav-crispr in a humanized mouse model of duchenne muscular dystrophy. Mol. Ther. 2021, 29, 3243-3257. [CrossRef]
132. Young, C.S.; Mokhonova, E.; Quinonez, M.; Pyle, A.D.; Spencer, M.J. Creation of a novel humanized dystrophic mouse model of duchenne muscular dystrophy and application of a crispr/cas9 gene editing therapy. J. Neuromuscul. Dis. 2017, 4, 139-145. [CrossRef] [PubMed]
133. Kenjo, E.; Hozumi, H.; Makita, Y.; Iwabuchi, K.A.; Fujimoto, N.; Matsumoto, S.; Kimura, M.; Amano, Y.; Ifuku, M.; Naoe, Y.; et al. Low immunogenicity of lnp allows repeated administrations of crispr-cas9 mrna into skeletal muscle in mice. Nat. Commun. 2021, 12, 7101. [CrossRef]
134. Larcher, T.; Lafoux, A.; Tesson, L.; Remy, S.; Thepenier, V.; Francois, V.; Le Guiner, C.; Goubin, H.; Dutilleul, M.; Guigand, L.; et al. Characterization of dystrophin deficient rats: A new model for duchenne muscular dystrophy. PLoS ONE 2014, 9, e110371. [CrossRef]

135. Taglietti, V.; Kefi, K.; Bronisz-Budzynska, I.; Mirciloglu, B.; Rodrigues, M.; Cardone, N.; Coulpier, F.; Periou, B.; Gentil, C.; Goddard, M.; et al. Duchenne muscular dystrophy trajectory in r-dmddel52 preclinical rat model identifies comp as biomarker of fibrosis. Acta Neuropathol. Commun. 2022, 10, 60. [CrossRef] [PubMed]
136. Nakamura, K.; Fujii, W.; Tsuboi, M.; Tanihata, J.; Teramoto, N.; Takeuchi, S.; Naito, K.; Yamanouchi, K.; Nishihara, M. Generation of muscular dystrophy model rats with a crispr/cas system. Sci. Rep. 2014, 4, 5635. [CrossRef] [PubMed]
137. Daoud, C. Characterization and crispr-based treatment of dmd\&45 rat model-p677. European society for gene and cell therapy, 11-14 October 2022. 2022. Available online: https://www.genethon.com/genethons-latest-research-including-advances-in-gene-editing-and-aav-delivery-vectors-for-gene-therapy-to-be-featured-at-european-society-for-gene-and-cell-therapy-oct-11-142022/ (accessed on 3 May 2024).
138. Sui, T.; Lau, Y.S.; Liu, D.; Liu, T.; Xu, L.; Gao, Y.; Lai, L.; Li, Z.; Han, R. A novel rabbit model of duchenne muscular dystrophy generated by crispr/cas9. Dis. Model. Mech. 2018, 11, dmm032201. [CrossRef] [PubMed]
139. Echigoya, Y.; Trieu, N.; Duddy, W.; Moulton, H.M.; Yin, H.; Partridge, T.A.; Hoffman, E.P.; Kornegay, J.N.; Rohret, F.A.; Rogers, C.S.; et al. A dystrophin exon-52 deleted miniature pig model of duchenne muscular dystrophy and evaluation of exon skipping. Int. J. Mol. Sci. 2021, 22, 13065. [CrossRef]
140. Yu, H.H.; Zhao, H.; Qing, Y.B.; Pan, W.R.; Jia, B.Y.; Zhao, H.Y.; Huang, X.X.; Wei, H.J. Porcine zygote injection with cas9/sgrna results in dmd-modified pig with muscle dystrophy. Int. J. Mol. Sci. 2016, 17, 1668. [CrossRef]
141. Xie, J.; Ge, W.; Li, N.; Liu, Q.; Chen, F.; Yang, X.; Huang, X.; Ouyang, Z.; Zhang, Q.; Zhao, Y.; et al. Efficient base editing for multiple genes and loci in pigs using base editors. Nat. Commun. 2019, 10, 2852. [CrossRef]
142. Sharp, N.J.; Kornegay, J.N.; Van Camp, S.D.; Herbstreith, M.H.; Secure, S.L.; Kettle, S.; Hung, W.Y.; Constantinou, C.D.; Dykstra, M.J.; Roses, A.D.; et al. An error in dystrophin mrna processing in golden retriever muscular dystrophy, an animal homologue of duchenne muscular dystrophy. Genomics 1992, 13, 115-121. [CrossRef] [PubMed]
143. Cooper, B.J.; Winand, N.J.; Stedman, H.; Valentine, B.A.; Hoffman, E.P.; Kunkel, L.M.; Scott, M.O.; Fischbeck, K.H.; Kornegay, J.N.; Avery, R.J.; et al. The homologue of the duchenne locus is defective in x-linked muscular dystrophy of dogs. Nature 1988, 334, 154-156. [CrossRef] [PubMed]
144. Shimatsu, Y.; Katagiri, K.; Furuta, T.; Nakura, M.; Tanioka, Y.; Yuasa, K.; Tomohiro, M.; Kornegay, J.N.; Nonaka, I.; Takeda, S. Canine x-linked muscular dystrophy in japan (cxmdj). Exp. Anim. 2003, 52, 93-97. [CrossRef] [PubMed]
145. Duan, D. Duchenne muscular dystrophy gene therapy in the canine model. Hum. Gene Ther. Clin. Dev. 2015, 26, 57-69. [CrossRef] [PubMed]
146. Oh, H.J.; Chung, E.; Kim, J.; Kim, M.J.; Kim, G.A.; Lee, S.H.; Ra, K.; Eom, K.; Park, S.; Chae, J.H.; et al. Generation of a dystrophin mutant in dog by nuclear transfer using crispr/cas9-mediated somatic cells: A preliminary study. Int. J. Mol. Sci. 2022, 23, 2898. [CrossRef] [PubMed]
147. Chen, Y.; Zheng, Y.; Kang, Y.; Yang, W.; Niu, Y.; Guo, X.; Tu, Z.; Si, C.; Wang, H.; Xing, R.; et al. Functional disruption of the dystrophin gene in rhesus monkey using crispr/cas9. Hum. Mol. Genet. 2015, 24, 3764-3774. [CrossRef] [PubMed]
148. Aartsma-Rus, A.; Van Deutekom, J.C.; Fokkema, I.F.; Van Ommen, G.J.; Den Dunnen, J.T. Entries in the leiden duchenne muscular dystrophy mutation database: An overview of mutation types and paradoxical cases that confirm the reading-frame rule. Muscle Nerve 2006, 34, 135-144. [CrossRef] [PubMed]
149. Erkut, E.; Yokota, T. Crispr therapeutics for duchenne muscular dystrophy. Int. J. Mol. Sci. 2022, 23, 1832. [CrossRef]
150. Chey, Y.C.J.; Arudkumar, J.; Aartsma-Rus, A.; Adikusuma, F.; Thomas, P.Q. Crispr applications for duchenne muscular dystrophy: From animal models to potential therapies. WIREs Mech. Dis. 2023, 15, e1580. [CrossRef]
151. Nelson, C.E.; Hakim, C.H.; Ousterout, D.G.; Thakore, P.I.; Moreb, E.A.; Castellanos Rivera, R.M.; Madhavan, S.; Pan, X.; Ran, F.A.; Yan, W.X.; et al. In vivo genome editing improves muscle function in a mouse model of duchenne muscular dystrophy. Science 2016, 351, 403-407. [CrossRef]
152. Long, C.; Amoasii, L.; Mireault, A.A.; McAnally, J.R.; Li, H.; Sanchez-Ortiz, E.; Bhattacharyya, S.; Shelton, J.M.; Bassel-Duby, R.; Olson, E.N. Postnatal genome editing partially restores dystrophin expression in a mouse model of muscular dystrophy. Science 2016, 351, 400-403. [CrossRef]
153. Tabebordbar, M.; Zhu, K.; Cheng, J.K.W.; Chew, W.L.; Widrick, J.J.; Yan, W.X.; Maesner, C.; Wu, E.Y.; Xiao, R.; Ran, F.A.; et al. In vivo gene editing in dystrophic mouse muscle and muscle stem cells. Science 2016, 351, 407-411. [CrossRef]
154. Xu, L.; Lau, Y.S.; Gao, Y.; Li, H.; Han, R. Life-long aav-mediated crispr genome editing in dystrophic heart improves cardiomyopathy without causing serious lesions in mdx mice. Mol. Ther. 2019, 27, 1407-1414. [CrossRef]
155. Nelson, C.E.; Wu, Y.; Gemberling, M.P.; Oliver, M.L.; Waller, M.A.; Bohning, J.D.; Robinson-Hamm, J.N.; Bulaklak, K.; Castellanos Rivera, R.M.; Collier, J.H.; et al. Long-term evaluation of aav-crispr genome editing for duchenne muscular dystrophy. Nat. Med. 2019, 25, 427-432. [CrossRef]
156. Hakim, C.H.; Wasala, N.B.; Nelson, C.E.; Wasala, L.P.; Yue, Y.; Louderman, J.A.; Lessa, T.B.; Dai, A.; Zhang, K.; Jenkins, G.J.; et al. Aav crispr editing rescues cardiac and muscle function for 18 months in dystrophic mice. JCI Insight 2018, 3, e124297. [CrossRef]
157. Duchene, B.L.; Cherif, K.; Iyombe-Engembe, J.P.; Guyon, A.; Rousseau, J.; Ouellet, D.L.; Barbeau, X.; Lague, P.; Tremblay, J.P. Crispr-induced deletion with sacas9 restores dystrophin expression in dystrophic models in vitro and in vivo. Mol. Ther. 2018, 26, 2604-2616. [CrossRef]

158. Bengtsson, N.E.; Hall, J.K.; Odom, G.L.; Phelps, M.P.; Andrus, C.R.; Hawkins, R.D.; Hauschka, S.D.; Chamberlain, J.R.; Chamberlain, J.S. Muscle-specific crispr/cas9 dystrophin gene editing ameliorates pathophysiology in a mouse model for duchenne muscular dystrophy. Nat. Commun. 2017, 8, 14454. [CrossRef]
159. Bengtsson, N.E.; Tasfaout, H.; Hauschka, S.D.; Chamberlain, J.S. Dystrophin gene-editing stability is dependent on dystrophin levels in skeletal but not cardiac muscles. Mol. Ther. 2021, 29, 1070-1085. [CrossRef]
160. Maino, E.; Wojtal, D.; Evagelou, S.L.; Farheen, A.; Wong, T.W.Y.; Lindsay, K.; Scott, O.; Rizvi, S.Z.; Hyatt, E.; Rok, M.; et al. Targeted genome editing in vivo corrects a dmd duplication restoring wild-type dystrophin expression. EMBO Mol. Med. 2021, 13, e13228. [CrossRef]
161. Moretti, A.; Fonteyne, L.; Giesert, F.; Hoppmann, P.; Meier, A.B.; Bozoglu, T.; Baehr, A.; Schneider, C.M.; Sinnecker, D.; Klett, K.; et al. Somatic gene editing ameliorates skeletal and cardiac muscle failure in pig and human models of duchenne muscular dystrophy. Nat. Med. 2020, 26, 207-214. [CrossRef]
162. Amoasii, L.; Hildyard, J.C.W.; Li, H.; Sanchez-Ortiz, E.; Mireault, A.; Caballero, D.; Harron, R.; Stathopoulou, T.R.; Massey, C.; Shelton, J.M.; et al. Gene editing restores dystrophin expression in a canine model of duchenne muscular dystrophy. Science 2018, 362, 86-91. [CrossRef]
163. Ousterout, D.G.; Kabadi, A.M.; Thakore, P.I.; Majoros, W.H.; Reddy, T.E.; Gersbach, C.A. Multiplex crispr/cas9-based genome editing for correction of dystrophin mutations that cause duchenne muscular dystrophy. Nat. Commun. 2015, 6, 6244. [CrossRef]
164. van Deutekom, J.C.; van Ommen, G.J. Advances in duchenne muscular dystrophy gene therapy. Nat. Rev. Genet. 2003, 4, 774-783. [CrossRef]
165. Nakamura, A. Moving towards successful exon-skipping therapy for duchenne muscular dystrophy. J. Hum. Genet. 2017, 62, 871-876. [CrossRef] [PubMed]
166. Chen, G.; Wei, T.; Yang, H.; Li, G.; Li, H. Crispr-based therapeutic gene editing for duchenne muscular dystrophy: Advances, challenges and perspectives. Cells 2022, 11, 2964. [CrossRef]
167. Zhang, Y.; Li, H.; Nishiyama, T.; McAnally, J.R.; Sanchez-Ortiz, E.; Huang, J.; Mammen, P.P.A.; Bassel-Duby, R.; Olson, E.N. A humanized knockin mouse model of duchenne muscular dystrophy and its correction by crispr-cas9 therapeutic gene editing. Mol. Ther. Nucleic Acids 2022, 29, 525-537. [CrossRef]
168. Karri, D.R.; Zhang, Y.; Chemello, F.; Min, Y.L.; Huang, J.; Kim, J.; Mammen, P.P.A.; Xu, L.; Liu, N.; Bassel-Duby, R.; et al. Long-term maintenance of dystrophin expression and resistance to injury of skeletal muscle in gene edited dmd mice. Mol. Ther. Nucleic Acids 2022, 28, 154-167. [CrossRef]
169. Zhang, Y.; Li, H.; Min, Y.L.; Sanchez-Ortiz, E.; Huang, J.; Mireault, A.A.; Shelton, J.M.; Kim, J.; Mammen, P.P.A.; Bassel-Duby, R.; et al. Enhanced crispr-cas9 correction of duchenne muscular dystrophy in mice by a self-complementary aav delivery system. Sci. Adv. 2020, 6, eaay6812. [CrossRef] [PubMed]
170. Li, H.L.; Fujimoto, N.; Sasakawa, N.; Shirai, S.; Ohkame, T.; Sakuma, T.; Tanaka, M.; Amano, N.; Watanabe, A.; Sakurai, H.; et al. Precise correction of the dystrophin gene in duchenne muscular dystrophy patient induced pluripotent stem cells by talen and crispr-cas9. Stem Cell Rep. 2015, 4, 143-154. [CrossRef]
171. Zhang, Y.; Nishiyama, T.; Li, H.; Huang, J.; Atmanli, A.; Sanchez-Ortiz, E.; Wang, Z.; Mireault, A.A.; Mammen, P.P.A.; Bassel-Duby, R.; et al. A consolidated aav system for single-cut crispr correction of a common duchenne muscular dystrophy mutation. Mol. Ther. Methods Clin. Dev. 2021, 22, 122-132. [CrossRef]
172. Long, C.; McAnally, J.R.; Shelton, J.M.; Mireault, A.A.; Bassel-Duby, R.; Olson, E.N. Prevention of muscular dystrophy in mice by crispr/cas9-mediated editing of germline DNA. Science 2014, 345, 1184-1188. [CrossRef] [PubMed]
173. Zhang, Y.; Long, C.; Li, H.; McAnally, J.R.; Baskin, K.K.; Shelton, J.M.; Bassel-Duby, R.; Olson, E.N. Crispr-cpf1 correction of muscular dystrophy mutations in human cardiomyocytes and mice. Sci. Adv. 2017, 3, e1602814. [CrossRef] [PubMed]
174. Zhu, P.; Wu, F.; Mosenson, J.; Zhang, H.; He, T.C.; Wu, W.S. Crispr/cas9-mediated genome editing corrects dystrophin mutation in skeletal muscle stem cells in a mouse model of muscle dystrophy. Mol. Ther. Nucleic Acids 2017, 7, 31-41. [CrossRef] [PubMed]
175. Mata Lopez, S.; Balog-Alvarez, C.; Vitha, S.; Bettis, A.K.; Canessa, E.H.; Kornegay, J.N.; Nghiem, P.P. Challenges associated with homologous directed repair using crispr-cas9 and talen to edit the dmd genetic mutation in canine duchenne muscular dystrophy. PLoS ONE 2020, 15, e0228072.
176. Suzuki, K.; Tsunekawa, Y.; Hernandez-Benitez, R.; Wu, J.; Zhu, J.; Kim, E.J.; Hatanaka, F.; Yamamoto, M.; Araoka, T.; Li, Z.; et al. In vivo genome editing via crispr/cas9 mediated homology-independent targeted integration. Nature 2016, 540, 144-149. [CrossRef] [PubMed]
177. Suzuki, K.; Izpisua Belmonte, J.C. In vivo genome editing via the hiti method as a tool for gene therapy. J. Hum. Genet. 2018, 63, 157-164. [CrossRef] [PubMed]
178. Aartsma-Rus, A.; Fokkema, I.; Verschuuren, J.; Ginjaar, I.; van Deutekom, J.; van Ommen, G.J.; den Dunnen, J.T. Theoretic applicability of antisense-mediated exon skipping for duchenne muscular dystrophy mutations. Hum. Mutat. 2009, 30, 293-299. [CrossRef] [PubMed]
179. Jin, S.; Zong, Y.; Gao, Q.; Zhu, Z.; Wang, Y.; Qin, P.; Liang, C.; Wang, D.; Qiu, J.L.; Zhang, F.; et al. Cytosine, but not adenine, base editors induce genome-wide off-target mutations in rice. Science 2019, 364, 292-295. [CrossRef]

180. Lee, H.K.; Willi, M.; Miller, S.M.; Kim, S.; Liu, C.; Liu, D.R.; Hennighausen, L. Targeting fidelity of adenine and cytosine base editors in mouse embryos. Nat. Commun. 2018, 9, 4804. [CrossRef]
181. Ryu, S.M.; Koo, T.; Kim, K.; Lim, K.; Baek, G.; Kim, S.T.; Kim, H.S.; Kim, D.E.; Lee, H.; Chung, E.; et al. Adenine base editing in mouse embryos and an adult mouse model of duchenne muscular dystrophy. Nat. Biotechnol. 2018, 36, 536-539. [CrossRef]
182. Xu, L.; Zhang, C.; Li, H.; Wang, P.; Gao, Y.; Mokadam, N.A.; Ma, J.; Arnold, W.D.; Han, R. Efficient precise in vivo base editing in adult dystrophic mice. Nat. Commun. 2021, 12, 3719. [CrossRef]
183. Li, J.; Wang, K.; Zhang, Y.; Qi, T.; Yuan, J.; Zhang, L.; Qiu, H.; Wang, J.; Yang, H.T.; Dai, Y.; et al. Therapeutic exon skipping through a crispr-guided cytidine deaminase rescues dystrophic cardiomyopathy in vivo. Circulation 2021, 144, 1760-1776. [CrossRef] [PubMed]
184. Chemello, F.; Chai, A.C.; Li, H.; Rodriguez-Caycedo, C.; Sanchez-Ortiz, E.; Atmanli, A.; Mireault, A.A.; Liu, N.; Bassel-Duby, R.; Olson, E.N. Precise correction of duchenne muscular dystrophy exon deletion mutations by base and prime editing. Sci. Adv. 2021, 7, eabg4910. [CrossRef]
185. Davis, J.R.; Wang, X.; Witte, I.P.; Huang, T.P.; Levy, J.M.; Raguram, A.; Banskota, S.; Seidah, N.G.; Musunuru, K.; Liu, D.R. Publisher correction: Efficient in vivo base editing via single adenoassociated viruses with size-optimized genomes encoding compact adenine base editors. Nat. Biomed. Eng. 2022, 6, 1317. [CrossRef] [PubMed]
186. Godbout, K.; Tremblay, J.P. Prime editing for human gene therapy: Where are we now? Cells 2023, 12, 536. [CrossRef]
187. Happi Mbakam, C.; Roustant, J.; Rousseau, J.; Yameogo, P.; Lu, Y.; Bigot, A.; Mamchaoui, K.; Mouly, V.; Lamothe, G.; Tremblay, J.P. Prime editing strategies to mediate exon skipping in dmd gene. Front Med. 2023, 10, 1128557. [CrossRef] [PubMed]
188. Happi Mbakam, C.; Rousseau, J.; Lu, Y.; Bigot, A.; Mamchaoui, K.; Mouly, V.; Tremblay, J.P. Prime editing optimized rtt permits the correction of the c. $8713 \mathrm{cs}+$ mutation in dmd gene. Mol Ther Nucleic Acids 2022, 30, 272-285. [CrossRef]
189. Casas-Mollano, J.A.; Zinselmeier, M.H.; Erickson, S.E.; Smanski, M.J. Crispr-cas activators for engineering gene expression in higher eukaryotes. CRISPR J. 2020, 3, 350-364. [CrossRef]
190. Guiraud, S.; Roblin, D.; Davies, K.E. The potential of utrophin modulators for the treatment of duchenne muscular dystrophy. Expert. Opin. Orphan Drugs 2018, 6, 179-192. [CrossRef]
191. Love, D.R.; Hill, D.F.; Dickson, G.; Spurr, N.K.; Byth, B.C.; Marsden, R.F.; Walsh, F.S.; Edwards, Y.H.; Davies, K.E. An autosomal transcript in skeletal muscle with homology to dystrophin. Nature 1989, 339, 55-58. [CrossRef]
192. Tinsley, J.M.; Blake, D.J.; Roche, A.; Fairbrother, U.; Riss, J.; Byth, B.C.; Knight, A.E.; Kendrick-Jones, J.; Suthers, G.K.; Love, D.R.; et al. Primary structure of dystrophin-related protein. Nature 1992, 360, 591-593. [CrossRef]
193. Tinsley, J.; Deconinck, N.; Fisher, R.; Kahn, D.; Phelps, S.; Gillis, J.M.; Davies, K. Expression of full-length utrophin prevents muscular dystrophy in mdx mice. Nat. Med. 1998, 4, 1441-1444. [CrossRef]
194. Squire, S.; Raymackers, J.M.; Vandebrouck, C.; Potter, A.; Tinsley, J.; Fisher, R.; Gillis, J.M.; Davies, K.E. Prevention of pathology in mdx mice by expression of utrophin: Analysis using an inducible transgenic expression system. Hum. Mol. Genet. 2002, 11, 3333-3344. [CrossRef]
195. Fisher, R.; Tinsley, J.M.; Phelps, S.R.; Squire, S.E.; Townsend, E.R.; Martin, J.E.; Davies, K.E. Non-toxic ubiquitous over-expression of utrophin in the mdx mouse. Neuromuscul. Disord. 2001, 11, 713-721. [CrossRef] [PubMed]
196. Guiraud, S.; Squire, S.E.; Edwards, B.; Chen, H.; Burns, D.T.; Shah, N.; Babbs, A.; Davies, S.G.; Wynne, G.M.; Russell, A.J.; et al. Second-generation compound for the modulation of utrophin in the therapy of dmd. Hum. Mol. Genet. 2015, 24, 4212-4224. [CrossRef]
197. Wojtal, D.; Kemaladewi, D.U.; Malam, Z.; Abdullah, S.; Wong, T.W.; Hyatt, E.; Baghestani, Z.; Pereira, S.; Stavropoulos, J.; Mouly, V.; et al. Spell checking nature: Versatility of crispr/cas9 for developing treatments for inherited disorders. Am. J. Hum. Genet. 2016, 98, 90-101. [CrossRef] [PubMed]
198. Liao, H.K.; Hatanaka, F.; Araoka, T.; Reddy, P.; Wu, M.Z.; Sui, Y.; Yamauchi, T.; Sakurai, M.; O'Keefe, D.D.; Nunez-Delicado, E.; et al. In vivo target gene activation via crispr/cas9-mediated trans-epigenetic modulation. Cell 2017, 171, 1495-1507 e1415. [CrossRef]
199. Lek, A.; Wong, B.; Keeler, A.; Blackwood, M.; Ma, K.; Huang, S.; Sylvia, K.; Batista, A.R.; Artinian, R.; Kokoski, D.; et al. Death after high-dose raav9 gene therapy in a patient with duchenne's muscular dystrophy. N. Engl. J. Med. 2023, 389, 1203-1210. [CrossRef]
200. Duan, D. Lethal immunotoxicity in high-dose systemic aav therapy. Mol. Ther. 2023, 31, 3123-3126. [CrossRef] [PubMed]
201. Klaver-Flores, S.; Zittersteijn, H.A.; Cante-Barrett, K.; Lankester, A.; Hoeben, R.C.; Goncalves, M.; Pike-Overzet, K.; Staal, F.J.T. Genomic engineering in human hematopoietic stem cells: Hype or hope? Front. Genome Ed. 2020, 2, 615619. [CrossRef] [PubMed]
202. Duan, D. Systemic aav micro-dystrophin gene therapy for duchenne muscular dystrophy. Mol. Ther. 2018, 26, 2337-2356. [CrossRef] [PubMed]
203. Vos, P.D.; Gandadireja, A.P.; Rossetti, G.; Siira, S.J.; Mantegna, J.L.; Filipovska, A.; Rackham, O. Mutational rescue of the activity of high-fidelity cas9 enzymes. Cell Rep. Methods 2024, 4, 100756. [CrossRef] [PubMed]
204. Kanter, J.; Thompson, A.A.; Pierciey, F.J., Jr.; Hsieh, M.; Uchida, N.; Leboulch, P.; Schmidt, M.; Bonner, M.; Guo, R.; Miller, A.; et al. Lovo-cel gene therapy for sickle cell disease: Treatment process evolution and outcomes in the initial groups of the hgb-206 study. Am. J. Hematol. 2023, 98, 11-22. [CrossRef] [PubMed]
205. Sheridan, C. The world's first crispr therapy is approved: Who will receive it? Nat. Biotechnol. 2024, 42, 3-4. [CrossRef] [PubMed]

206. Mendell, J.R.; Proud, C.; Zaidman, C.M.; Mason, S.; Darton, E.; Wang, S.; Wandel, C.; Murphy, A.P.; Mercuri, E.; Muntoni, F.; et al. Practical considerations for delandistrogene moxeparvovec gene therapy in patients with duchenne muscular dystrophy. Pediatr. Neurol. 2024, 153, 11-18. [CrossRef]
207. Ferdosi, S.R.; Ewaisha, R.; Moghadam, F.; Krishna, S.; Park, J.G.; Ebrahimkhani, M.R.; Kiani, S.; Anderson, K.S. Multifunctional crispr-cas9 with engineered immunosilenced human t cell epitopes. Nat. Commun. 2019, 10, 1842. [CrossRef]

Disclaimer/Publisher's Note: The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.